<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Genotype / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">

AXB/BXA Genotypes Database

 <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
<P><A HREF="http://gn1.genenetwork.org/genotypes/AXBXA.geno" class="fs14">Download</A> the entire AXB/BXA genotype file used in GeneNetwork (n = 2446 unique strain distribution patterns based on a total of 8514 informative markers). We have modified the orginal <A HREF="http://www.well.ox.ac.uk/mouse/INBREDS/" class="fs14">Wellcome-CTC</A> genotypes by adding selected microsatellite markers. We have also curate the data and have removed somewhat improbable double-recombinant haplotypes and by imputing genotypes for a few untyped strains using very tightly linked markers. This genotype "smoothing" may remove some genuine recombinations and may result in linkage maps that will be very slightly conservative.</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used in these studies:</P>

<Blockquote><P>
The AXB and BXA recombinant inbred strains were derived from a reciprocal cross between A/J (A) and C57BL/6J (B6 or B). Both parental strains have been sequenced, making this a particularly powerful set of RI strains for functional and genetic analyses. Data acquired using AXB and BXA subsets should be combined; the only difference being the polarity of intercross matings that generated (A x B)F1s and (B x A)F1s. AXB and BXA strains were all produced by Muriel Nesbitt at UCSD in the mid and late 1970s and first used in the early 1980s (Skamene et al., 1984; Peleg and Nesbitt, 1984; Marshal and Paigen, <a href="http://jaxmice.jax.org/library/notes/454a.htm" target="_blank" class="fs14">1993</a>). The set was imported into The Jackson Laboratory by Beverly Paigen (Pgn) in the early 1990s. As of 2004, approximately 25 viable and fully independent AXB/BXA strains are available.
</P>


<P>Several nominally independent strains in the AXB and BXA sets are very closely related. These duplicates should not be used without special statistical precaution. The most obvious option is to combine and average data from these strains except when their phenotypes differ significantly (Taylor <a href="http://jaxmice.jax.org/library/notes/465b.html" target="_blank" class="fs14">1996</a>; Williams et al., <a href="http://genomebiology.com/2001/2/11/research/0046" target="_blank" class="fs14">2001</a>).

<BR>
<BR>AXB13=AXB14: 92.74% identity in an analysis of 8429 markers.  AXB14/PgnJ (JAX001684) was renamed AXB13a/PgnJ (see JAXNotes issue number 504, Winter 2006).
<BR>AXB18=AXB19=AXB20:  97 to 99% identity  (AXB18 to AXB19 = 98.16% identity, AXB18 to AXB20 = 95.72% identity, AXB19 to AXB20 = 97.34% identity n an analysis of 8429 markers). AXB18 (JAX001686) was renamed AXB19a; AXB19 (JAX001687) was NOT renamed and is still AXB19, and AXB20 (JAX001688) was renamed AXB19b (see JAXNotes issue number 504, Winter 2006).
<BR>BXA8=BXA17: 99.79% identity in an analysis of 8429 markers. BXA17 has been discarded as a strain. The orginal BXA17 was lost between 1989 and 1990. (Updated from Williams et al. 2001; see JAXNotes issue number 504, Winter 2006).).
</P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the genotypes associated with these strains: </P>

<Blockquote><P>
Please see <a href="http://genomebiology.com/2001/2/11/research/0046"  target="_blank" class="fs14"> The Genetic Structure of Recombinant Inbred Mice.</a></P>
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the acquisition these data:</P>
<Blockquote><P>Published phenotypes were obtained through a literature search of all PubMed indexed journals. Whenever possible,  exact values of graphically represented data were obtained from the authors. In all other cases graphs were measured using a vernier caliper. Additional published and unpublished phenotypes were submitted directly by investigators. These records have Record ID numbers less than 1.
</P>


<P>The special <A HREF="./genotypes/AXBXA.geno" class="fs14"> AXB/BXA genotype data set</A> that we use in GeneNetwork may be download as a "AXBXA.geno" file and opened with any text editor or even a spreadsheet program. This file is tab-delimited and includes the approximately centimorgan and basepair (megabasepair) location of the marekers, as well as the genotypes. Genotypes for several sets of strains have been combined. To obtain the original uncombined genotypes, please link to http://www.well.ox.ac.uk/mouse/INBREDS/ .
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;How to obtain these strains: </P>
<Blockquote><P>Please see <a href="http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml"  class="fs14"> http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml </a>
</P>
<P>
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Submitting data and reporting errors:</P>
<Blockquote><P>The utility of the AXB/BXA phenotype database increases significantly as each new phenotype is incorporated. To submit new data or report errors, please contact <a href="mailto:echesler@utmem.edu"> <span style="font-weight: 400">Elissa J. Chesler</span></a> and <a href="mailto:rwilliam@nb.utmem.edu"><span style="font-weight: 400">Robert W. Williams</span></a> at University of Tennessee Health Science Center<p></P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Acknowledgments:</P>
<Blockquote><P>
The initial construction of this phenotype database was performed with the help of Ryan McNeive, Nathan Copeland, and Mary-Kathleen Sullivan at University of Tennessee Health Sciences Center with support by a Human Brain Project to RWW. The extension and curation of these RI phenotype files is managed by Elissa J. Chesler. </P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;References:</P>
<Blockquote><P>
<P>
Peleg L, Nesbitt MN (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=6715864&dopt=Abstract" class="fs14">1984</a>) Genetic control of thymus size in inbred mice. J Hered. 75:126-130.
</P>
<P>
Skamene E, James SL, Meltzer MS, Nesbitt MN (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=6584512&dopt=Abstract" class="fs14">1984</a>) Genetic control of macrophage activation for killing of extracellular targets. J Leukoc Biol 35:65-69.
</P>
<P>
Sampson SB, Higgins DC, Elliot RW, Taylor BA, Lueders KK, Koza RA, Paigen B (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=9716653&dopt=Abstract" class="fs14">1998</a>) An edited linkage map for the AXB and BXA recombinant inbred mouse strains. Mamm Genome 9:688-694.
</P>
<P>
Williams RW, Gu J, Qi S, Lu L (<a href="http://genomebiology.com/2001/2/11/research/0046" class="fs14">2001</a>) The genetic structure of recombinant inbred mice: High-resolution consensus maps for complex trait analysis. Genome Biology 2:RESEARCH0046.
</P></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:</P>
<Blockquote><P>This text file was originally written by EJC, March 2004. Updated by RWW, October 30, 2004, EJC June 6, 2005.
</P></Blockquote>



</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
