<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>BXH Genotype / WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">

<P class="title">BXH Genotypes Database (Dec 2001)

 <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote><P>
This BXH genotype data set is taken directly from Williams and colleagues (2001) without modification.
</P>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the genotypes used in these studies:</P>

<Blockquote>The BXH genotype data set consists of 472 MIT CA-repeat dinucleotide microsatellite markers that were typed at UTHSC from 1998 through 2000. The file is taken directly from Williams and colleagues (2001) without any significant modification in genotypes. This order of markers has been updated to conform with the March 2005 assembly of the mouse genome (Build 34 or UCSC mm6). The entire BXH genotypes data set may be <A HREF="http://gn1.genenetwork.org/genotypes/BXH.geno" class="fs14"> downloaded</A>.
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the marker sets:</P>

<Blockquote> <U><B>Mit</B></U><br>

<I>Mit</I> markers, described by William Dietrich and colleagues (<a href="http://www.broad.mit.edu/cgi-bin/mouse/sts_info?database=mouserelease" target="_blank" class="fs14">1992</a>), are the most widely used of the three marker sets. These markers typically consist of regions of repeated dinucleotides (so-called CA repeat microsatellites) that vary in length among strains. The CA repeat polymorphisms are flanked by unique sequence that can be used to design polymerase chain reaction (PCR) primers that will selectively amplify the intervening variable region. While many of the <i>Mit</i> markers have been typed in the BXD strain set by a number of investigators, the genotypes used here are those reported in the consensus map created by Williams and colleagues (<a href="http://www.genomebiology.com/2001/2/11/research/0046" target="_blank" class="fs14">2001</a>).

<br>
<br><i>Mit</i> marker names: D + (Chr of Marker) + Mit + (Order Found) <UL>
<LI>D indicates that the marker is a DNA segment.
<LI><i>Mit</i>  indicates that the marker was identified at the Massachusetts Institute of Technology.
<LI>Order Found indicates the order in which the markers were identified. </UL>
</Blockquote>

<Blockquote><B><U>Gnf</U></B>
<br><i>Gnf</i> markers are single nucleotide polymorphisms (SNPs) identified between B6 and D2 by genomic sequence sampling. Polymorphisms were typed by Mathew Pletcher and Tim Wiltshire using the Sequenom MassEXTEND system (Wiltshire et al., <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12612341&dopt=Abstract" target="_blank" class="fs14">2003</A>) For BXD8 as well as BXD67 and BXD68, genotypes were ofteninferred from flanking markers. Each of the genotyping reactions was set up in duplicate. Physical positions were determined for each marker and integrated with previous BXD RI mapping data based on a combination of physical and genetic positions. Unsupported double crossovers were verified by manual inspection to ensure accuracy of calls. A full list of SNPs identified in the sequence sampling can be found at <a href="http://www.gnf.org/SNP/" class = "normalsize">http://www.gnf.org/SNP</a>.
<br>
<br><i>Gnf</i> marker names: S + (Chr of Marker) + Gnf + (Mb position) <UL>
<LI>S indicates the marker is a SNP
<LI><i>Gnf</i> indicates that the marker originated at the Genomics Institute of the Novartis Research Foundation.
<LI>Mb position may include decimal values. </UL>
</Blockquote>

<Blockquote><B><U>Msw</U></B>

<br><i>Msw</i> markers are variable length tracts of nucleotide repeats designed and tested by Grant Morahan, Keith Satterley, Robert W. Williams, and Jing Gu. In contrast to the variable CA repeats of Mit markers, the <i>Msw</i> markers exploit polymorphisms in tri- tetra-, penta-, and hexa-nucleotide repeats. <i>Msw</i> markers were typed by Shuhua Qi and Jing Gu at UTHSC using previously described methods (Williams et al. <a href="http://www.genomebiology.com/2001/2/11/research/0046" target="_blank" class="fs14">2001</a>). Genotypes for BXD67 and BXD68 were often inferred from flanking markers. Physical positions were determined for each marker by BLAT analysis of the microsatellite sequence against the most recent assembly of the mouse genome (currently mm5 of May 2004) and integrated with previous BXD RI mapping data based on a combination of physical and genetic positions.
<br>

<br><i>Msw</i> marker names: D + (Chr of Marker) + <i>Msw</i> + (Mb Position)<UL>
<LI>D indicates that the marker is a DNA segment <i>Msw</i> indicates the marker source.
<LI>Mb Position is marker position to the nearest megabase.
<LI>Mb position may include decimal values and, in rare cases, a letter suffix (a or b) if alternative primers were used to amplify the same repeat.
</UL></Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Acknowledgments:</P>
<Blockquote>
Genotypes for the Mit and Msw marker sets were determined by Jing Gu.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Reference:</P>
<Blockquote><P>
Williams RW, Gu J, Qi S, Lu L (<a href="http://www.genomebiology.com/2001/2/11/research/0046" class=normal>2001</a>) The genetic structure of recombinant inbred mice: High-resolution consensus maps for complex trait analysis. Genome Biology 2:RESEARCH0046
</P></Blockquote>

</TD>
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">



<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->



</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->

<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
