<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Eye AXBXA Illumina V6.2(Oct08) RankInv</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">

<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>

<SCRIPT SRC="/javascript/dhtml.js"></SCRIPT>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff >

<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD  bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5>
<TR>

<!-- split from Here -->
<!-- Body Start from Here -->


<P class="title">&nbsp;&nbsp;&nbsp;&nbsp;Eye AXBXA Illumina V6.2(Oct08) RankInv <A HREF="/webqtl/main.py?FormID=editHtml">

<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=210">GN210</A></P>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<Blockquote>
<P>
<B>FINAL RECOMMENDED AXB/BXA EYE DATA SET</B>. The Eye AXBXA Illumina Illumina V6.2 (Oct08) data set provides estimates of mRNA expression for whole eyes of 28 strains of mice, including 26 AXB/BXA recombinant inbred strains, and two parental strains, A/J and C57BL/6J. All eye samples were obtained from normal adult control animals raised in a standard laboratory environment at the Jackson Laboratory. We used the <A HREF="http://www.illumina.com/pages.ilmn?ID=198" target="_empty" class="fs14">Illumina Sentrix MouseWG-6 v2 BeadChip</A> (despite the nomenclature, this is actually the third version of the Illumina Mouse-6 platform).


<P>Users of these mouse eye data may also find the following complementary resources extremely useful:
<OL>
<LI> <A HREF="http://neibank.nei.nih.gov/index.shtml" target="_blank" class="fs14">NEIBank</A> collection of ESTs and SAGE data
<LI><A HREF="http://www.sph.uth.tmc.edu/Retnet/" target="_blank" class="fs14">RetNet</A>: the Retinal Information Network--tables of genes and loci causing inherited retinal diseases
<LI><A HREF="http://cepko.med.harvard.edu/" target="_blank" class="fs14">Mouse Retina SAGE Library</A> from the Cepko laboratory. This site provides extensive developmental data from as early as embryonic day E12.5.
<LI><A HREF="http://microarray-pubs.stanford.edu/eyecompartments/" target="_blank" class="fs14">Data sets on differential gene expression in anatomical compartments of the human eye</A> from Pat Brown's lab. View expression signatures for different ocular tissues using the <B>geneXplorer 2.0</B>.
</OL>

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the cases used to generate this set of data:</P>

<Blockquote>

<P><P>A total of 54 pooled whole eye samples were processed using approximately 10 <A HREF="http://www.illumina.com/pages.ilmn?ID=198/" target="_empty" class="fs14">Illumina Sentrix Mouse WG-6 v2 oligomer BeadChip</A> slides. All 10 slides and a total of 54 samples passed stringent quality control and error checking. This particular data set was processed using the Illumina "Rank Invariant" protocol. Values were log2 transformed. Variance of each array was stabilized to 4 units (SD of 2 units) and recentered to a mean of 8. values range from a low of 6.3 (e.g., ILMN_1225143, no expression) to a high of about 19.7 for ILMN_2772482 (<I>Crygd</I>, extremely highly expressed). Data were entered by Arthur Centeno, Hongqiang Li, Robert W. Williams, and Lu Lu, October 1, 2008.

<P>As a measure of data quality we often count the number of probes that are associated with LOD scores of greater than 10. In this data set, 255 probes have LRS values >46 (LOD >10). The maximum LOD score achieved in this data set is 27.7 for <I>Zfp330</I> (LRS of 127.9 using ILMN_2825109).

<P>The probe ILMN_2475156 can be used to check sex assignment. With three exceptions, all 28 strains are represented by one male sample and one female sample. The three exceptions are are follows: both AXB13/14 cases are males, BXA25 is represented by a single male sample or a mixed sex sample, and BXA11 is represented by a single female sample.


<P>
<IMG src="/images/upload/AXB_BXA_Eye_Sex.gif" valign="top">

<P><SMALL><B>Legend:</B> Bar chart of the expression of <I>Xist</I> probe ILMN_2475156 in the AXB/BXA eye data set. This probe is used to check sex. Strains represented by equal numbers of male and female arrays (usually one of each) should have intermediate values and a high error term. Strains represented only by males will have very low values (for example, AXB13/14 is represented by only one male) and strains represented by only females will have very high expression (for example, BXA11 is represented by only one female).  
</SMALL></P>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the strains used to generate this set of data:</P>

<P>The <A HREF="http://gn1.genenetwork.org/mouseCross.html#AXBXA" target="_empty" class="fs14">AXB/BXA</A> genetic reference panel of recombinant inbred strains consists of just about 26 fully independent strains. All of these strains are fully inbred, many well beyond the 100th filial (F) generation of inbreeding. All of these strains were been genotyped at 13,377 SNPs in 2005 (Shifman et al., 2006).
</blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the animals and tissue used to generate this set of data:</P>

<Blockquote><P>AXB/BXA animals were obtained directly from The Jackson Laboratory. Animals were housed at UTHSC before sacrifice. Mice were killed by cervical dislocation and eyes and brains were removed and placed in RNAlater.

<P>Animals used in this study were between 51 and 90 days of age (see Table 1 below). All animals were sacrifice between 9 AM and 5 PM during the light phase.
</Blockquote>

<Blockquote>
<P><B>Sample Processing:</B> Samples were processed by Lu Lu and colleagues in the Illumina Core at UTHSC between August 2008 and September 2008. All processing steps were performed by Dr. David Li. In brief, RNA purity was evaluated using the 260/280 nm absorbance ratio, and values had to be greater than 1.8 to pass our quality control (QC). The majority of samples had values between 1.9 and 2.1. RNA integrity was assessed using the Agilent Bioanalyzer 2100. The standard Eberwine T7 polymerase method was used to catalyze the synthesis of cDNA template from polyA-tailed RNA using the Ambion/Illumina (http://www.ambion.com/catalog/CatNum.php?AMIL1791) TotalPrep RNA amplication kit (Cat#IL1791). The biotin labeled cRNA was then evaluated using both the 260/280 ratio (values of 2.0-2.3 are acceptable) using a NanoDrop ND-1000 (http://www.nanodrop.com/nd-1000-overview.html). Those samples that passed QC steps (approximately 10% failed and new RNA samples had to be acquired and processed) were immediately used on BeadChips. The slides were hybridized and washed following standard Illumina protocols.

<P><B>Replication and Sample Balance:</B> We obtained a male sample pool and female sample pool from 25 of the 28 strains. Three strains are represented by samples from a single sex.

<P><B>Experimental Design and Batch Structure:</B> This data set consists arrays processed September 2008 and all arrays in this data set were processed using a single protocol by a single operator. Processing was supervised directly by Dr. Lu Lu. All samples were scanned on a single Illumina Beadstation housed in the Hamilton Eye Institute in September 2008. Details on sample assignment to slides and batches is provide in the table below.
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data Table 1:</P>

<Blockquote>

<Blockquote>
<P>This table lists all arrays by order of strain (<B>index</B>) and includes data on tube ID, strain, age, sex, number of animals in each sample pool (<B>pool size</B>), slide ID, slide position (A through F), batch by slide number (1 or 2), and Source of animals.</P>

<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" width="100%" align="Center"><tr><td>

<table width="600" border="1" cellpadding="5" cellspacing="0">

<tr bgcolor="royalblue"><td><font color=#FFFFFF>Index</font></td><td><font color=#FFFFFF>Tube ID</font></td><td><font color=#FFFFFF>Strain</font></td><td><font color=#FFFFFF>Age</font></td><td><font color=#FFFFFF>Sex</font></td><td><font color=#FFFFFF>Pool size</font></td><td><font color=#FFFFFF>Slide Id</font></td><td><font color=#FFFFFF>Slide Position</font></td><td><font color=#FFFFFF>Batch Id</font></td><td><font color=#FFFFFF>Source</font></td></tr>
<tr bgcolor="#eeeeee"><td>1</td><td>R4893E1</td><td>A/J</td><td>59</td><td>F</td><td>0</td><td>4252491010</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>2</td><td>R4982E1</td><td>A/J</td><td>79</td><td>M</td><td>2</td><td>4252491006</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>3</td><td>R4894E1</td><td>A/J</td><td>59</td><td>M</td><td>0</td><td>4252491011</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>4</td><td>R3655E1</td><td>A/J</td><td>86</td><td>M</td><td>0</td><td>4252491031</td><td>A</td><td>2</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>5</td><td>R4897E1</td><td>AXB1</td><td>90</td><td>F</td><td>0</td><td>4252491005</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>6</td><td>R5005E1</td><td>AXB1</td><td>56</td><td>M</td><td>0</td><td>4252491008</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>7</td><td>R5001E1</td><td>AXB10</td><td>63</td><td>F</td><td>2</td><td>4252491008</td><td>B</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>8</td><td>R5002E1</td><td>AXB10</td><td>63</td><td>M</td><td>2</td><td>4252491021</td><td>B</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>9</td><td>R4891E1</td><td>AXB12</td><td>63</td><td>F</td><td>0</td><td>4252491009</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>10</td><td>R4999E1</td><td>AXB12</td><td>57</td><td>M</td><td>2</td><td>4252491005</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>11</td><td>R5000E1</td><td>AXB13</td><td>57</td><td>M</td><td>2</td><td>4252491008</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>12</td><td>R5003E1</td><td>AXB15</td><td>63</td><td>F</td><td>2</td><td>4252491005</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>13</td><td>R4963E1</td><td>AXB15</td><td>63</td><td>M</td><td>2</td><td>4252491031</td><td>B</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>14</td><td>R3661E1</td><td>AXB19</td><td>89</td><td>F</td><td>0</td><td>4252491008</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>15</td><td>R4962E1</td><td>AXB19</td><td>62</td><td>M</td><td>0</td><td>4252491005</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>16</td><td>R4975E1</td><td>AXB2</td><td>79</td><td>F</td><td>2</td><td>4252491006</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>17</td><td>R4976E1</td><td>AXB2</td><td>79</td><td>M</td><td>2</td><td>4252491021</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>18</td><td>R4973E1</td><td>AXB23</td><td>66</td><td>F</td><td>2</td><td>4252491009</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>19</td><td>R4972E1</td><td>AXB23</td><td>66</td><td>M</td><td>2</td><td>4252491006</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>20</td><td>R4959E1</td><td>AXB24</td><td>100</td><td>F</td><td>2</td><td>4252491006</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>21</td><td>R4960E1</td><td>AXB24</td><td>100</td><td>M</td><td>2</td><td>4252491019</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>22</td><td>R4994E1</td><td>AXB4</td><td>54</td><td>F</td><td>2</td><td>4252491031</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>23</td><td>R5007E1</td><td>AXB4</td><td>54</td><td>M</td><td>2</td><td>4252491006</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>24</td><td>R4995E1</td><td>AXB5</td><td>61</td><td>F</td><td>2</td><td>4252491021</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>25</td><td>R4996E1</td><td>AXB5</td><td>61</td><td>M</td><td>2</td><td>4252491009</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>26</td><td>R4997E1</td><td>AXB6</td><td>60</td><td>F</td><td>2</td><td>4252491010</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>27</td><td>R4998E1</td><td>AXB6</td><td>60</td><td>M</td><td>2</td><td>4252491021</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>28</td><td>R4958E1</td><td>AXB8</td><td>52</td><td>F</td><td>2</td><td>4252491019</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>29</td><td>R4957E1</td><td>AXB8</td><td>52</td><td>M</td><td>2</td><td>4252491010</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>30</td><td>R4991E1</td><td>BXA1</td><td>54</td><td>F</td><td>2</td><td>4252491010</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>31</td><td>R4990E1</td><td>BXA1</td><td>54</td><td>M</td><td>2</td><td>4252491009</td><td>B</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>32</td><td>R4980E1</td><td>BXA11</td><td>52</td><td>F</td><td>0</td><td>4252491020</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>33</td><td>R5006E1</td><td>BXA12</td><td>48</td><td>F</td><td>2</td><td>4252491011</td><td>C</td><td>2</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>34</td><td>R4993E1</td><td>BXA12</td><td>48</td><td>M</td><td>2</td><td>4252491031</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>35</td><td>R4968E1</td><td>BXA13</td><td>61</td><td>F</td><td>2</td><td>4252491011</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>36</td><td>R4969E1</td><td>BXA13</td><td>61</td><td>M</td><td>2</td><td>4252491008</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>37</td><td>R4966E1</td><td>BXA14</td><td>56</td><td>F</td><td>2</td><td>4252491031</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>38</td><td>R4967E1</td><td>BXA14</td><td>56</td><td>M</td><td>2</td><td>4252491011</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>39</td><td>R4970E1</td><td>BXA16</td><td>51</td><td>F</td><td>2</td><td>4252491011</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>40</td><td>R5004E1</td><td>BXA16</td><td>51</td><td>M</td><td>2</td><td>4252491031</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>41</td><td>R4981E1</td><td>BXA2</td><td>50</td><td>F</td><td>2</td><td>4252491008</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>42</td><td>R4965E1</td><td>BXA2</td><td>54</td><td>M</td><td>0</td><td>4252491020</td><td>B</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>43</td><td>R4984E1</td><td>BXA24</td><td>54</td><td>F</td><td>2</td><td>4252491021</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>44</td><td>R4974E1</td><td>BXA24</td><td>54</td><td>M</td><td>2</td><td>4252491019</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>45</td><td>R4988E1</td><td>BXA25</td><td>70</td><td>M</td><td>0</td><td>4252491020</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>46</td><td>R4964E1</td><td>BXA26</td><td>54</td><td>F</td><td>0</td><td>4252491019</td><td>B</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>47</td><td>R3636E1</td><td>BXA26</td><td>87</td><td>M</td><td>0</td><td>4252491009</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>48</td><td>R4977E1</td><td>BXA4</td><td>65</td><td>F</td><td>0</td><td>4252491020</td><td>C</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>49</td><td>R3638E1</td><td>BXA4</td><td>87</td><td>M</td><td>0</td><td>4252491019</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>50</td><td>R4978E1</td><td>BXA7</td><td>65</td><td>F</td><td>2</td><td>4252491019</td><td>F</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>51</td><td>R4979E1</td><td>BXA7</td><td>65</td><td>M</td><td>2</td><td>4252491021</td><td>A</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>52</td><td>R5008E1</td><td>BXA8</td><td>52</td><td>F</td><td>2</td><td>4252491020</td><td>E</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>53</td><td>R4983E1</td><td>BXA8</td><td>52</td><td>M</td><td>2</td><td>4252491020</td><td>D</td><td>1</td><td>JAX</td></tr>
<tr bgcolor="#eeeeee"><td>54</td><td>R5012E1</td><td>C57BL/6J</td><td>87</td><td>F</td><td>2</td><td>4252491006</td><td>B</td><td>1</td><td>UTHSC RW</td></tr>
<tr bgcolor="#eeeeee"><td>55</td><td>R5010E1</td><td>C57BL/6J</td><td>87</td><td>F</td><td>2</td><td>4252491011</td><td>B</td><td>1</td><td>UTHSC RW</td></tr>
<tr bgcolor="#eeeeee"><td>56</td><td>R5011E1</td><td>C57BL/6J</td><td>79</td><td>M</td><td>2</td><td>4252491005</td><td>B</td><td>1</td><td>UTHSC RW</td></tr>
<tr bgcolor="#eeeeee"><td>57</td><td>R5009E1</td><td>C57BL/6J</td><td>79</td><td>M</td><td>2</td><td>4252491010</td><td>B</td><td>1</td><td>UTHSC RW</td></tr>
</table>
</Blockquote>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Downloading all data:</P>
<Blockquote>
<P>All data links (right-most column above) will be made active as soon as the global analysis of these data has been accepted for publication. Please see text on <A HREF="http://gn1.genenetwork.org/dataSharing.html" target="_empty" class="fs14">Data Sharing Policies</A>, and <A HREF="http://gn1.genenetwork.org/conditionsofUse.html" target="_empty" class="fs14">Conditions and Limitations</A>, and <A HREF="http://gn1.genenetwork.org/statusandContact.html" target="_empty" class="fs14">Contacts</A>. Following publication, download a summary text file or Excel file of data. Please contact <A class="fs14" HREF="mailto:rwilliams@nb.utmem.edu">RW Williams</A> if you have any questions on the use of these open data. </P>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>
<Blockquote>
<P><B><A HREF="http://www.illumina.com/pages.ilmn?ID=198" target="_empty" class="fs14">Illumina Sentrix MouseWG-6 v2 BeadChip</A>: </B>This array consists of 45,281 unique probe sequences, each 50 nucleotides in length, that have been arrayed on glass slides using a novel bead technology.


<P>ANNOTATION:  In the summer of 2008, Xusheng Wang and Robert W. Williams reannotated all three Illumina Mouse 6 BeadChips, including the array used to process the AXB/BXA eye samples. This new annotation is now incorporated into GeneNetwork.  The annotation file can be accessed at <A HREF="http://gn1.genenetwork.org/share/annotations/">http://gn1.genenetwork.org/share/annotations/</A>, by selecting "Illumina Mouse WG-6 v2.0 (GPL6887)".

<P>Position data for the 50-mer Illumina probe sequences were aligned to the mm8 mouse genome build by Xusheng Wang as part of his master annotation of all Illumina mouse arrays. Manual annotation of this array was usually done by RW Williams. </P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote>
<P>This data set uses the standard Rank Invariant method developed by Illumina and described in their BeadStation Studio documentation.



</BLOCKQUOTE>

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote>

<P>All array data were generated with funds from Dr. Benjamin Reese to RW Williams and Lu Lu as part of NIH NEI grant EY011087 (Dispersion Patterns for Retinal Neuroblasts). Arrays were scanned in the UTHSC NEI Vision Core with support from P30 EY013080. Some informatics support, including annotation of the array, was provided by NIDA and NIAAA grants to RWW and LL (NIH U01AA13499, U24AA13513 Lu Lu, PI).
</UL>



</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this text file:</P>
<Blockquote><P>
Data uploaded by Arthur Centeno, Oct 1, 2008. This text file originally generated by RWW on Oct 10, 2008. Updated by RWW, May 11, 2009, May 26, 2009.


<P>


</P></Blockquote>




</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">
<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
<TR>
<TD colspan=3 class="fs12">
<UL>

</UL>
</TD>
</TR>
</TABLE>
<!--End of footer-->
</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->

</script>
</BODY>
</HTML>
