<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>LXS Hippocampus Illumina (Oct06) Rank Database Metadata</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>
</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		
		<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>
		<P class="title"> Hippocampus Illumina (Oct06) Rank Database<A HREF="/webqtl/main.py?FormID=editHtml">
		<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
		



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<blockquote>
INITIAL DATA SET (TEST PURPOSE ONLY):  The October 2006 INIA LXS Hippocampus data set provides estimates of mRNA expression in the adult hippocampus of 77 genetically diverse strains of mice including 75 LXS recombinant inbred strains and the two parental strains ILS/Ibg and ISS/Ibg (<A HREF="http://ibgwww.colorado.edu/" target="_empty" class="fs14">Institute of Behavioral Genetics</A>). 

<P>The hippocampus is an important and intriguing part of the forebrain that is crucial in memory formation and retrieval, and that is often affected in epilepsy, Alzheimer's disease, and schizophrenia. Unlike most other parts of the brain, the hippocampus contains a remarkable population of stems cells that continue to generate neurons and glial cells even in adult mammals (Kempermann, <A HREF="http://www.oup.com/us/catalog/general/subject/Medicine/Neurology/~~/c2Y9Y29taW5nc29vbiZzcz1wdWJkYXRlLmFzYyZzZD1hc2MmcGY9MTAmdmlldz11c2EmcHI9MTAmYm9va0NvdmVycz15ZXMmY2k9MDE5NTE3OTcxNA==" target="_empty" class="fs14">2005</A>). This genetic analysis of transcript expression in the hippocampus (dentate gyrus, CA1-CA3) is a joint effort of 14 investigators that is supported by numerous agencies described in the acknowledgments section. 

Samples were processed using a total of 240 samples and 40 
<A HREF="http://www.illumina.com/gex-mouse/" target="_empty" class="fs14">Illumina mouse-6 oligomer microarray</A>  slides. Twenty-seven mouse-6 slides and a total of 157 samples passed stringent quality control and error checking. We should note that this was our first experience using the Illumina Sentrix Mouse-6 v 1.0 platform and the initial set of 13 slides were of little use and are not included in this data set. This particular data set was processed using a simple Rank protocol developed in house at UTHSC. Values were log2 transformed and the current data range from 6 to 16.5.

<P>In this initial data set, 859 probes have LRS values greater than 50.

</blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the strains used to generate this set of data:</P>


<blockquote>
<P>The <A HREF="http://gn1.genenetwork.org/mouseCross.html#LXS" target="_empty" class="fs14">LXS</A> genetic reference panel of recombinant inbred strains consists of just over 77 strains. All of these strains have been inbred from more than 23 generation (F23). All of these strains have been genotyped at 13,377 SNPs.  


<P>These strains are available from Drs. Beth Bennett and Tom Johnson at the Institute of Behavioral Genetics, in Boulder Colorado. </P>

</Blockquote>

</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the animals and tissue used to generate this set of data:</P>

<Blockquote><P>All animals were raised at the IBG in Boulder Colorado by Bennett and colleagues in an SPF facility. No cases were MHV positive. Mice were killed by cervical dislocation. Whole brain dissections were performed at the IBG by Bennett and colleagues and shipped in RNAlater to UTHSC. Most hippocampal dissections (bilateral) were performed by Zhiping Jia. Cerebella, olfactory bulbs, and brain stem were also removed and stored at -80 deg C. Hippocampal samples are very close to complete (see Lu et al., 2001) but probably include variable amounts of fimbria. 

<P>A pool of dissected tissue from four hippocampi and two naive adults of the same strain, sex, and age was collected in one session and used to generate cRNA samples. All RNA samples were extracted at UTHSC by Zhiping Jia. 

<P>All animals used in this study were between 53 and 90 days of age (average of 72 days; see Table 1 below). 
</Blockquote>

<Blockquote>
<P><B>Sample Processing:</B> Samples were processed by Lu Lu and colleagues in the Illumina Core at UTHSC between July 25 and Oct 19, 2006. All processing steps were performed by Feng Jiao. In brief, RNA purity was evaluated using the 260/280 nm absorbance ratio, and values had to be greater than 1.83. The majority of samples were 1.9 to 2.1. RNA integrity was assessed using the Agilent Bioanalyzer 2100.  The standard Eberwine T7 polymerase method was used to catalyze the synthesis of cDNA template from polyA-tailed RNA using the Ambion/Illumina (http://www.ambion.com/catalog/CatNum.php?AMIL1791) TotalPrep RNA amplication kit (Cat#IL1791). The biotin labeled cRNA was then evaluated using both the 260/280 ratio (values of 2.0 or 2.1 are acceptable) using a NanoDrop ND-1000 (http://www.nanodrop.com/nd-1000-overview.html). Those samples that passed QC steps (1-3% failed and new RNA samples had to be acquired and processed) were immediately used on the mouse-6 slide. The slides were hybridized and washed following standard Illumina protocols.

<P><B>Replication and Sample Balance:</B> We obtained a male sample pool and female sample pool from each strain. While all strains were orginally represented by matched male and female samples, one strain LXS34 consists of a single male sample. 

<!--

Strains LXS114 apprears to represent both male and female pools since very low variance for the <I>Xist</I> probe; the same is true for LXS110, LXS100, LXS60. We should see if sex designation of these strains is incorrect.

-->

<P><B>Experimental Design and Batch Structure:</B> This data set consists arrays processed in 12 groups over a three month period (July 2006 to Oct 2006). Most groups consisted of 12 samples. All arrays in this data set were processed using a single protocol by a single operator, Feng Yiao. 




</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data Table 1:</P>

<Blockquote>

<Blockquote>
This table lists all arrays by order of processing (<B>Run</B>), Sample ID, Strain, Sex, Age, number of animals in each sample pool (<B>Pool</B>), F generation number when less than 30 (<B>GenN</B>, and the Source of animals. SampleID is the ID number of the pooled RNA sample with a H1 through H3 suffix to indicate the actual hippocampal RNA aliquot used to prepare cRNA. <B>Grp</B> is the sequential group processing number (1 - 6). 
</Blockquote>


<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="100%" align="Center"><tr><td>

<table width="100%" border="0" cellpadding="5" cellspacing="1">

<tr bgcolor="royalblue">
<td><font color=#FFFFFF>index</font></td>
<td><font color=#FFFFFF>tube ID</font></td>
<td><font color=#FFFFFF>strain</td>
<td><font color=#FFFFFF>age</font></td>
<td><font color=#FFFFFF>sex</font></td>
<td><font color=#FFFFFF>batch ID</td>
<td><font color=#FFFFFF>pool sizel</td>
<td><font color=#FFFFFF>RMA outlier</font></td>
<td><font color=#FFFFFF>scale factor</font></td>
<td><font color=#FFFFFF>back ground average</font></td>
<td><font color=#FFFFFF>present</td>
<td><font color=#FFFFFF>absent</td>
<td><font color=#FFFFFF>marginal</td>
<td><font color=#FFFFFF>AFFX-b-ActinMur (3'/5')</font></td>
<td><font color=#FFFFFF>AFFX-GapdhMur (3'/5')</font></td>
<td><font color=#FFFFFF>source</font></td>
</tr>

<tr bgcolor="#eeeeee"><td>1</td><td>R2028H2</td><td>129S1/SvImJ</td><td>66</td><td>F</td><td>5</td><td>3</td><td>0.1</td><td>4.362</td><td>64.49</td><td>0.497</td><td>0.484</td><td>0.019</td><td>2.78</td><td>1.13</td><td>JAX</td></tr><tr bgcolor="#eeeeee"><td>205</td><td>R2199H1</td><td>WSB/EiJ</td><td>58</td><td>M</td><td>5</td><td>3</td><td>0.04</td><td>3.171</td><td>54.95</td><td>0.475</td><td>0.505</td><td>0.02</td><td>1.32</td><td>0.81</td><td>JAX</td></tr></table>
			  </td>
          </tr>
        </table>
</Blockquote>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Downloading all data:</P>
<Blockquote>
<P>All data links (right-most column above) will be made active as sooon as the global analysis of these data has been accepted for publication. Please see text on <A HREF="http://gn1.genenetwork.org/dataSharing.html" target="_empty" class="fs14">Data Sharing Policies</A>, and <A HREF="http://gn1.genenetwork.org/conditionsofUse.html" target="_empty" class="fs14">Conditions and Limitations</A>, and <A HREF="http://gn1.genenetwork.org/statusandContact.html" target="_empty" class="fs14">Contacts</A>. Following publication, download a summary text file or Excel file of data. Contact Lu Lu regarding data access probelms.
</P>
</Blockquote>




<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>
<Blockquote>
<P><B>Illumina Sentrix Mouse6 Bead Array Platform: </B>The <A HREF="http://www.illuminca.com" target="_blank" class="fs14">Mouse6</A> array consists of .... 

<P>Position data for the 50-mer Illumina Mouse-6 array was downloaded from Sanger at http://www.sanger.ac.uk/Users/avc/Illumina/Mouse-6_V1.gff.gz</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote>
<P>This data set uses a simple rank order method in which mean expression of all probes are computed across all good arrays. The means are then ranked.  This ranked list of probe mean values is used as a lookup table to assign values to ranked data from the individual arrays. This produces a set of array data that have precisely the same range and distribution of values per array. This is an extreme form of normalizing.

<P>Sex of the samples was validated using sex-specific probe sets such as <I>Xist</I> (probe scl00213742.1_141-S) and <I>Ddx3y</I> (scl0013129.1_159-S).

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds to Lu Lu, Beth Bennett, Mike Miles, Melloni Cook from INIA.

<LI>Lu Lu, M.D.  <!--Tissue acquisition, RNA processing, experimental design--> 
<BR>Grant Support: NIH U01AA13499, U24AA13513 (Lu Lu, PI)
</UL>



</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this text file:</P>
<Blockquote><P>
Data uploaded by Hongqiang Li, Oct 30, 2006. This text file originally generated by LL and RWW on November 29, 2006. Updated by LL, Dec 1, 2006.


<P>		
		

</P></Blockquote>



<P></P>

		</TD>
	</TR>
	<TR>
		<TD colspan=2 align=center bgColor=#ddddff>
	 
	 
	
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->



		</TD>
	</TR>
</TABLE><!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>

<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	// Note where menu initialization block is located in HTML document.
	// Don't try to position menu locating menu initialization block in
	// some table cell or other HTML element. Always put it before </body>

	// each menu gets two parameters (see demo files)
	// 1. items structure
	// 2. geometry structure

	new menu (MENU_ITEMS, MENU_POS);
	// make sure files containing definitions for these variables are linked to the document
	// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
	// error in menu_tpl.js file or that file isn't linked properly.
	
	// also take a look at stylesheets loaded in header in order to set styles
	//-->
</script>
</BODY>
</HTML>
