<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>HTML Template/ WebQTL</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		
		<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>
		<P class="title">Hippocampus Consortium M430v2 (Oct05) PDNN <A HREF="/webqtl/main.py?FormID=editHtml">
		<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>
		

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<blockquote>
PRELIMINARY:   The October 2005 Hippocampus Consortium data set provides estimates of mRNA expression in the adult hippocampus of approximately 99 genetically diverse strains of mice including 68 BXD recombinant inbred strains, 13 CXB recombinant inbred strains, a set of 16 diverse inbred strains, and 2 reciprocal F1 hybrids. The hippocampus is an important and intriguing part of the forebrain that is crucial for memory formation, and that is often affected in epilepsy, Alzheimer's disease, and schizophrenia. Unlike most other parts of the brain, the hippocampus contains a remarkable population of stems cells that continue to generate neurons and glial cells even in adult mammals (Kempermann, <A HREF="http://www.oup.com/us/catalog/general/subject/Medicine/Neurology/~~/c2Y9Y29taW5nc29vbiZzcz1wdWJkYXRlLmFzYyZzZD1hc2MmcGY9MTAmdmlldz11c2EmcHI9MTAmYm9va0NvdmVycz15ZXMmY2k9MDE5NTE3OTcxNA==" target="_empty" class="fs14">2005</A>). This genetic analysis of transcript expression in the hippocampus (dentate gyrus, CA1-CA3, and parts of the subiculum) is a joint effort of 14 investigators that is supported by numerous agencies described in the acknowledgments section. Samples were processed using a total of 206 Affymetrix Mouse Expression 430 2.0 short oligomer microarrays (MOE430 2.0 or M430v2), of which 179 passed stringent quality control and error checking. This particular data set was processed using the position-dependent nearest neighbor method (<a href="http://odin.mdacc.tmc.edu/~zhangli/PerfectMatch/" target="_blank" class="fs14">PDNN</a>) of Zhang and colleagues. To simplify comparison among the transforms we have used, the quantile normalized PDNN values from each arrray have been adjusted to an average expression of 8 units and a standard deviation of 2 units.
</blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the strains used to generate this set of data:</P>

<Blockquote> 
<P>This analysis has used 68 of BXD strains, the complete set of 13 CXB recombinant inbred strain sets, and  a mouse diversity panel consisting of 16 inbred strains and a pair of reciprocal F1 hybrids (B6D2F1 and D2B6F1). 

<!--All BXD42 and BALB/cByJ samples did not pass QC -->

<P>The <A HREF="http://gn1.genenetwork.org/mouseCross.html#BXD" target="_empty" class="fs14">BXD</A> genetic reference population of recombinant inbred strains consists of approximately 80 strains. Approximately <A HREF="http://gn1.genenetwork.org/mouseCross.html#BXD" target="_empty" class="fs14">800</A> classical phenotypes from sets of 10 to 70 of these strains been integrated in the GeneNetwork. The BXD strains in this data set include 29 of the BXD strains made by Benjamin Taylor at the Jackson Laboratory in the 1970s and 1990s (BXD1 through BXD40). All of these strains are fully inbred, many well beyond the 100th filial (F) generation of inbreeding. We have also included 39 inbred (25 strains at F20+) and nearly inbred (14 strains between F14 and F20) BXD lines generated by Lu and Peirce. All of these strains, including those between F14 and F20, have been genotyped at 13,377 SNPs.  

<P>The <A HREF="http://gn1.genenetwork.org/mouseCross.html#CXB" target="_empty" class="fs14">CXB</A> is the first and oldest set of recombinant inbred strains. Over <A HREF="http://gn1.genenetwork.org/mouseCross.html#CXB" target="_empty" class="fs14">500</A> classical phenotypes from these strains been integrated in the GeneNetwork. It is noteworthy that the CXB strains segregate for the hippocampal lamination defect (<I>Hld</I>),characterized by Nowakowski and colleagues (<A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=6536729&dopt=Citation" target="_empty" class="fs14">1984</A>). All of the CXBs have been recently genotyped at 13,377 SNPs.

<P><B>Mouse Diversity Panel (MDP)</B>. We have profiled a MDP consisting 16 inbred strains and a pair of reciprocal F1 hybrids; B6D2F1 and D2B6F1. These strains were selected for several reasons: 
<UL>
<LI> genetic and phenotypic diversity, including use by the <A HREF="http://aretha.jax.org/pub-cgi/phenome/mpdcgi?rtn=strains%2Flist&listmode=pri&typemode=all&genaltmode=no&submit1=%A0%A0Go%A0%A0" target="_empty" class="fs14">Phenome Project</A>
<LI> their use in making genetic reference populations including recombinant inbred strains, cosomic strains, congenic and recombinant congenic strains
<LI> their use by the <A HREF="http://www.complextrait.org" target="_empty" class="fs14">Complex Trait Consortium</A> to make the Collaborative Cross (Nairobi/Wellcome, Oak Ridge/DOE, and Perth/UWA)
<LI> genome sequence data from three sources (NHGRI, Celera, and Perlegen-NIEHS)
<LI> availability from The Jackson Laboratory 
</UL>

<P>All eight parents of the Collaborative Cross (129, A, C57BL/6J, CAST, NOD, NZO, PWK, and WSB) have been included in the MDP (noted below in the list). Twelve MDP strains have been sequenced, or are currently being resequenced by Perlegen for the NIEHS. This panel will be extremely helpful in systems genetic analysis of a wide variety of traits, and will be a powerful adjunct in fine mapping modulators using what is essentially an association analysis of sequence variants.

<OL>
<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/129.shtml" target="_empty" class="fs14">129S1/SvImJ </A> 
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; background for many knockouts; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/A.shtml" target="_empty" class="fs14">A/J</A>    
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by Perlegen/NIEHS; parent of the AXB/BXA panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/AKR.shtml" target="_empty" class="fs14">AKR/J</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NIEHS; Phenome Project B list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/BALB.shtml" target="_empty" class="fs14">BALB/cByJ</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;STILL IN PROGRESS (samples did not pass quality control); Sequenced by NIEHS; maternal parent of the CXB panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/BALB.shtml" target="_empty" class="fs14">BALB/cJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Widely used strain with forebrain abnormalities (callosal defects); Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C3H.shtml" target="_empty" class="fs14">C3H/HeJ</A>      <BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; paternal parent of the BXH panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C57BL.shtml" target="_empty" class="fs14">C57BL/6J</A>      
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NHGRI; parental strain of AXB/BXA, BXD, and BXH; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C57BL.shtml" target="_empty" class="fs14">C57BL/6ByJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Paternal substrain of B6 used to generate the CXB panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/CAST.shtml" target="_empty" class="fs14">CAST/Ei</A>    
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/DBA.shtml" target="_empty" class="fs14">DBA/2J</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS and Celera; paternal parent of the BXD panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/KK.shtml" target="_empty" class="fs14">KK/HIJ</A>     
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/LG.shtml" target="_empty" class="fs14">LG/J</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Paternal parent of the LGXSM panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NOD.shtml" target="_empty" class="fs14">NOD/LtJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project B list; diabetic

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NZO.shtml" target="_empty" class="fs14">NZO/HILtJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/PWD.shtml" target="_empty" class="fs14">PWD/PhJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; parental strain for a consomic set by Forjet and colleagues

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/PWK.shtml" target="_empty" class="fs14">PWK/PhJ</A>   
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain; Phenome Project D list

<LI><A HREF="   " target="_empty" class="fs14">WSB/EiJ</A>   <BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project C list

<LI>B6D2F1 and D2B6F1  
<BR>F1 hybrids generated by crossing C57BL/6J with DBA/2J  
</OL> 

<P>We have not combined data from reciprocal F1s because they have different Y chromosome and mitochondial haplotypes. Parent-of-origin effects (imprinting, maternal environment) may also lead to interesting differences in hippocampal transcript levels. 

<P>These strains are available from The Jackson Laboratory. BXD43 through BXD100 are available from Lu Lu and colleagues at UTHSC.</P>



</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the animals and tissue used to generate this set of data:</P>

<Blockquote><P>BXD animals were obtained from UTHSC, UAB, or directly from The Jackson Laboratory (see Table 1 below). Animals were housed at UTHSC, Beth Israel Deaconess, or the Jackson Laboratory before sacrifice. Virtually all CXB animals were obtained directly at the Jackson Laboratory by Lu Lu. We thanks Muriel Davission for making it possible to collect these cases on site. Standard inbred strain stock was from The Jackson Laboratory, but most animals were housed or reared at UTHSC. Mice were killed by cervical dislocation and brains were removed and placed in RNAlater prior to dissection. Cerebella and olfactory bulbs were removed; brains were hemisected, and both hippocampi were dissected whole by Hong Tao Zhang in the Lu lab. Hippocampal samples are very close to complete (see Lu et al., 2001) but probably include variable amounts of subiculum and fimbria. 

<P>A pool of dissected tissue typically from six hippocampi and three naive adults of the same strain, sex, and age was collected in one session and used to generate cRNA samples. Two-hundred and one RNA samples were extracted at UTHSC by Zhiping Jia, four samples by Shuhua Qi (R2331H1, R2332H1, P2350H1, R2349H1), and one by Siming Shou (R0129H2). 

<P>A great majority of animals used in this study were between 45 and 90 days of age (average of 66 days, maximum range from 41 to 196 days; see Table 1 below). 
</Blockquote>

<Blockquote>
<P><B>Sample Processing:</B> Samples were processed in the INIA Bioanalytical Core at the W. Harry Feinstone Center for Genomic Research, The University of Memphis, led by Thomas R. Sutter. All processing steps were performed by Shirlean Goodwin. In brief, RNA purity was evaluated using the 260/280 nm absorbance ratio, and values had to be greater than 1.8. The majority of samples were 1.9 to 2.1. RNA integrity was assessed using the Agilent Bioanalyzer 2100. We required an RNA integrity number (RIN) of greater than 8. This RIN value is based on the intensity ratio and amplitude of 18S and 28S rRNA signals. The standard Eberwine T7 polymerase method was used to catalyze the synthesis of cDNA template from polyA-tailed RNA using Superscript II reverse transcriptase (Invitrogen Inc.). The Enzo Life Sciences, Inc., BioArray High Yield RNA Transcript Labeling Kit (T7, Part No. 42655) was used to synthesize labeled cRNA. The cRNA was evaluated using both the 260/280 ratio (values of 2.0 or 2.1 are acceptable) and the Bioanalyzer output (a dark cRNA smear on the 2100 output centered roughly between 600 and 2000 nucleotides is required). Those samples that passed both QC steps (10% usually failed and new RNA samples had to be acquired and processed) were then sheared using a fragmentation buffer included in the Affymetrix GeneChip Sample Cleanup Module (Part No. 900371). Fragmented cRNA samples were either stored at -80 deg. C until use or were immediately injected onto the array. The arrays were hybridized and washed following standard Affymetrix protocols.

<P><B>Replication and Sample Balance:</B> We obtained a male sample pool and female sample pool from each isogenic group. While all strains were orginally represented by matched male and female samples, not all data sets passed the final quality control steps. Seventy-seven of 99 strains are represented by pairs or (rarely) trios of arrays.  The first and last samples are technical replicates of a B6D2F1 hippocampal pool (aliquotes R1291H3 and R1291H4). 

<P><B>Experimental Design and Batch Structure:</B> This data set consists arrays processed in six groups over a three month period (May 2005 to August 2005). Each group consists of 32 to 34 arrays.Sex, strain, and strain type (BXD, CXB, and MDP) were interleaved among groups to ensure reasonable balance and to minimize group-by-strain statistical confounds in group normalization. The two independent samples from a single strain were always run in different groups. All arrays were processed using a single protocol by a single operator, Shirlean Goodwin. 

<P>All samples in a group were labeled on one day, except for a few cases that failed QC on their first pass. The hybridization station accommodates up to 20 samples, and for this reason each group was split into a large first set of 20 samples and a second set of 12 to 14 samples. Samples were washed in groups of four and then held in at 4 deg C until all 20 (or 12-14) arrays were ready to scan. The last four samples out of the wash stations were scanned directly. Samples were scanned in sets of four. 
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data Table 1:</P>

<Blockquote>

<Blockquote>
This table lists all arrays by order of processing (<B>Run</B>), Sample ID, Strain, Sex, Age, number of animals in each sample pool (<B>Pool</B>), F generation number when less than 30 (<B>GenN</B>, and the Source of animals. SampleID is the ID number of the pooled RNA sample with a H1 through H3 suffix to indicate the actual hippocampal RNA aliquot used to prepare cRNA. <B>Grp</B> is the sequential group processing number (1 - 6). 
</Blockquote>


<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="75%" align="Center"><tr><td>

<table width="100%" border="0" cellpadding="5" cellspacing="1">

<tr bgcolor="royalblue">
<td><font color=#FFFFFF>Sort</font></td>
<td><font color=#FFFFFF>Run</font></td>
<td><font color=#FFFFFF>SampleID</td>
<td><font color=#FFFFFF>Strain</font></td>
<td><font color=#FFFFFF>Sex</font></td>
<td><font color=#FFFFFF>Age</font></td>
<td><font color=#FFFFFF>GenN</td>
<td><font color=#FFFFFF>Source</td>
<td><font color=#FFFFFF>Pool</td>
<td><font color=#FFFFFF>Grp</font></td>
<td><font color=#FFFFFF>Notes</font></td>
<td><font color=#FFFFFF>Data_Links_to_Affy_Files</font></td>
</tr>

 <tr bgcolor="#eeeeee"><td bgcolor="#eeeeee">1</td><td bgcolor="#eeeeee">4</td><td>R1509H1</td><td>BXD01</td><td>F</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>u</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
  <tr bgcolor="#eeeeee"><td>2</td><td>74</td><td>R1507H1</td><td>BXD01</td><td>M</td><td>58</td><td>&gt;50</td><td>GDR</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
  <tr bgcolor="#eeeeee"><td>3</td><td>102</td><td>R1520H1</td><td>BXD02</td><td>F</td><td>56</td><td>&gt;50</td><td>GDR</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>4</td><td>6</td><td>R1516H1</td><td>BXD02</td><td>M</td><td>61</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>5</td><td>8</td><td>R1593H2</td><td>BXD05</td><td>F</td><td>60</td><td>&gt;50</td><td>GDR</td><td>3</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>6</td><td>80</td><td>R1692H1</td><td>BXD05</td><td>M</td><td>60</td><td>&gt;50</td><td>GDR</td><td>2</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>7</td><td>10</td><td>R1539H2</td><td>BXD06</td><td>F</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>8</td><td>127</td><td>R1538H1</td><td>BXD06</td><td>M</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>9</td><td>12</td><td>R1518H1</td><td>BXD08</td><td>F</td><td>56</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>t</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>10</td><td>189</td><td>R1548H1</td><td>BXD08</td><td>M</td><td>59</td><td>&gt;50</td><td>GDR</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>11</td><td>14</td><td>R1350H2</td><td>BXD09</td><td>F</td><td>86</td><td>&gt;50</td><td>UMem</td><td>3</td><td>1</td><td>u</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>12</td><td>117</td><td>R1351H3</td><td>BXD09</td><td>M</td><td>86</td><td>&gt;50</td><td>UMem</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>13</td><td>173</td><td>R1531H1</td><td>BXD11</td><td>F</td><td>56</td><td>&gt;50</td><td>GDR</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>14</td><td>16</td><td>R1367H1</td><td>BXD11</td><td>M</td><td>56</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>15</td><td>18</td><td>R1530H1</td><td>BXD12</td><td>F</td><td>58</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>16</td><td>119</td><td>R1567H1</td><td>BXD12</td><td>M</td><td>58</td><td>&gt;50</td><td>GDR</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>17</td><td>177</td><td>R1529H1</td><td>BXD13</td><td>F</td><td>58</td><td>&gt;50</td><td>GDR</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>18</td><td>20</td><td>R1662H1</td><td>BXD13</td><td>M</td><td>60</td><td>&gt;50</td><td>GDR</td><td>3</td><td>1</td><td>NA</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>19</td><td>22</td><td>R1280H2</td><td>BXD14</td><td>F</td><td>56</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>20</td><td>121</td><td>R1544H1</td><td>BXD14</td><td>M</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>21</td><td>179</td><td>R1524H1</td><td>BXD15</td><td>F</td><td>60</td><td>&gt;50</td><td>GDR</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>22</td><td>24</td><td>R1515H1</td><td>BXD15</td><td>M</td><td>61</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>23</td><td>26</td><td>R1661H1</td><td>BXD16</td><td>F</td><td>61</td><td>&gt;50</td><td>GDR</td><td>3</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>24</td><td>123</td><td>R1594H1</td><td>BXD16</td><td>M</td><td>61</td><td>&gt;50</td><td>GDR</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>25</td><td>181</td><td>R1568H1</td><td>BXD19</td><td>F</td><td>60</td><td>&gt;50</td><td>GDR</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>26</td><td>28</td><td>R1471H1</td><td>BXD19</td><td>M</td><td>157</td><td>&gt;50</td><td>JBo</td><td>2</td><td>1</td><td>t</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>27</td><td>30</td><td>R1573H1</td><td>BXD20</td><td>F</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>28</td><td>32</td><td>R1347H2</td><td>BXD21</td><td>F</td><td>64</td><td>&gt;50</td><td>UMem</td><td>3</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>29</td><td>125</td><td>R1349H3</td><td>BXD21</td><td>M</td><td>64</td><td>&gt;50</td><td>UMem</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>30</td><td>183</td><td>R1848H1</td><td>BXD22</td><td>F</td><td>196</td><td>&gt;50</td><td>UAB</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>31</td><td>34</td><td>R1525H1</td><td>BXD22</td><td>M</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>32</td><td>156</td><td>R1254H1</td><td>BXD23</td><td>F</td><td>66</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>33</td><td>36</td><td>R1337H2</td><td>BXD23</td><td>M</td><td>102</td><td>&gt;50</td><td>UAB</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>34</td><td>38</td><td>R1343H2</td><td>BXD24</td><td>F</td><td>71</td><td>&gt;50</td><td>UMem</td><td>2</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>35</td><td>94</td><td>R1517H1</td><td>BXD24</td><td>M</td><td>57</td><td>&gt;50</td><td>GDR</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>36</td><td>40</td><td>R1366H1</td><td>BXD27</td><td>F</td><td>60</td><td>&gt;50</td><td>GDR</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>37</td><td>158</td><td>R1849H1</td><td>BXD27</td><td>M</td><td>70</td><td>&gt;50</td><td>UAB</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>38</td><td>76</td><td>R1353H1</td><td>BXD28</td><td>F</td><td>79</td><td>&gt;50</td><td>UMem</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>39</td><td>42</td><td>R2332H1</td><td>BXD28</td><td>M</td><td>60</td><td>&gt;50</td><td>GDR</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>40</td><td>44</td><td>R1532H1</td><td>BXD29</td><td>F</td><td>57</td><td>&gt;50</td><td>GDR</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>41</td><td>160</td><td>R1356H1</td><td>BXD29</td><td>M</td><td>76</td><td>&gt;50</td><td>UMem</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>42</td><td>96</td><td>R1242H2</td><td>BXD31</td><td>F</td><td>61</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>43</td><td>46</td><td>R1240H2</td><td>BXD31</td><td>M</td><td>61</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>44</td><td>162</td><td>R1470H1</td><td>BXD32</td><td>F</td><td>76</td><td>&gt;50</td><td>UMem</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>45</td><td>48</td><td>R1508H2</td><td>BXD32</td><td>M</td><td>58</td><td>&gt;50</td><td>GDR</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>46</td><td>50</td><td>R1345H3</td><td>BXD33</td><td>F</td><td>65</td><td>&gt;50</td><td>UMem</td><td>2</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>47</td><td>97</td><td>R1581H1</td><td>BXD33</td><td>M</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>48</td><td>52</td><td>R1527H1</td><td>BXD34</td><td>F</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>49</td><td>168</td><td>R1339H1</td><td>BXD34</td><td>M</td><td>74</td><td>&gt;50</td><td>UMem</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>50</td><td>88</td><td>R1469H1</td><td>BXD36</td><td>F</td><td>83</td><td>&gt;50</td><td>UMem</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>51</td><td>54</td><td>R1363H1</td><td>BXD36</td><td>M</td><td>77</td><td>&gt;50</td><td>UMem</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>52</td><td>92</td><td>R1855H1</td><td>BXD38</td><td>F</td><td>55</td><td>&gt;50</td><td>GDR</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>53</td><td>56</td><td>R1510H1</td><td>BXD38</td><td>M</td><td>65</td><td>&gt;50</td><td>UMem</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>54</td><td>58</td><td>R1528H2</td><td>BXD39</td><td>F</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>55</td><td>99</td><td>R1514H1</td><td>BXD39</td><td>M</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>56</td><td>100</td><td>R1522H1</td><td>BXD40</td><td>F</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>57</td><td>60</td><td>R1359H1</td><td>BXD40</td><td>M</td><td>73</td><td>&gt;50</td><td>UMem</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>58</td><td>62</td><td>R1519H1</td><td>BXD42</td><td>F</td><td>58</td><td>&gt;50</td><td>GDR</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>59</td><td>101</td><td>R1512H1</td><td>BXD42</td><td>M</td><td>59</td><td>&gt;50</td><td>GDR</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>60</td><td>5</td><td>R1334H2</td><td>BXD43</td><td>F</td><td>59</td><td>22</td><td>LuLu</td><td>4</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>61</td><td>84</td><td>R1303H1</td><td>BXD43</td><td>M</td><td>63</td><td>24</td><td>LuLu</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>62</td><td>67</td><td>R1326H1</td><td>BXD44</td><td>F</td><td>65</td><td>20</td><td>LuLu</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>63</td><td>7</td><td>R1577H2</td><td>BXD44</td><td>M</td><td>56</td><td>20</td><td>LuLu</td><td>4</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>64</td><td>103</td><td>R1399H2</td><td>BXD45</td><td>F</td><td>58</td><td>20</td><td>LuLu</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>65</td><td>191</td><td>R1465H1</td><td>BXD45</td><td>M</td><td>62</td><td>20</td><td>LuLu</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>66</td><td>105</td><td>R1316H1</td><td>BXD48</td><td>F</td><td>58</td><td>21</td><td>LuLu</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>67</td><td>78</td><td>R1575H3</td><td>BXD48</td><td>M</td><td>65</td><td>22</td><td>LuLu</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>68</td><td>175</td><td>R1879H1</td><td>BXD50</td><td>F</td><td>69</td><td>18</td><td>LuLu</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>69</td><td>13</td><td>R1944H2</td><td>BXD50</td><td>M</td><td>81</td><td>18</td><td>LuLu</td><td>2</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>70</td><td>72</td><td>R2331H1</td><td>BXD51</td><td>F</td><td>66</td><td>25</td><td>LuLu</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>71</td><td>193</td><td>R1330H1</td><td>BXD51</td><td>M</td><td>65</td><td>21</td><td>LuLu</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>72</td><td>107</td><td>R2095H2</td><td>BXD55</td><td>F</td><td>61</td><td>18</td><td>LuLu</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>73</td><td>17</td><td>R1474H1</td><td>BXD55</td><td>M</td><td>57</td><td>15</td><td>LuLu</td><td>2</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>74</td><td>109</td><td>R1331H1</td><td>BXD60</td><td>F</td><td>60</td><td>21</td><td>LuLu</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>75</td><td>19</td><td>R1281H2</td><td>BXD60</td><td>M</td><td>59</td><td>22</td><td>LuLu</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>76</td><td>111</td><td>R1914H2</td><td>BXD61</td><td>F</td><td>63</td><td>20</td><td>LuLu</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>77</td><td>21</td><td>R1856H2</td><td>BXD61</td><td>M</td><td>94</td><td>19</td><td>LuLu</td><td>2</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>78</td><td>23</td><td>R1246H1</td><td>BXD62</td><td>F</td><td>54</td><td>22</td><td>LuLu</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>79</td><td>195</td><td>R1585H1</td><td>BXD62</td><td>M</td><td>64</td><td>20</td><td>LuLu</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>80</td><td>25</td><td>R1945H1</td><td>BXD63</td><td>F</td><td>107</td><td>21</td><td>LuLu</td><td>4</td><td>1</td><td>t</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>81</td><td>197</td><td>R2093H1</td><td>BXD63</td><td>M</td><td>70</td><td>21</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>82</td><td>27</td><td>R2062H2</td><td>BXD64</td><td>F</td><td>65</td><td>19</td><td>LuLu</td><td>2</td><td>1</td><td>u</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>83</td><td>95</td><td>R2061H1</td><td>BXD64</td><td>M</td><td>87</td><td>17</td><td>LuLu</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>84</td><td>29</td><td>R2054H2</td><td>BXD65</td><td>F</td><td>55</td><td>20</td><td>LuLu</td><td>2</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>85</td><td>199</td><td>R2056H1</td><td>BXD65</td><td>M</td><td>89</td><td>17</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>86</td><td>31</td><td>R1941H2</td><td>BXD66</td><td>F</td><td>78</td><td>20</td><td>LuLu</td><td>4</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>87</td><td>115</td><td>R1949H2</td><td>BXD66</td><td>M</td><td>96</td><td>21</td><td>LuLu</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>88</td><td>185</td><td>R2060H1</td><td>BXD67</td><td>F</td><td>54</td><td>20</td><td>LuLu</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>89</td><td>33</td><td>R2052H1</td><td>BXD67</td><td>M</td><td>61</td><td>20</td><td>LuLu</td><td>3</td><td>1</td><td>t</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>90</td><td>142</td><td>R2074H1</td><td>BXD68</td><td>F</td><td>60</td><td>19</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>91</td><td>35</td><td>R1928H1</td><td>BXD68</td><td>M</td><td>72</td><td>16</td><td>LuLu</td><td>2</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>92</td><td>37</td><td>R1439H3</td><td>BXD69</td><td>F</td><td>60</td><td>21</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>93</td><td>86</td><td>R1559H1</td><td>BXD69</td><td>M</td><td>64</td><td>20</td><td>LuLu</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>94</td><td>144</td><td>R2134H1</td><td>BXD70</td><td>F</td><td>64</td><td>21</td><td>LuLu</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>95</td><td>39</td><td>R2063H1</td><td>BXD70</td><td>M</td><td>55</td><td>20</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>96</td><td>113</td><td>R1277H1</td><td>BXD73</td><td>F</td><td>60</td><td>20</td><td>LuLu</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>97</td><td>41</td><td>R1443H2</td><td>BXD73</td><td>M</td><td>76</td><td>21</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>98</td><td>43</td><td>R2055H2</td><td>BXD74</td><td>M</td><td>79</td><td>18</td><td>LuLu</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>99</td><td>146</td><td>R2316H1</td><td>BXD74</td><td>M</td><td>193</td><td>18</td><td>LuLu</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>100</td><td>45</td><td>R1871H1</td><td>BXD75</td><td>F</td><td>61</td><td>21</td><td>LuLu</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>101</td><td>90</td><td>R1844H2</td><td>BXD75</td><td>M</td><td>90</td><td>20</td><td>LuLu</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>102</td><td>47</td><td>R1948H2</td><td>BXD76</td><td>F</td><td>81</td><td>16</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>103</td><td>166</td><td>R2094H1</td><td>BXD76</td><td>M</td><td>61</td><td>17</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>104</td><td>98</td><td>R2262H1</td><td>BXD77</td><td>F</td><td>62</td><td>24</td><td>LuLu</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>105</td><td>49</td><td>R1423H1</td><td>BXD77</td><td>M</td><td>62</td><td>20</td><td>LuLu</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>106</td><td>51</td><td>R1947H1</td><td>BXD79</td><td>F</td><td>108</td><td>17</td><td>LuLu</td><td>2</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>107</td><td>169</td><td>R2092H1</td><td>BXD79</td><td>M</td><td>86</td><td>15</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>108</td><td>164</td><td>R1880H1</td><td>BXD80</td><td>F</td><td>68</td><td>19</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>109</td><td>53</td><td>R1881H2</td><td>BXD80</td><td>M</td><td>68</td><td>19</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>110</td><td>55</td><td>R2075H1</td><td>BXD83</td><td>F</td><td>60</td><td>15</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>111</td><td>187</td><td>R2076H1</td><td>BXD83</td><td>M</td><td>60</td><td>15</td><td>LuLu</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>112</td><td>171</td><td>R2077H1</td><td>BXD84</td><td>F</td><td>62</td><td>17</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>113</td><td>57</td><td>R2135H3</td><td>BXD84</td><td>M</td><td>75</td><td>17</td><td>LuLu</td><td>2</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>114</td><td>59</td><td>R1473H1</td><td>BXD85</td><td>F</td><td>79</td><td>20</td><td>LuLu</td><td>4</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>115</td><td>129</td><td>R1597H1</td><td>BXD85</td><td>M</td><td>86</td><td>21</td><td>LuLu</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>116</td><td>130</td><td>R1415H1</td><td>BXD86</td><td>F</td><td>77</td><td>20</td><td>LuLu</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>117</td><td>61</td><td>R1419H1</td><td>BXD86</td><td>M</td><td>58</td><td>21</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>118</td><td>131</td><td>R1946H2</td><td>BXD87</td><td>F</td><td>101</td><td>20</td><td>LuLu</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>119</td><td>63</td><td>R1710H1</td><td>BXD87</td><td>M</td><td>96</td><td>20</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>120</td><td>64</td><td>R1872H2</td><td>BXD89</td><td>F</td><td>90</td><td>20</td><td>LuLu</td><td>2</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>121</td><td>132</td><td>R1850H2</td><td>BXD89</td><td>M</td><td>82</td><td>19</td><td>LuLu</td><td>4</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>122</td><td>65</td><td>R2058H1</td><td>BXD90</td><td>F</td><td>61</td><td>23</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>123</td><td>133</td><td>R1453H1</td><td>BXD90</td><td>M</td><td>61</td><td>20</td><td>LuLu</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>124</td><td>66</td><td>R1301H2</td><td>BXD92</td><td>F</td><td>58</td><td>21</td><td>LuLu</td><td>3</td><td>2</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>125</td><td>134</td><td>R1309H1</td><td>BXD92</td><td>M</td><td>59</td><td>21</td><td>LuLu</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>126</td><td>148</td><td>R2057H1</td><td>BXD93</td><td>F</td><td>92</td><td>19</td><td>LuLu</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>127</td><td>9</td><td>R2059H1</td><td>BXD93</td><td>M</td><td>58</td><td>19</td><td>LuLu</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>128</td><td>82</td><td>R2313H1</td><td>BXD94</td><td>F</td><td>59</td><td>14</td><td>LuLu</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>129</td><td>136</td><td>R2314H1</td><td>BXD94</td><td>M</td><td>59</td><td>14</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>130</td><td>150</td><td>R1847H1</td><td>BXD96</td><td>F</td><td>70</td><td>20</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>131</td><td>11</td><td>R1846H2</td><td>BXD96</td><td>M</td><td>63</td><td>20</td><td>LuLu</td><td>4</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>132</td><td>138</td><td>R2053H1</td><td>BXD97</td><td>F</td><td>55</td><td>21</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>133</td><td>15</td><td>R1927H2</td><td>BXD97</td><td>M</td><td>67</td><td>20</td><td>LuLu</td><td>3</td><td>1</td><td>r</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>134</td><td>154</td><td>R1942H1</td><td>BXD98</td><td>F</td><td>62</td><td>19</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>135</td><td>68</td><td>R1943H2</td><td>BXD98</td><td>M</td><td>62</td><td>19</td><td>LuLu</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>136</td><td>70</td><td>R2197H1</td><td>BXD99</td><td>F</td><td>70</td><td>14</td><td>LuLu</td><td>4</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>137</td><td>140</td><td>R2315H1</td><td>BXD99</td><td>M</td><td>84</td><td>14</td><td>LuLu</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>138</td><td>69</td><td>R2116H1</td><td>CXB1</td><td>F</td><td>55</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>139</td><td>104</td><td>R2096H1</td><td>CXB1</td><td>M</td><td>55</td><td>&gt;50</td><td>JAX</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>140</td><td>124</td><td>R2124H1</td><td>CXB10</td><td>F</td><td>53</td><td>&gt;50</td><td>JAX</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>141</td><td>87</td><td>R2108H1</td><td>CXB10</td><td>M</td><td>53</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>142</td><td>89</td><td>R2125H1</td><td>CXB11</td><td>F</td><td>58</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>143</td><td>114</td><td>R2128H1</td><td>CXB11</td><td>M</td><td>58</td><td>&gt;50</td><td>JAX</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>144</td><td>126</td><td>R2126H1</td><td>CXB12</td><td>F</td><td>47</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>145</td><td>91</td><td>R2109H1</td><td>CXB12</td><td>M</td><td>47</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>146</td><td>93</td><td>R2127H2</td><td>CXB13</td><td>F</td><td>56</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>147</td><td>128</td><td>R2110H1</td><td>CXB13</td><td>M</td><td>56</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>148</td><td>116</td><td>R2117H1</td><td>CXB2</td><td>F</td><td>62</td><td>&gt;50</td><td>JAX</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>149</td><td>71</td><td>R2098H1</td><td>CXB2</td><td>M</td><td>68</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>150</td><td>73</td><td>R2118H1</td><td>CXB3</td><td>F</td><td>47</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>151</td><td>106</td><td>R2100H1</td><td>CXB3</td><td>M</td><td>47</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>152</td><td>118</td><td>R2119H1</td><td>CXB4</td><td>F</td><td>58</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>153</td><td>75</td><td>R2101H1</td><td>CXB4</td><td>M</td><td>58</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>154</td><td>77</td><td>R0129H2</td><td>CXB5</td><td>M</td><td>70</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>155</td><td>108</td><td>R2131H1</td><td>CXB5</td><td>M</td><td>42</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>156</td><td>79</td><td>R2120H1</td><td>CXB6</td><td>F</td><td>49</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>157</td><td>120</td><td>R2102H1</td><td>CXB6</td><td>M</td><td>49</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>158</td><td>110</td><td>R2121H1</td><td>CXB7</td><td>F</td><td>63</td><td>&gt;50</td><td>JAX</td><td>2</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>159</td><td>81</td><td>R2104H2</td><td>CXB7</td><td>M</td><td>58</td><td>&gt;50</td><td>JAX</td><td>2</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>160</td><td>83</td><td>R2122H1</td><td>CXB8</td><td>F</td><td>54</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>161</td><td>122</td><td>R2105H1</td><td>CXB8</td><td>M</td><td>41</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>162</td><td>85</td><td>R2123H1</td><td>CXB9</td><td>F</td><td>54</td><td>&gt;50</td><td>JAX</td><td>3</td><td>3</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>163</td><td>112</td><td>R2106H1</td><td>CXB9</td><td>M</td><td>54</td><td>&gt;50</td><td>JAX</td><td>3</td><td>4</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>164</td><td>135</td><td>R2028H2</td><td>129S1/SvImJ</td><td>F</td><td>66</td><td>&gt;50</td><td>JAX</td><td>3</td><td>5</td>&nbsp;<td></td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>165</td><td>170</td><td>R2029H1</td><td>129S1/SvImJ</td><td>M</td><td>66</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>6</td>&nbsp;<td></td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>166</td><td>186</td><td>R2031H2</td><td>A/J</td><td>F</td><td>57</td><td>&gt;50</td><td>JAX</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>167</td><td>149</td><td>R2030H1</td><td>A/J</td><td>M</td><td>57</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>168</td><td>151</td><td>R2032H2</td><td>AKR/J</td><td>F</td><td>66</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>169</td><td>172</td><td>R2033H2</td><td>AKR/J</td><td>M</td><td>67</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>170</td><td>188</td><td>R2034H2</td><td>BALB/cByJ</td><td>F</td><td>63</td><td>&gt;50</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>171</td><td>152</td><td>R2035H2</td><td>BALB/cByJ</td><td>M</td><td>63</td><td>&gt;50</td><td>JAX</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>172</td><td>137</td><td>R2036H2</td><td>BALB/cJ</td><td>F</td><td>51</td><td>&gt;50</td><td>JAX</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>173</td><td>174</td><td>R2037H2</td><td>BALB/cJ</td><td>M</td><td>51</td><td>&gt;50</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>174</td><td>190</td><td>R2038H2</td><td>C3H/HeJ</td><td>F</td><td>63</td><td>&gt;50</td><td>JAX</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>175</td><td>153</td><td>R2039H1</td><td>C3H/HeJ</td><td>M</td><td>63</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>176</td><td>139</td><td>R2137H1</td><td>C57BL/6ByJ</td><td>F</td><td>55</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>177</td><td>176</td><td>R2136H1</td><td>C57BL/6ByJ</td><td>M</td><td>55</td><td>&gt;50</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>178</td><td>192</td><td>R2040H2</td><td>C57BL/6J</td><td>F</td><td>64</td><td>&gt;50</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>179</td><td>2</td><td>R2041H2</td><td>C57BL/6J</td><td>M</td><td>65</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>180</td><td>155</td><td>R1449H2</td><td>C57BL/6J</td><td>M</td><td>71</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>181</td><td>141</td><td>R2042H2</td><td>CAST/EI</td><td>F</td><td>64</td><td>&gt;50</td><td>LuLu</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>182</td><td>178</td><td>R2043H2</td><td>CAST/EI</td><td>M</td><td>64</td><td>&gt;50</td><td>JAX</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>183</td><td>165</td><td>R1602H2</td><td>DBA/2J</td><td>F</td><td>60</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>5</td>&nbsp;<td></td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>184</td><td>203</td><td>R2044H2</td><td>DBA/2J</td><td>F</td><td>63</td><td>&gt;50</td><td>LuLu</td><td>3</td><td>6</td>&nbsp;<td></td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>185</td><td>3</td><td>R2045H2</td><td>DBA/2J</td><td>M</td><td>65</td><td>&gt;50</td><td>LuLu</td><td>2</td><td>1</td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>186</td><td>194</td><td>R1683H1</td><td>KK/HIJ</td><td>F</td><td>72</td><td>&gt;50</td><td>JAX</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>187</td><td>157</td><td>R1687H2</td><td>KK/HIJ</td><td>M</td><td>72</td><td>&gt;50</td><td>JAX</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>188</td><td>143</td><td>R2046H1</td><td>LG/J</td><td>F</td><td>63</td><td>&gt;50</td><td>JAX</td><td>5</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>189</td><td>180</td><td>R2047H1</td><td>LG/J</td><td>M</td><td>63</td><td>&gt;50</td><td>LuLu</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>190</td><td>196</td><td>R2048H1</td><td>NOD/LtJ</td><td>F</td><td>77</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>191</td><td>159></td><td>R2049H2</td><td>NOD/LtJ</td><td>M</td><td>76</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>192</td><td>182</td><td>R2350H1</td><td>NZ0/HILtJ</td><td>M</td><td>96</td><td>&gt;50</td><td>JAX</td><td>2</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>193</td><td>145</td><td>R2200H1</td><td>NZO/H1LtJ</td><td>F</td><td>62</td><td>&gt;50</td><td>DanG</td><td>4</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>194</td><td>198</td><td>R2050H1</td><td>PWD/PhJ</td><td>F</td><td>65</td><td>&gt;50</td><td>JAX</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>195</td><td>161</td><td>R2051H2</td><td>PWD/PhJ</td><td>M</td><td>64</td><td>&gt;50</td><td>JAX</td><td>2</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>196</td><td>147</td><td>R2322H1</td><td>PWK/PHJ</td><td>F</td><td>63</td><td>&gt;50</td><td>JAX</td><td>3</td><td>5.</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>197</td><td>184</td><td>R2349H1</td><td>PWK/PHJ</td><td>M</td><td>83</td><td>&gt;50</td><td>JAX</td><td>1</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>198</td><td>200</td><td>R2198H1</td><td>WSB/EiJ</td><td>F</td><td>58</td><td>&gt;50</td><td>LuLu</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>199</td><td>163</td><td>R2199H1</td><td>WSB/EiJ</td><td>M</td><td>58</td><td>&gt;50</td><td>JAX</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>200</td><td>201</td><td>R1289H1</td><td>B6D2F1</td><td>F</td><td>64</td><td>NA</td><td>LuLu</td><td>4</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>201</td><td>1</td><td>R1291H3</td><td>B6D2F1</td><td>M</td><td>66</td><td>NA</td><td>LuLu</td><td>4</td><td></td><td>s</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>202</td><td>204</td><td>R1291H4</td><td>B6D2F1</td><td>M</td><td>66</td><td>NA</td><td>JAX</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>203</td><td>167</td><td>R1595H1</td><td>D2B6F1</td><td>F</td><td>63</td><td>NA</td><td>LuLu</td><td>3</td><td>5</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
 <tr bgcolor="#eeeeee"><td>204</td><td>202</td><td>R1286H1</td><td>D2B6F1</td><td>F</td><td>57</td><td>NA</td><td>LuLu</td><td>3</td><td>6</td><td>&nbsp;</td><td>EXP RPT TXT CEL DAT</td>
 </tr>
</table>
			  </td>
          </tr>
        </table>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Downloading all data:</P>
<Blockquote>
<P>All data links (right-most column above) will be made active as sooon as the global analysis of these data by the Consoritum has been accepted for publication. Please see text on <A HREF="http://gn1.genenetwork.org/dataSharing.html" target="_empty" class="fs14">Data Sharing Policies</A>, and <A HREF="http://gn1.genenetwork.org/conditionsofUse.html" target="_empty" class="fs14">Conditions and Limitations</A>, and <A HREF="http://gn1.genenetwork.org/statusandContact.html" target="_empty" class="fs14">Contacts</A>. Following publication, download a summary text file or Excel file of the PDNN probe set data. Contact RW Williams regarding data access probelms.
</P>
</Blockquote>




<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>
<Blockquote>
<P><B>Affymetrix Mouse Genome 430 2.0 array: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="fs14">430v2</A> array consists of 992936 useful 25-nucleotide probes that estimate the expression of approximately 39,000 transcripts and the majority of known genes and expressed sequence tags. The array sequences were selected late in 2002 using Unigene Build 107 by Affymetrix. The UTHSC group has recently reannotated all probe sets on this array, producing more accurate data on probe and probe set targets. All probes were aligned to the most recent assembly of the Mouse Genome (Build 34, mm6) using Jim Kent's BLAT program. Many of the probe sets have been manually curated by Jing Gu and Rob Williams. </P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote>
<A HREF="http://www.biomedcentral.com/1471-2105/6/65" target="_empty" class="fs14">Harshlight</A> was used to examine the image quality of the array (CEL files). Bad areas (bubbles, scratches, blemishes) of arrays were masked.

<P>First pass data quality control:  Affymetrix GCOS provides useful array quality control data including: 
<OL>
<LI>The scale factor used to normalize mean probe intensity. This averaged 3.3 for the 179 arrays that passed and 6.2 for arrays that were excluded. The scale factor is not a particular critical parameter.
<LI>The average background level. Values averaged 54.8 units for the data sets that passed and 55.8 for data sets that were excluded. This factor is not important for quality control.
<LI>The percentage of probe sets that are associated with good signal ("present" calls). This averaged 50% for the 179 data sets that passed and 42% for those that failed. Values for passing data sets extended from 43% to 55%. This is a particularly important criterion. 
<LI>The 3':5' signal ratios of actin and Gapdh. Values for passing data sets averaged 1.5 for actin and 1.0 for Gapdh. Values for excluded data sets averaged 12.9 for actin and 9.6 for Gapdh. This is a highly discriminative QC criterion, although one must keep in mind that only two transcripts are being tested. Sequence variation among strains (particularly wild derivative strains such as CAST/Ei) may affect these ratios. 
</OL>

<P>The second step in our post-processing QC involves a count of the number of probe sets in each array that are more than 2 standard deviations (z score units) from the mean across the entire 206 array data sets. This was the most important criterion used to eliminate "bad" data sets. All 206 arrays were processed togther using standard RMA and PDNN methods. The count and percentage of probe sets in each array that were beyond the 2 z theshold was computed. Using the RMA transform the average percentage of probe sets beyond the 2 z threshold for the 179 arrays that finally passed of QC procedure was 1.76% (median of 1.18%). In contrast the 2 z percentage was more than 10-fold higher (mean of 22.4% and median 20.2%) for those arrays that were excluded. This method is not very senstive to the transformation method that is used. Using the PDNN transform the average percent of probe sets exceeding  was 1.31% for good arrays and was 22.6% for those that were excluded. In our opinion, this 2 z criterion is the most useful criterion for the final decision of whether or not to include arrays, although again, allowances need to be made for wild strains that one expects to be different from the majority of conventional inbred strains. For examploe, if a data set has excellent characteristics on all of the Affymetrix GCOS metrics listed above, but generates a high 2 z percentage, then one whould include the ssample if one can verify that there are no problems in sample and data set identification.

<P>The entire procedure can be reapplied once the initial outlier data sets have been eliminated to detect any remaining outlier data sets.


<P><A HREF="http://www.datadesk.com/products/data_analysis/datadesk/" target="_empty" class="fs14">DataDesk</A> was used to examine the statistical quality of the of the probe level (CEL) data after step 5 below. DataDesk allows a rapid detection of subsets of probes that are particular sensitive to still unknown factors in array processing. Arrays can then be categorized at the probe level into "reaction classes." A reaction class is a group of arrays for which the expression of essentially all probes are colinear over the full range of log2 values. A single but large group of arrays (n = 32) processed in essentially the identical manner by a single operator can produce arrays belonging to as many as four different reaction classes. Reaction classes are NOT related to strain, age, sex, treatment, or any known biological parameter (technical replicates can belong to different reaction classes). We do not yet understand the technical origins of reaction classes. The number of probes that contribute to the definition of reaction classes is quite small (<10% of all probes). We have categorized all arrays in this data set into one of 5 reaction classes. These have then been treated as if they were separate batches. Probes in these data type "batches" have been aligned to a common mean as described below. 

<P><B>Probe (cell) level data from the CEL file: </B>These CEL values produced by <a href="http://www.affymetrix.com/support/technical/product_updates/gcos_download.affx" target="_blank" class="fs14">GCOS</a> are 75% quantiles from a set of 91 pixel values per cell.
<0L>

<LI>We added an offset of 1.0 unit to each cell signal to ensure that all values could be logged without generating negative values. We then computed the log base 2 of each cell.

<LI>We performed a quantile normalization of the log base 2 values for all arrays using the same initial steps used by the RMA transform. 

<LI>We computed the Z scores for each cell value.

<LI>We multiplied all Z scores by 2.

<LI>We added 8 to the value of all Z scores. The consequence of this simple set of transformations is to produce a set of Z scores that have a mean of 8, a variance of 4, and a standard deviation of 2. The advantage of this modified Z score is that a two-fold difference in expression level (probe brightness level) corresponds approximately to a 1 unit difference.

<LI>inally, we computed the arithmetic mean of the values for the set of microarrays for each strain. Technical replicates were averaged before computing the mean for independent biological samples. Note, that we have not (yet) corrected for variance introduced by differences in sex or any interaction terms. We have not corrected for background beyond the background correction implemented by Affymetrix in generating the CEL file. We eventually hope to add statistical controls and adjustments for some of these variables.

</UL>

<P><B>Probe set data from the CHP file: </B>The expression values were generated using PDNN. The same simple steps described above were also applied to these values. Every microarray data set therefore has a mean expression of 8 with a standard deviation of 2. A 1 unit difference represents roughly a two-fold difference in expression level. Expression levels below 5 are usually close to background noise levels. </Blockquote>

<Blockquote>

<P>Probe level QC: Log2 probe data of all arrays were inspected in DataDesk before and after quantile normalization. Inspection involved examining scatterplots of pairs of arrays for signal homogeneity (i.e., high correlation and linearity of the bivariate plots) and looking at all pairs of correlation coefficients. XY plots of probe expression and signal variance were also examined. Probe level array data sets were organized into reaction groups. Arrays with probe data that were not homogeneous when compared to other arrays were flagged.

<P>Probe set level QC: The final normalized individual array data were evaluated for outliers. This involved counting the number of times that the probe set value for a particular array was beyond two standard deviations of the mean. This outlier analysis was carried out using the PDNN, RMA and MAS5 transforms and outliers across different levels of expression.  Arrays that were associated with an average of more than 8% outlier probe sets across all transforms and at all expression levels were eliminated. In contrast, most other arrays generated fewer than 5% outliers. 
</Blockquote>

<P>Validation of strains and sex of each array data set: A subset of probes and probe sets with a Mendelian pattern of inheritance were used to construct a expression correlation matrix for all arrays and the ideal Mendelian expectation for each strain constructed from the genotypes. There should naturally be a very high correlation in the expression patterns of transcripts with Mendelian phenotypes within each strain, as well as with the genotype strain distribution pattern of markers for the strain.  

<P>Sex of the samples was validated using sex-specific probe sets such as <I>Xist</I> and <I>Dby</I>.


</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds provided by a variety of public and private source to members of the Consortium. All of us thank Muriel Davisson, Cathy Lutz, and colleagues at the Jackson Laboratory for making it possible for us to add all of the CXB strains, and one or more samples from KK/HIJ, WSB/Ei, NZO/HILtJ, LG/J, CAST/Ei, PWD/PhJ, and PWK/PhJ to this study. We thank Yan Cui at UTHSC for allowing us to use his Linux cluster to align all M430 2.0 probes and probe sets to the mouse genome. We thank Hui-Chen Hsu and John Mountz for providing us BXD tissue samples, as well as many strains of BXD stock. We thanks Douglas Matthews (UMem in Table 1) and John Boughter (JBo in Table 1) for sharing BXD stock with us. Members of the Hippocampus Consortium thank the following sources for financial support of this effort:

<UL>
<LI>David C. Airey, Ph.D.  <!--$5,000 contribution -->
<BR>Grant Support: Vanderbilt Institute for Integratie Genomics
<BR>Department of Pharmacology
<BR>david.airey at vanderbilt.edu

<LI>Lu Lu, M.D.  <!--Tissue acquisition, RNA processing, experimental design--> 
<BR>Grant Support: NIH U01AA13499, U24AA13513

<LI><A HREF="http://www.salk.edu/faculty/faculty/details.php?id=23" target="_empty" class="fs14">Fred H. Gage, Ph.D.</A>  <!--$10,000 contribution -->
<BR>Grant Support: NIH XXXX

<LI>Dan Goldowitz, Ph.D. <!--$25,000 contribution -->
<BR>Grant Support: NIAAA INIA AA013503
<BR>University of Tennessee Health Science Center
<BR>Dept. Anatomy and Neurobiology
<BR>email: dgold@nb.utmem.edu

<LI>Shirlean Goodwin, Ph.D.  <!--All array processing-->
<BR>Grant Support: NIAAA INIA U01AA013515

<LI><A HREF="http://www.bccn-berlin.de/ResearchGroups/Kempermann" target="_empty" class="fs14">Gerd Kempermann, M.D.</A> <!--$25,000 contribution -->
<BR>Grant Support: The <A HREF="http://www.volkswagen-stiftung.de/" target="_empty" class="fs14">Volkswagen Foundation</A> Grant on Permissive and Persistent Factors in Neurogenesis in the Adult Central Nervous System  
<BR>Humboldt-Universitat Berlin 
<BR>Universitatsklinikum Charite 
<BR>email: gerd.kempermann at mdc-berlin.de

<LI>Kenneth F. Manly, Ph.D.  <!--Data handling in The GeneNetwork-->
<BR>Grant Support: NIH P20MH062009 and U01CA105417

<LI>Richard S. Nowakowski, Ph.D. <!--$5,000? contribution-->
<BR>Grant Support: R01 NS049445-01

<LI>Yanhua Qu, Ph.D. <!--All data normalization and transformation -->
<BR>Grant Support: NIH U01CA105417

<LI>Glenn D. Rosen, Ph.D. <!--Tissue and dissections-->
<BR>Grant Support: NIH P20

<LI>Leonard C. Schalkwyk, Ph.D.  <!--$5,000 contribution -->
<BR>Grant Support: MRC Career Establishment Grant G0000170
<BR>Social, Genetic and Developmental Psychiatry
<BR>Institute of Psychiatry,Kings College London
<BR>PO82, De Crespigny Park London SE5 8AF
<BR>L.Schalkwyk@iop.kcl.ac.uk

<LI>Guus Smit, Ph.D. <!--$6,000 contribution -->
<BR>Dutch NeuroBsik Mouse Phenomics Consortium
<BR>Center for Neurogenomics & Cognitive Research
<BR>Vrije Universiteit Amsterdam, The Netherlands
<BR>e-mail: guus.smit at falw.vu.nl
<BR>Grant Support: BSIK 03053

<LI>Thomas Sutter, Ph.D.  <!--All array handling and ~$20,000 for array chemistry -->
<BR>Grant Support: INIA U01 AA13515 and the W. Harry Feinstone Center for Genome Research

<LI>Stephen Whatley, Ph.D.  <!--$5,000 contribution -->
<BR>Grant Support: XXXX

<LI>Robert W. Williams, Ph.D. <!--Consortium director, design, error checking, metadata, and GeneNetwork-->
<BR>Grant Support: NIH U01AA013499, P20MH062009, U01AA013499, U01AA013513

</UL>



</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this text file:</P>
<Blockquote><P>
This text file originally generated prospectively by RWW on July 30 2005. Updated by RWW July 31, 2005.
</P></Blockquote>




<P></P>

		</TD>
	</TR>
	<TR>
		<TD colspan=2 align=center bgColor=#ddddff>
	 
	 
	
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->



		</TD>
	</TR>
</TABLE><!-- /Footer -->

<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>

<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	// Note where menu initialization block is located in HTML document.
	// Don't try to position menu locating menu initialization block in
	// some table cell or other HTML element. Always put it before </body>

	// each menu gets two parameters (see demo files)
	// 1. items structure
	// 2. geometry structure

	new menu (MENU_ITEMS, MENU_POS);
	// make sure files containing definitions for these variables are linked to the document
	// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
	// error in menu_tpl.js file or that file isn't linked properly.
	
	// also take a look at stylesheets loaded in header in order to set styles
	//-->
</script>
</BODY>
</HTML>
