<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>HQF BXD Neocortex ILM6v1.1 (Feb08) RankInv</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">

<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>

<SCRIPT SRC="/javascript/dhtml.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff >

<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD  bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5>
<TR>

<!-- split from Here -->
<!-- Body Start from Here -->
<P class="title">HQF BXD Neocortex ILM6v1.1 (Feb08) RankInv <A HREF="/webqtl/main.py?FormID=editHtml">
<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=157">GN157</A></P>






<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>

<blockquote>
The February 2008 High Q Foundation Neocortex data set provides estimates of mRNA expression in the cerebral cortex of 73 lines of mice, including 52 BXD strains, 20 standard inbred strains, and B6D2F1 isogenic hybrids. All samples are from normal adult control animals raised in a standard laboratory environment. All data were generated with funds provided by the High Q Foundation using the Illumina Mouse 6.1 bead array (the second version of the Illumina Mouse-6 platform).

<P>While this February data release is still a provisional, we are not aware of any specific errors.

<P><P>A total of 129 pooled neocortex samples were processed using approximately XX <A HREF="http://www.illumina.com/gex-mouse/" target="_empty" class="fs14">Illumina Sentrix Mouse-6.1 oligomer microarray</A> BeadArray slides. XX Mouse-6.1 slides and a total of <b>128</b> samples passed stringent quality control and error checking. This data set is a companion to the High Q Foundation Striatum data set and was processed using very closely matched methods and most of the same samples. This is our third large data set generated using the Illumina platform. This particular data set was processed using the Illumina "Rank Invariant" protocol. Values were log2 transformed and the current data range from XXX (very low or no expression) to XXXX (extremely high).

<P>As a measure of data quality we often count the number of probes that are associated with LOD scores of greater than 10 (LRS > 46). In this Neocortex Illumina (Feb 08) RankInv data set, 1564 probes have LRS values >46 (LOD >10).


<!--
<P>
<IMG src="/images/upload/Ilm103520706_May07RankInv.gif" valign="top">

<DIR><P><SMALL><B>Legend:</B> UPDATE FIGURE: Bar chart of the expression of <I>Dab1</I> probe ILM103520706 in the BXD data set. This probe has a Mendelian segregation pattern and is associated with an LRS 360.3 in this RankInv data set.  </SMALL></P>
</DIR>

-->


</BLOCKQUOTE>


<BLOCKQUOTE>


<P>Users of these mouse neocortex data may also find the following complementary resources and papers useful:
<OL>
<LI> <A HREF="http://www.jneurosci.org/cgi/content/full/26/39/9956" target="_blank" class="fs14">Rossner and colleagues, 2006</A>: a paper on the transcriptome of identified subtypes of neurons in the mouse neocortex.
<LI> <A HREF="http://rosenlab.net/Movie/P14.mov" target="_blank" class="fs14">A movie of the dissection of the brain by Dr. Glenn Rosen.</a>
</OL>
</Blockquote>





<P>ABOUT THE NEOCORTEX

</blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;

About the strains used to generate this set of data:</P>

<blockquote>
<P>The <A HREF="http://gn1.genenetwork.org/mouseCross.html#BXD" target="_empty" class="fs14">BXD</A> genetic reference panel of recombinant inbred strains consists of just over 80 strains. The BXDs in this data set include 27 of the BXD strains made by Benjamin Taylor at the Jackson Laboratory in the 1970s and 1990s (BXD1 through BXD42). All of these strains are fully inbred, many well beyond the 100th filial (F) generation of inbreeding. We have also included 25 new inbred strains BXD (F21+) generated by Lu and Peirce. All of these strains were been genotyped at 13,377 SNPs in 2005 (Shifman et al., 2006).

<blockquote>
<P><B>Mouse Diversity Panel (MDP)</B>. We have profiled a MDP consisting 20 inbred strains and an F1 hybrid (B6D2F1). These strains were selected for several reasons:
<UL>
<LI> genetic and phenotypic diversity, including use by the <A HREF="http://aretha.jax.org/pub-cgi/phenome/mpdcgi?rtn=strains%2Flist&listmode=pri&typemode=all&genaltmode=no&submit1=%A0%A0Go%A0%A0" target="_empty" class="fs14">Phenome Project</A>
<LI> their use in making genetic reference populations including recombinant inbred strains, cosomic strains, congenic and recombinant congenic strains
<LI> their use by the <A HREF="http://www.complextrait.org" target="_empty" class="fs14">Complex Trait Consortium</A> to make the Collaborative Cross (Nairobi/Wellcome, Oak Ridge/DOE, and Perth/UWA)
<LI> genome sequence data from three sources (NHGRI, Celera, and Perlegen-NIEHS)
<LI> availability from The Jackson Laboratory
</UL>

<P>All eight parents of the Collaborative Cross (129, A, C57BL/6J, CAST, NOD, NZO, PWK, and WSB) have been included in the MDP (noted below in the list). Twelve MDP strains have been sequenced, or are currently being resequenced by Perlegen for the NIEHS. This panel will be extremely helpful in systems genetic analysis of a wide variety of traits, and will be a powerful adjunct in fine mapping modulators using what is essentially an association analysis of sequence variants.

<OL>
<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/129.shtml" target="_empty" class="fs14">129S1/SvImJ </A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; background for many knockouts; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/A.shtml" target="_empty" class="fs14">A/J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by Perlegen/NIEHS; parent of the AXB/BXA panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/AKR.shtml" target="_empty" class="fs14">AKR/J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NIEHS; Phenome Project B list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/BALB.shtml" target="_empty" class="fs14">BALB/cByJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NIEHS; maternal parent of the CXB panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/BALB.shtml" target="_empty" class="fs14">BALB/cJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Widely used strain with forebrain abnormalities (callosal defects); Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C3H.shtml" target="_empty" class="fs14">C3H/HeJ</A>      <BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; paternal parent of the BXH panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C57BL.shtml" target="_empty" class="fs14">C57BL/6J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by NHGRI; parental strain of AXB/BXA, BXD, and BXH; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/C57BL.shtml" target="_empty" class="fs14">C57BL/6ByJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Paternal substrain of B6 used to generate the CXB panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/CAST.shtml" target="_empty" class="fs14">CAST/EiJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/DBA.shtml" target="_empty" class="fs14">DBA/2J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS and Celera; paternal parent of the BXD panel; Phenome Project A list

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/KK.shtml" target="_empty" class="fs14">KK/HlJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/LG.shtml" target="_empty" class="fs14">LG/J</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Paternal parent of the LGXSM panel

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NOD.shtml" target="_empty" class="fs14">NOD/LtJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project B list; diabetic

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/NZO.shtml" target="_empty" class="fs14">NZO/HlLtJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/PWD.shtml" target="_empty" class="fs14">PWD/PhJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Sequenced by Perlegen/NIEHS; parental strain for a consomic set by Forjet and colleagues

<LI><A HREF="http://www.informatics.jax.org/external/festing/mouse/docs/PWK.shtml" target="_empty" class="fs14">PWK/PhJ</A>
<BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain; Phenome Project D list

<LI><A HREF="   " target="_empty" class="fs14">WSB/EiJ</A>   <BR>&nbsp;&nbsp;&nbsp;&nbsp;Collaborative Cross strain sequenced by NIEHS; Phenome Project C list

<LI>B6D2F1
<BR>This F1 hybrid was generated by crossing C57BL/6J with DBA/2J.
</OL>


<P>These strains are available from The Jackson Laboratory. BXD43 through BXD100 strains are available from Lu Lu and colleagues at UTHSC.</P>

</Blockquote>

</Blockquote>





<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the animals and tissue used to generate this set of data:</P>

<Blockquote><P>All animals were raised at the Jackson Laboratory or at UTHSC in SPF facilities. All mice were killed by cervical dislocation. Whole brain dissections were performed at either Beth Israel Deaconess Medical Center by Glenn Rosen or at UTHSC by Lu Lu and colleagues. Neocortex samples were close to complete but are likely to include variable amounts of underlying white matter. Samples may also include parts of the pyriform cortex and subiculum.

<P>A pool of dissected neocortical tissue from two to three naive adults of the same strain, sex, and age was collected in one session and used to generate RNA samples. The great majority (75%) of animals were sacrificed between 9:30 AM and 11:30 AM. All animals were sacrificed between 9 AM and 5 PM during the light phase. All RNA samples were extracted at UTHSC by Zhiping Jia (CHECK LAST STATEMENT WITH LU).

<P>All animals used in this study were between XX and XX days of age (average of XX days; see Table 1 below). All animals were sacrifice between 9 AM and 5 PM during the light phase.
</Blockquote>

<Blockquote>
<P><B>Sample Processing:</B> Samples were processed by Lu Lu and colleagues in the Illumina Core at UTHSC between December 2007 and January 2008. All processing steps were performed by Feng Jiao. In brief, RNA purity was evaluated using the 260/280 nm absorbance ratio, and values had to be greater than 1.8 to pass our quality control (QC). The majority of samples had values between 1.9 and 2.1. RNA integrity was assessed using the Agilent Bioanalyzer 2100. The standard Eberwine T7 polymerase method was used to catalyze the synthesis of cDNA template from polyA-tailed RNA using the Ambion/Illumina (http://www.ambion.com/catalog/CatNum.php?AMIL1791) TotalPrep RNA amplication kit (Cat#IL1791). The biotin labeled cRNA was then evaluated using both the 260/280 ratio (values of 2.0-2.3 are acceptable) using a NanoDrop ND-1000 (http://www.nanodrop.com/nd-1000-overview.html). Those samples that passed QC steps (1-3% failed and new RNA samples had to be acquired and processed) were immediately used on Mouse-6 v 1.1 slide. The slides were hybridized and washed following standard Illumina protocols.

<P><B>Replication and Sample Balance:</B> We obtained a male sample pool and female sample pool from as many strains as possible. However, a number of strains are represented by samples from a single sex (see figure at bottom of page).


<P><B>Experimental Design and Batch Structure:</B> This data set consists arrays processed in XX groups over a XX month period (from Month Year to Month Year). Most groups consisted of XX samples. All arrays in this data set were processed using a single protocol by a single operator, NAME HERE. Processing was supervised directly by Dr. Lu Lu. All samples were scanned on a single Illumina Beadstation housed in the Hamilton Eye Institute between Month Day and Month Day, Year. Details on sample assignment to slides and batches is provide in the table below.

<P><B> Error checking</B>
<UL>
<LI>Checked for genotypes of BXD strains on all chromosomes using a battery of test Mendelian transcripts (transcripts with a Mendelian segregation pattern in the BXDs). Peak LRS of 260.2 for <I>Prdx2</I> using Illumina probe ILM5340577. There are no errors in the strain assignment but there are possible genotyping errors, such as:
<BR>Thumpd1 (ILM7510148) in BXD34 using marker rs6271956
<BR>H2-D2 (ILM2190725) in BXD69 using marker gnf17.035.152
<BR>Fcer1g (ILM5550020) in BXD100 using marker rs3722740 (incorrectly scored as a heterozygote).

<BR>
<BR>These genotype discrepancies are either due to recombination between the marker and the probe or a genotyping errors. (RWW, Feb 27, 2008)

<LI>Total count of transcripts/probes with LOD greater than 10 is 1564 with 52 BXD strains (BXD1 through BXD43 from (n = 27) from JAX, and BXD43 through 100 (n = 25) from UTHSC).

<LI>Sex assignment checked using <I>Xist</I> probe ILM104280446.
<BR>All female samples: Strains BXD43, BXD42, BXD68, BXD77, NZW/LacJ, and NZO/HlLtJ
<BR>All male samples: Strains BXD1, PWK/PhJ, BXD66, BXD97, BXD10, BXD75, BXD44, BXD89, BXD86, BXD80, BXD69

</UL>





</Blockquote>

<P class="subtitle">Data Table 1:</P>

<Blockquote>

<Blockquote>
This table lists all arrays by order of strain (<B>index</B>) and includes data on strain, sex, slide ID and slide position (A through F).
</Blockquote>


<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" width="100%" align="Center"><tr><td>

<table width="600" border="1" cellpadding="5" cellspacing="0">

<tr bgcolor="royalblue"><td><font color=#FFFFFF>Index</font></td><td><font color=#FFFFFF>Strain</font></td><td><font color=#FFFFFF>Sex</font></td><td><font color=#FFFFFF>Slide ID</font></td><td><font color=#FFFFFF>Slide<br>Position</td></tr>
<tr bgcolor="#eeeeee"><td>1</td><td>B6D2F1</td><td>F</td><td>1848071018</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>2</td><td>B6D2F1</td><td>M</td><td>1957998076</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>3</td><td>C57BL/6J</td><td>F</td><td>1957998083</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>4</td><td>C57BL/6J</td><td>M</td><td>1833451021</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>5</td><td>DBA/2J</td><td>F</td><td>1957998083</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>6</td><td>DBA/2J</td><td>M</td><td>1833451021</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>7</td><td>BXD1</td><td>M</td><td>4051964030</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>8</td><td>BXD5</td><td>F</td><td>1736925307</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>9</td><td>BXD5</td><td>M</td><td>4051964028</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>10</td><td>BXD6</td><td>F</td><td>4051964028</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>11</td><td>BXD6</td><td>M</td><td>1736925307</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>12</td><td>BXD8</td><td>F</td><td>4060001025</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>13</td><td>BXD8</td><td>M</td><td>1957998111</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>14</td><td>BXD9</td><td>F</td><td>4060001025</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>15</td><td>BXD9</td><td>M</td><td>1736925359</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>16</td><td>BXD11</td><td>F</td><td>4051964030</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>17</td><td>BXD11</td><td>M</td><td>1848071017</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>18</td><td>BXD12</td><td>F</td><td>4051964030</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>19</td><td>BXD12</td><td>M</td><td>1848071017</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>20</td><td>BXD13</td><td>F</td><td>4051964030</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>21</td><td>BXD13</td><td>M</td><td>1848071017</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>22</td><td>BXD14</td><td>F</td><td>4051964065</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>23</td><td>BXD14</td><td>M</td><td>1848071017</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>24</td><td>BXD15</td><td>F</td><td>4051964065</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>25</td><td>BXD15</td><td>M</td><td>1848071017</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>26</td><td>BXD16</td><td>F</td><td>1848071024</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>27</td><td>BXD16</td><td>M</td><td>4051964065</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>28</td><td>BXD18</td><td>F</td><td>4051964065</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>29</td><td>BXD18</td><td>M</td><td>1848071024</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>30</td><td>BXD19</td><td>F</td><td>4051964065</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>31</td><td>BXD19</td><td>M</td><td>1848071024</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>32</td><td>BXD21</td><td>F</td><td>1848071024</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>33</td><td>BXD21</td><td>M</td><td>4051964065</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>34</td><td>BXD23</td><td>F</td><td>1848071024</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>35</td><td>BXD23</td><td>M</td><td>4051964022</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>36</td><td>BXD27</td><td>F</td><td>1848071024</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>37</td><td>BXD27</td><td>M</td><td>4051964022</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>38</td><td>BXD28</td><td>F</td><td>1848071025</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>39</td><td>BXD28</td><td>M</td><td>4051964022</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>40</td><td>BXD31</td><td>F</td><td>4051964022</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>41</td><td>BXD31</td><td>M</td><td>1848071025</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>42</td><td>BXD32</td><td>F</td><td>4051964022</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>43</td><td>BXD32</td><td>M</td><td>1848071025</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>44</td><td>BXD33</td><td>F</td><td>4051964022</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>45</td><td>BXD33</td><td>M</td><td>1848071025</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>46</td><td>BXD34</td><td>F</td><td>4051964023</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>47</td><td>BXD34</td><td>M</td><td>1848071025</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>48</td><td>BXD36</td><td>F</td><td>1848071025</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>49</td><td>BXD36</td><td>M</td><td>4051964023</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>50</td><td>BXD38</td><td>F</td><td>4051964023</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>51</td><td>BXD38</td><td>M</td><td>1957998101</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>52</td><td>BXD39</td><td>F</td><td>4051964023</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>53</td><td>BXD39</td><td>M</td><td>1957998101</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>54</td><td>BXD40</td><td>F</td><td>4051964023</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>55</td><td>BXD40</td><td>M</td><td>1957998101</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>56</td><td>BXD42</td><td>F</td><td>4060001026</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>57</td><td>BXD43</td><td>F</td><td>1957998101</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>58</td><td>BXD43</td><td>F</td><td>4051964023</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>59</td><td>BXD44</td><td>F</td><td>1957998101</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>60</td><td>BXD44</td><td>M</td><td>4051964028</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>61</td><td>BXD45</td><td>F</td><td>4051964028</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>62</td><td>BXD45</td><td>M</td><td>1957998101</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>63</td><td>BXD51</td><td>F</td><td>4051964028</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>64</td><td>BXD51</td><td>M</td><td>1736925307</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>65</td><td>BXD55</td><td>F</td><td>1736925307</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>66</td><td>BXD55</td><td>M</td><td>4051964028</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>67</td><td>BXD60</td><td>F</td><td>4060001014</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>68</td><td>BXD60</td><td>M</td><td>1736925307</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>69</td><td>BXD61</td><td>F</td><td>4060001014</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>70</td><td>BXD61</td><td>M</td><td>1736925307</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>71</td><td>BXD62</td><td>F</td><td>4060001014</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>72</td><td>BXD62</td><td>M</td><td>1957998111</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>73</td><td>BXD65</td><td>F</td><td>1957998111</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>74</td><td>BXD65</td><td>M</td><td>4060001014</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>75</td><td>BXD66</td><td>M</td><td>4060001026</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>76</td><td>BXD68</td><td>F</td><td>4060001026</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>77</td><td>BXD69</td><td>M</td><td>1957998111</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>78</td><td>BXD69</td><td>M</td><td>4060001014</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>79</td><td>BXD70</td><td>M</td><td>4060001026</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>80</td><td>BXD73</td><td>F</td><td>1957998111</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>81</td><td>BXD73</td><td>M</td><td>4060001014</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>82</td><td>BXD75</td><td>M</td><td>4060001026</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>83</td><td>BXD77</td><td>F</td><td>4060001027</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>84</td><td>BXD80</td><td>M</td><td>4060001027</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>85</td><td>BXD84</td><td>F</td><td>1957998111</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>86</td><td>BXD84</td><td>M</td><td>4060001025</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>87</td><td>BXD86</td><td>M</td><td>4060001027</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>88</td><td>BXD87</td><td>F</td><td>4060001027</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>89</td><td>BXD87</td><td>M</td><td>4060001025</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>90</td><td>BXD89</td><td>M</td><td>4060001027</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>91</td><td>BXD90</td><td>F</td><td>1736925359</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>92</td><td>BXD90</td><td>M</td><td>4060001025</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>93</td><td>BXD96</td><td>F</td><td>4060001025</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>94</td><td>BXD96</td><td>M</td><td>1736925359</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>95</td><td>BXD97</td><td>M</td><td>4060001027</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>96</td><td>BXD100</td><td>F</td><td>1848071017</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>97</td><td>BXD100</td><td>M</td><td>4051964030</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>98</td><td>129S1/SvImJ</td><td>F</td><td>1736925359</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>99</td><td>129S1/SvImJ</td><td>M</td><td>1848071018</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>100</td><td>A/J</td><td>F</td><td>1848071018</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>101</td><td>A/J</td><td>M</td><td>1736925359</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>102</td><td>AKR/J</td><td>F</td><td>1848071018</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>103</td><td>AKR/J</td><td>M</td><td>1957998076</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>104</td><td>BALB/cByJ</td><td>F</td><td>1957998076</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>105</td><td>BALB/cByJ</td><td>M</td><td>1953348019</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>106</td><td>C3H/HeJ</td><td>F</td><td>1953348019</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>107</td><td>C3H/HeJ</td><td>M</td><td>1957998076</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>108</td><td>CAST/EiJ</td><td>F</td><td>1833451021</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>109</td><td>CAST/EiJ</td><td>M</td><td>1957998083</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>110</td><td>KK/HlJ</td><td>F</td><td>1957998083</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>111</td><td>KK/HlJ</td><td>M</td><td>1848071023</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>112</td><td>BXSB/MpJ</td><td>F</td><td>1957998076</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>113</td><td>BXSB/MpJ</td><td>M</td><td>1953348019</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>114</td><td>FVB/NJ</td><td>F</td><td>1833451021</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>115</td><td>FVB/NJ</td><td>M</td><td>1957998083</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>116</td><td>MOLF/EiJ</td><td>F</td><td>1957998083</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>117</td><td>MOLF/EiJ</td><td>M</td><td>1848071001</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>118</td><td>NOD/LtJ</td><td>F</td><td>1848071001</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>119</td><td>NOD/LtJ</td><td>M</td><td>4060001004</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>120</td><td>NZB/BlNJ</td><td>F</td><td>4060001004</td><td>B</td></tr>
<tr bgcolor="#eeeeee"><td>121</td><td>NZB/BlNJ</td><td>M</td><td>1848071001</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>122</td><td>NZO/HlLtJ</td><td>F</td><td>4060001004</td><td>C</td></tr>
<tr bgcolor="#eeeeee"><td>123</td><td>NZW/LacJ</td><td>F</td><td>4060001004</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>124</td><td>PWD/PhJ</td><td>F</td><td>4060001004</td><td>E</td></tr>
<tr bgcolor="#eeeeee"><td>125</td><td>PWK/PhJ</td><td>M</td><td>4060001004</td><td>F</td></tr>
<tr bgcolor="#eeeeee"><td>126</td><td>WSB/EiJ</td><td>F</td><td>4051964030</td><td>A</td></tr>
<tr bgcolor="#eeeeee"><td>127</td><td>BTBRT<+>tf/J</td><td>F</td><td>1957998076</td><td>D</td></tr>
<tr bgcolor="#eeeeee"><td>128</td><td>BTBRT<+>tf/J</td><td>M</td><td>1953348019</td><td>B</td></tr>

</table>
</Blockquote>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Downloading all data:</P>
<Blockquote>
<P>All data is available <a href="http://gn1.genenetwork.org/share/data/ref.php?InfoPageName=HQFNeoc_0208_RankInv" target="_blank">here</a>. Please see text on <A HREF="http://gn1.genenetwork.org/share/policies/" target="_empty" class="fs14">Data Sharing Policies</A>, and <A HREF="http://gn1.genenetwork.org/share/policies/conditions.php" target="_empty" class="fs14">Conditions and Limitations</A>, and <A HREF="http://gn1.genenetwork.org/statusandContact.html" target="_empty" class="fs14">Contacts</A>. Following publication, download a summary text file or Excel file of data. Please contact <A class="fs14" HREF="mailto:rwilliams@nb.uthsc.edu">RW Williams</A> if you have any questions on the use of these open data. </P>
</Blockquote>



<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>
<Blockquote>
<P><B>Illumina Sentrix Mouse-6.1 BeadArray Platform (ILM6v1.1): </B>The <A HREF="http://www.illumina.com" target="_blank" class="fs14">Mouse6.1</A> array consists of 46,643 unique probe sequences, each 50 nucleotides in length, that have been arrayed on glass slides using a novel bead technology.

<P>Dunning M, Smith M, Thorne N, Tavare S (2006) beadarray: An R package to analyse Illumina BeadArrays. R News (the Newsletter of the P Project) 6:17-23. (see pages 17-23 of http://CRAN.R-project.org/doc/Rnews/Rnews_2006-5.pdf).

<P>ANNOTATION:  In summer of 2008, Xusheng Wang and Robert W. Williams reannotated the Illumina Mouse-6.1 array content. This new annotation is now incorporated into GeneNetwork. For 46643 probes on the Mouse 6.1 array platform (including control probes) we have identified XXXXX NCBI Entrez Gene IDs; XXXXX matched human Gene IDs; XXXXX matched rat Gene IDs; XXXXX NCBI HomoloGene IDs; and XXXXX OMIM IDs.

<P>Position data for the 50-mer Illumina Mouse-6 array were initially downloaded from Sanger at http://www.sanger.ac.uk/Users/avc/Illumina/Mouse-6_V1.gff.gz but we then updated all positions by BLAT analysis from mm6 positions to mm8 positions (Hongqiang Li).</P>
</Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About data processing:</P>

<Blockquote>
<P>This data set uses the standard Rank Invariant method developed by Illumina and described in their BeadStation Studio documentation.

<P>Sex of the samples was validated using sex-specific probe set: <I>Xist</I> probe ILM104280446.

<!--
<P>
<IMG src="/images/upload/Xist_LXS_Hipp.gif" valign="top">


<DIR><P><SMALL><B>Legend:</B> Checking that the sex of samples was labeled correctly in mouse array data sets using <I>Xist</I> expression measured by probe ILM106520068. In this bar chart the expression of <I>Xist</I> is very low in LXS114 and has a low error term. This is because both arrays are male samples rather than 1 male and 1 female sample. In contrast LXS34 has very high expression and no error bar because the sample is from a single female pool.  </SMALL></P>
</DIR>

-->


</BLOCKQUOTE>

</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data source acknowledgment:</P>
<Blockquote><P>Data were generated with funds to RW Williams, Glenn D. Rosen, Weikuan Gu, and Lu Lu from the High Q Foundation. Informatics support also provided by NIH NIAAA INIA grants to RWW and LL.

<LI>Lu Lu, M.D.  <!--Tissue acquisition, RNA processing, experimental design-->
<BR>Grant Support: NIH U01AA13499, U24AA13513 (Lu Lu, PI)
</UL>



</P></Blockquote>

<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this text file:</P>
<Blockquote><P>
Data uploaded by Arthur Centeno, Feb 22, 2008. This text file originally generated by RWW on Feb 27, 2008. Updated by A.C on March 11, 2010.


<P>


</P></Blockquote>



<P></P>

</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">

<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->

</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->

</script>
</BODY>
</HTML>
