<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<!--<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">-->
<HTML><HEAD><TITLE>UCLA BXD and BXH Cartilage v2 **</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<LINK RELl="stylesheet" href="ajax/css/ajax-tooltip.css" media="screen" type="text/css">
<script type="text/javascript" src="ajax/js/ajax-dynamic-content.js"></script>
<script type="text/javascript" src="ajax/js/ajax.js"></script>
<script type="text/javascript" src="ajax/js/ajax-tooltip.js"></script>

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->
<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
<P class="title">UCLA BXD and BXH Cartilage Illumina WG-6 v2 **  <A HREF="/webqtl/main.py?FormID=editHtml"><img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=208">GN208</A></P>
<P><font color="#FF0000"></font></P>

<P class="subtitle">Summary:</P>

<Blockquote>
<P><B>Associated Publication</B>: <A HREF="https://ncbi.nlm.nih.gov/pubmed/20954177">Jaijam Suwanwela, Aldons J. Lusis, and colleagues, 2011</A>. <A HREF="/images/upload/Suwanwela_Lusis_2011.pdf" target="_blank" class="fwn"><I>PDF</I> </A> version


20954177

<P>Description from the paper by Suwanwela and colleagues: "The recombinant inbred strains that have been proven to have significant differences in bone traits and that were available from the Jackson Laboratories (Bar Harbor, ME, USA) were used. Those strains were C3H/HeJ, C57BL/6J, DBA/2J, B6Cc3-1/KccJ, BXH2/ TyJ, BXH4/TyJ, BXH6/TyJ, BXH7/TyJ, BXH8/TyJ, BXH9/TyJ, BXH14/ TyJ, BXH19/TyJ, BXH22/KccJ, BXD1/TyJ, BXD2/TyJ, BXD6/TyJ, BXD 12/TyJ, BXD16/TyJ, BXD 19/TyJ, BXD21/TyJ, BXD 24a/TyJ, BXD27/ TyJ BXD 28/TyJ, BXD 32/TyJ, BXD 39/TyJ, BXD 40/TyJ, and BXD 42/ TyJ recombinant inbred (RI) strains. All mouse protocols were performed according to the guidelines of the American Association for Accreditation of Laboratory Animal Care (AAALAC). Cartilage from the rib cage of 1- to 2-day-old male mice was dissected to remove bone and any adherent noncartilage tissue for microarray analysis. The cartilage was digested in 0.3% bacterial collagenase (Invitrogen, Carlsbad, CA, USA) for 10 hours, and the cells were collected by centrifugation. RNA was isolated and purified using the Rneasy kit (Qiagen, Valencia, CA, USA). It then was quantified and assessed for purity using a NanoDrop spectrophotometer (Rockland, DE, USA). RNA integrity was verified with a BioAnalyzer 2100 (Agilent, Santa Clara, CA, USA). Given the small amount of RNA we could isolate from the rib cages, we had to pool the RNA from three mice from the same strain. All 27 strains were applied separately to the Illumina arrays."

<P>"llumina Mouse-6 V1 BeadChip mouse whole-genome expres- sion arrays (Illumina, Inc., San Diego, CA, USA) were used in this study. Of the 27 RNA samples from 27 strains, 3 were hybridized twice and were used as technical replicates. A total of 200 ng of DNA-free, quality-checked RNA was amplified using the Ambion Illumina RNA amplification kit with biotin UTP (Enzo, Farming- dale, NY, USA) labeling. The Ambion Illumina RNA amplification kit uses T7 oligo(dT) primers to generate single-stranded cDNA, followed by a second-strand synthesis to generate double- stranded cDNA, which then is column-purified. In vitro transcription was done to synthesize biotin-labeled cRNA using T7 RNA polymerase. The cRNA then was column-purified. The cRNA then was checked for size and yield using the Molecular Probes Quant-iT RiboGreen assay (Invitrogen). A total of 1.5 mg of cRNA was hybridized to each array using standard Illumina protocols with streptavidin-Cy3 (Amersham, Sigma, St. Louis, Mo, USA) being used for detection. Slides were scanned on an Illumina Beadstation and processed using BeadStudio (Illumina, Inc.)."

<P>"The R software (http://cran.r-project.org/), a system for statistical computation and graphics, was used to analyze the data.(11) Data were normalized using Lumi,(12) a Bioconductor (Seattle, WA, USA) package designed to analyze Illumina microarray data that includes data input, quality control, variance stabilization, normalization, and gene annotation. The function ‘‘lumi- Expresso,’’ which uses a variance-stabilizing transformation (VST) algorithm, was used to take advantage of the technical replicates available on every Illumina microarray."

<P>Normative gene expression in cartilage tissue for a set of BXD and BXH strains of mice generated using the Mouse WG-6 v2 Illumina Sentrix bead array. The data set includes 14 BXD strains (the Jackson Laboratory series generated by Dr. Ben Taylor), 10 of the BXH series, and the 3 parental strains, C57BL/6J (B), DBA/2J (D), and C3H/HeJ (H). Data were generated by JJ Suwanwela, Evan Williams, and Dr. Jake Lusis.</P>

<!-- IF NEEDED
<P>Users of these mouse cartilage data may also find the following complementary resources extremely useful:
<OL>
<LI> <A HREF="http://gn1.genenetwork.org" target="_blank" class="fs14">Link 1</A>.
<LI> <A HREF="http://gn1.genenetwork.org" target="_blank" class="fs14">Link 2</A>.
</OL>
-->
</Blockquote>


<P class="subtitle">About the cases used to generate this set of data:</P>

<Blockquote>
<P>Based on the expression of <I>Xist</I>, probe ILM104280446, these samples are mixed sex pools. Expression of <I>Xist</I> varies from 7.2 to 9.0, consistent with mixed sex samples.</P>


</Blockquote>


<P class="subtitle">About the tissue used to generate this set of data: Please see publication and above text from the key paper.</P>

<Blockquote>
<P>Cartilage. JJ, please provide data on age and sex. Based on the expression of <I>Xist</I>, probe ILM104280446, these samples are mixed sex pools. Expression of <I>Xist</I> varies from 7.2 to 9.0, consistent with mixed sex samples.</P>
<P>

<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="600">
<tr><td>
<table width="600" border="0" cellpadding="5" cellspacing="1" align="left">
<tr bgcolor="royalblue">
<td><font color=#FFFFFF>Index</font></td><td><font color=#FFFFFF>TubeID</font></td><td><font color=#FFFFFF>Group</font></td><td><font color=#FFFFFF>Strain</font></td><td><font color=#FFFFFF>Age</font></td><td><font color=#FFFFFF>Sex</font></td><td><font color=#FFFFFF>Source</font></td></tr>
<tr bgcolor="#eeeeee"><td>1</td><td>CASEIDHERE</td><td>GDP</td><td>STRAINHERE</td><td>AGE</td><td>SEX</td><td>UCLA JL</td></tr>
</table>

</td>
</tr>
</table>
</Blockquote>
</p>

<P class="subtitle">About downloading this data set:</P>
<Blockquote>
<P>This is a confidential data set for both BXD and BXH strains of mice generated at UCLA by Jaijam Suwanwela, Evan Williams, and colleagues in the laboratory of Dr. Jake Lusis. Contact <jaijam1220@gmail.com> or Dr. Lusis for additional information. </P>
</Blockquote>


<P class="subtitle">About the array platfrom:</P>
<Blockquote>
<P>Illumina Mouse WG-6 v2</P>

</Blockquote>


<P class="subtitle">About data values and data processing:</P>

<Blockquote>
<P>Illumina rank invariant transform was further normalized by Jaijam Suwanwela using standard 2z + 8 method common to many data sets in GeneNetwork. The final values entered in GeneNetwork are a modified z score of rank invariant Log2 expression data, with a mean of 8 units and a SD of 2 units for each array. One unit of expression corresponds to approximately a 2-fold difference in expression. These values were entered by Evan G. Williams, Aug 2008, UCLA. QTL Reaper results were computed by EGW. All annotation files for this array in GeneNetwork were generated by Xusheng Wang and RW Williams.</P>
<P>

<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="600">
<tr><td>
<table width="600" border="0" cellpadding="5" cellspacing="1" align="left">
<tr bgcolor="royalblue">

<td><font color=#FFFFFF>Index</font></td><td><font color=#FFFFFF>TubeID</font></td><td><font color=#FFFFFF>Strain</font></td><td><font color=#FFFFFF>Original File ID</font></td><td><font color=#FFFFFF>QC1</font></td><td><font color=#FFFFFF>QC2</font></td><td><font color=#FFFFFF>QC3</font></td><td><font color=#FFFFFF>QC4</font></td><td><font color=#FFFFFF>QC5</font></td><td><font color=#FFFFFF>QC6</font></td><td><font color=#FFFFFF>QC7</font></td><td><font color=#FFFFFF>Batch Id</font></td><td><font color=#FFFFFF>Used for batch control</font></td></tr>
<tr bgcolor="#eeeeee"><td>1</td><td>R2595E.1</td><td>129S1/SvImJ</td><td>R2595E.1.CEL</td><td>1.79</td><td>115</td><td>61.00%</td><td>37.50%</td><td>1.50%</td><td>1.46</td><td>0.77</td><td>1</td><td>Y</td></tr>
</table>

</td>
</tr>
</table>
</Blockquote>
</p>


<P class="subtitle">Data source acknowledgment:</P>
<Blockquote>

<P>Alfons J. Lusis and colleagues. Dissertation work of Jaijam Suwanwela <jaijam1220@gmail.com></P>
</Blockquote>



<P class="subtitle">Information about this text file:</P>
<Blockquote>
<P>JJ needs to provide these data. Current skeleton text by RWW</P>
</Blockquote>

<!-- GEO SERIES DATA GOES HERE -->
<P class="subtitle">GEO Series Data: This section to be used for GEO submission of the whole series of arrays</P>
<Blockquote>
<P><B>GSE Series</B>: none
<P><B>Status</B>: unreviewed, unpublished
<P><B>Title</B>: Get from JJ
<P><B>Organism(s)</B>: M. m. musculus, inbred strains, BXD and BXH type
<P><B>Experiment type</B>: Normative expression of genes in young adult mouse cartilage
<P><B>Summary</B>

<P><B>Overall design</B>
<P><B>Contributor(s)</B>: Jaijam Suwanwela and Alfons J. Lusis, UCLA

<P><B>Citation(s)</B>

<P>
<BR><B>Submission date</B>
<BR><B>Contact name</B>: Jaijam Suwanwela and Alfons J. Lusis
<BR><B>E-mails</B>: <I>jaijam1220@gmail.com</I>
<BR><B>Phone</B>
<BR><B>FAX</B>
<BR><B>URL</B>
<BR><B>Organization name</B>: UCLA
<BR><B>Department(s)</B>: Genetics
<BR><B>Laboratory(s)</B>: Lusis 
<BR><B>Street address</B>
<BR><B>City</B>: Los Angeles
<BR><B>State/province</B>: CA
<BR><B>ZIP/Postal code</B>
<BR><B>Country</B>:  USA


<P><B>Platforms</B>: Illumina Mouse WG-6 v 2.0
<P><B>Samples</B>
<!-- GEO SERIES DATA ENDS HERE -->

</P></Blockquote>




</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">
<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
<TR>
<TD colspan=3 class="fs12">
<UL>

</UL>
</TD>
</TR>
</TABLE>
<!--End of footer-->
</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->

</script>
</BODY>
</HTML>
