<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>VCU BXD PFC Sal M430 2.0 (Dec06) RMA</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='/css/menu.css'>
<SCRIPT SRC="/javascript/webqtl.js"></SCRIPT>
</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
<TBODY>
<TR>
<script language="JavaScript" src="/javascript/header.js"></script>
</TR>
<TR>
<TD bgColor=#eeeeee class="solidBorder">
<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
<!-- Body Start from Here -->

<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>


<P class="title">Virginia Commonwealth University BXD Prefrontal Cortex
<BR>Saline Control M430 2.0 (Dec06) RMA Dataset <A HREF="/webqtl/main.py?FormID=editHtml">
<img src="/images/modify.gif" alt="modify this page" border= 0 valign="middle"></A><BR><BR>Accession number: <A HREF="/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=135">GN135</A></P>
<p>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Summary:</P>
<blockquote>

This BXD data set provides estimates of prefrontal cortex mRNA expression in response to a saline injection (injection control) across 27 BXD recombinant inbred strains and their B6 and D2 progenitor strains.  All samples are from a total of 468 adult male animals obtained from Jackson Laboratory and raised in a standard laboratory environment.  An average of 8 males per strain was used to measure anxiety-like behavior in response to restraint and saline treatment in the light-dark transition model of anxiety.  Four hours after treatment, animals were rapidly sacrificed by cervical dislocation, brains were removed, cooled and microdissected as previously described (Kerns et al., J. Neurosci. 25:2255, <A HREF="http://www.jneurosci.org/cgi/content/full/25/9/2255" target="_blank" class="fs14">2005</A>).
All RNA isolation and subsequent probe generation and hybridization to microarrays were completed using a supervised randomization procedure to minimize batch effects. Affymetrix M430 type 2.0 microarrays were used for hybridization using standard procedures.  Expression analysis was conducted by estimating the relative abundance of over 45,000 transcripts in the prefrontal cortex in response to saline using the Robust Multichip Average (RMA) method.

<P>This data set has not been normalized to a mean of 8 or a standard deviation of 2. The average expression value for all probe sets per array is approximately 6.0 with a standard deviation of 1.2.


</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the animals and tissue used to generate this set of data:</P>
<Blockquote>


All animals (males only) were obtained at 8-9 weeks of age from the Jackson Laboratory (Bar Harbor, ME). Animals were treated, behaviorally tested, and dissected by Alex Putman and colleagues at VCU. Following an hour acclimation period to the behavioral room, animals were restrained for 15 minutes, immediately injected (I.P.) with 0.9% saline or 1.8 g/kg ethanol, and 5 minutes later placed in the light-dark box for a 10 minute test session.  All behavioral testing occurred between 10 AM and 1 PM during the light phase over a 12 month period beginning August 2005.  Four hours after treatment, animals were rapidly sacrificed by cervical dislocation, brains were removed, cooled and microdissected.  Prefrontal cortex tissue was isolated by microdissection using a wedge-shaped slice taken from a 4-mm-thick brain slice extending rostrally from the optic chiasm. The wedge was centered on the inter-hemispheric fissure and extending 2 mm laterally on each side and ventrally to just above the corpus callosum. This tissue and all other brain regions were dissected in less than 5 minutes per mouse and were immediately frozen in liquid nitrogen followed by storage at -80 deg C prior to RNA isolation.  A pool of dissected tissue from 3 mice of the same strain was used to generate RNA samples. All RNA samples were extracted at VCU by Alex Putman during October 2006 and the order of RNA isolation was randomized across all strains and treatment groups (since ethanol treated animals were processed concurrently).


<p>

</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Sample Processing:
<Blockquote>

<p>
All samples were processed by Paul Vorster and Alex Putman at VCU between October and November 2006.  The BioRad Experion RNA analyzer was used to assess total RNA integrity and to verify equal molar ratios of 18S and 28S ribosomal RNA. Standard Affymetrix reagents and protocols were used for generation of cDNA and biotinylated cRNA from total RNA samples. Integrity of cRNA was checked by Experion analysis prior to microarray hybridizations. All probes exceeded a maximum size of 3000 nt for the upper border of the cRNA size distribution.
<p>

</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Replication and Sample Balance:
<Blockquote>

<p>
At present, this saline prefrontal cortex mRNA expression BXD data set is represented by a total of one microarray for each BXD strain and three microarrays for each progenitor strain.
<p>



<!--ALEX, upload a revised vesion of this figure after Arthur puts in the error terms for B6 and D2-->

<P>
<DIR><IMG src="/images/upload/Kcnj9_PFCMiles.gif" valign="top">
</DIR>

<DIR><P><SMALL><B>Legend:</B>An example of a probe set with a Mendelian bimodal distribution of phenotypes  that can be used as a genetic marker to confirm the correct assignment of strain assignments to RNA samples. Apparent <I>Kcnj9</I> mRNA levels are either high like the DBA/2J parent or low like the C57BL/6J parent.
</SMALL></P>
</DIR>




<P>The correct assignment of RNA samples and strains was confirmed by checking the expression signal of probe sets known to have Mendelian segregation patterns in the BXD strains. For example, probe set 1450712_at for <I>Kcnj9</I> (Chr 1 at 174 Mb) has high expression in all strains that inherit an allele from DBA/2J (D2) and has low in expression in all strains that inherit an allele from C57BL/6J (B6). The correlation between values of this probe set and the genotype of SNP rs3707910 in the same strains is 0.994 when the D2 allele is scored as +1 and the B6 alelle is scored as -1. This indicates that the strain assignments of all samples are perfectly aligned with respect to the the expected genotypes at this marker. This probe set is associated with an LRS score of 117.2 (n = 29 strains) when using the VCU BXD PFC Sal M430 2.0 (Dec06) RMA data set.


</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Experimental Design and Batch Structure:
<Blockquote>


<p>This data set was generated concurrently with the VCU ethanol prefrontal cortex BXD RMA data and therefore consisted of 64 microarrays processed in 5 groups of 8 to 16 microarrays during the month of September 2006.  All RNA extractions, cRNA synthesis, and hybridizations were randomized across strain and treatment groups to minimize batch effects.


<!--
START HIDDEN TEXT EXAMPLE OF DATA TABLE

</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data Table 1:</P>
<Blockquote>

<Blockquote>
This table lists all arrays by order of processing (<B>Run</B>), Sample ID, Strain, Sex, Age, number of animals in each sample pool (<B>Pool</B>), F generation number when less than 30 (<B>GenN</B>, and the Source of animals. SampleID is the ID number of the pooled RNA sample with a H1 through H3 suffix to indicate the actual RNA aliquot used to prepare cRNA. <B>Grp</B> is the sequential group processing number (1 - 6).
</Blockquote>


<Blockquote>
<table border="0" cellpadding="0" cellspacing="0" bgcolor="#000000" width="100%" align="Center"><tr><td>

<table width="100%" border="0" cellpadding="5" cellspacing="1">

<tr bgcolor="royalblue">
<td><font color=#FFFFFF>index</font></td>
<td><font color=#FFFFFF>tube ID</font></td>
<td><font color=#FFFFFF>strain</td>
<td><font color=#FFFFFF>age</font></td>
<td><font color=#FFFFFF>sex</font></td>
<td><font color=#FFFFFF>batch ID</td>
<td><font color=#FFFFFF>pool sizel</td>
<td><font color=#FFFFFF>RMA outlier</font></td>
<td><font color=#FFFFFF>scale factor</font></td>
<td><font color=#FFFFFF>back ground average</font></td>
<td><font color=#FFFFFF>present</td>
<td><font color=#FFFFFF>absent</td>
<td><font color=#FFFFFF>marginal</td>
<td><font color=#FFFFFF>AFFX-b-ActinMur (3'/5')</font></td>
<td><font color=#FFFFFF>AFFX-GapdhMur (3'/5')</font></td>
<td><font color=#FFFFFF>source</font></td>
</tr>

<tr bgcolor="#eeeeee"><td>1</td><td>R2028H2</td><td>129S1/SvImJ</td><td>66</td><td>F</td><td>5</td><td>3</td><td>0.1</td><td>4.362</td><td>64.49</td><td>0.497</td><td>0.484</td><td>0.019</td><td>2.78</td><td>1.13</td><td>JAX</td></tr>

<tr bgcolor="#eeeeee"><td>205</td><td>R2199H1</td><td>WSB/EiJ</td><td>58</td><td>M</td><td>5</td><td>3</td><td>0.04</td><td>3.171</td><td>54.95</td><td>0.475</td><td>0.505</td><td>0.02</td><td>1.32</td><td>0.81</td><td>JAX</td></tr>

</table>


</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Downloading all data:</P>
<Blockquote>

<P>These data will be available following publication in 2008. Please see text on <A HREF="http://gn1.genenetwork.org/dataSharing.html" target="_empty" class="fs14">Data Sharing Policies</A>, and <A HREF="http://gn1.genenetwork.org/conditionsofUse.html" target="_empty" class="fs14">Conditions and Limitations</A>, and <A HREF="http://gn1.genenetwork.org/statusandContact.html" target="_empty" class="fs14">Contacts</A>. Following publication, download a summary text file or Excel file of the PDNN probe set data. Contact Michael Miles or RW Williams regarding data access probelms.
</P>
</Blockquote>

<P>References:</B>
<P>Many of the techniques used to generate this data set are described in a recent publication in the <A HREF="http://www.jneurosci.org/cgi/content/full/25/9/2255" class="fs14"  target="_empty">Journal of Neuroscience</A>.



END HIDDEN TEXT

-->

</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About the array platform:</P>
<Blockquote>

<P><B>Affymetrix Mouse Genome 430 2.0 array: </B>The <A HREF="http://www.affymetrix.com/support/technical/byproduct.affx?product=moe430-20" target="_blank" class="fs14">Mouse Expression 430 2.0</A> array consists of 992936 useful 25-nucleotide probes that estimate the expression of approximately 39,000 transcripts and the majority of known genes and expressed sequence tags. The array sequences were selected late in 2002 using Unigene Build 107 by Affymetrix. The UTHSC group has recently reannotated all probe sets on this array, producing more accurate data on probe and probe set targets. All probes were aligned to the most recent assembly of the Mouse Genome (Build 34, mm6) using Jim Kent's BLAT program. Many of the probe sets have been manually curated by Jing Gu and Rob Williams. </P>


</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Data Source Acknowledgments:</P>
<Blockquote>

Work supported by NIAAA grants R01 AA13678 to Michael Miles and F31 AA016052 to Alex Putman.
</p>


</Blockquote>
<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;Information about this text file:
<Blockquote>

This text file originally generated by Mike Miles and Alex Putman, August 15, 2007. Minor additions on quality control by RWW, August 17, 2007.
</P>
</Blockquote>




</TD>

<!-- Body End Here -->
</TR></TABLE>
</TD>
</TR>
<TR>
<TD align=center bgColor=#ddddff class="solidBorder">




<!--Start of footer-->
<TABLE width="90%">
<script language='JavaScript' src='/javascript/footer.js'></script>
</TABLE>
<!--End of footer-->




</TD>
</TR>
</TABLE>
<!-- /Footer -->
<!-- menu script itself. you should not modify this file -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<!-- items structure. menu hierarchy and links are stored there -->
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<!-- files with geometry and styles structures -->
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
<!--//
// Note where menu initialization block is located in HTML document.
// Don't try to position menu locating menu initialization block in
// some table cell or other HTML element. Always put it before </body>
// each menu gets two parameters (see demo files)
// 1. items structure
// 2. geometry structure
new menu (MENU_ITEMS, MENU_POS);
// make sure files containing definitions for these variables are linked to the document
// if you got some javascript error like "MENU_POS is not defined", then you've made syntax
// error in menu_tpl.js file or that file isn't linked properly.

// also take a look at stylesheets loaded in header in order to set styles
//-->
</script>
</BODY>
</HTML>
