<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Correlation Page Annotation</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='css/menu.css'>
<SCRIPT SRC="javascript/webqtl.js"></SCRIPT>
</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script> 
	</TR>
	
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>

<P class="title">Explanations of Different Types of Correlations

<A HREF="/webqtl/main.py?FormID=editHtml"><img src="images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>



<P class="title"><a name="literatureCorr">Literature Correlations</a></P>

<P>Literature correlations are calculated using the <A HREF="https://grits.eecs.utk.edu/sgo/sgo.html" target="_blank"  class="fs14"> Semantic Gene Organizer (SGO)</A>. The SGO software uses a concept-based vector space model called  latent semantic indexing (LSI) to automatically extract gene-gene relations from titles and abstracts in MEDLINE citations (Homayouni et al. <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15308538" target="_blank"  class="fs14">2005</A>). 

<P>These LSI literature correlations are all positive and range from 0 to 1. They were initially computed in 2005 and 2007 by Dr. Ramin Homayouni and colleagues and entered into GeneNetwork by Nick Furlotte. All values were updated late 2016 by Ramin Homayouni and Sujoy Roy (University of Memphis) using the complete 2016 PubMed collection for mouse genes and abstracts. Although computed using mouse data, the literature correlation feature also works well for human and rat expression data sets. The 2016 matrix of gene-to-gene literature correlations is approximately 20k x 20k/2. This matrix was converted by Lei Yan into a MySQL table called LCorrRamin3 that is 204,980,628 rows deep and with the simple tuple: GeneId1, GeneId2, value. 

<P>Vector space modeling is a classical information retrieval technique used to identify conceptually related documents, whereby the semantic structure of a document is represented as a vector in word space and the degree of similarity between documents is calculated by the angle between document vectors. LSI improves retrieval by using a singular value decomposition (or principal component analysis) to create a subspace of concepts in which text documents are represented as vectors. 

<P>Each gene is represented as a vector in word or concept space. The cosine of the angle between the query gene vector and all other gene vectors is used to rank related genes. The distribution of cosine values ranges between 1 and -1, where a value of 1 denotes the highest similarity. 

<P>An important advantage of LSI over other vector-based retrieval methods is that relations can be derived even if a direct link between genes has not been established in the literature. The fewer factors that are used for query matching, the more conceptual the relations, and vice versa. Therefore, genes may be conceptually related even if they have not been studied together directly. This utility of LSI makes it ideal for investigating the functional significance of gene associations identified in discovery oriented genomic studies. 

<P>SGO literature correlation values may be used to rapidly identify known relations between co-regulated genes and the latent relations between co-regulated genes based on current literature.

<DIR>
<P><B>Methods</B>

<P>Gene abstract documents are first compiled using titles and abstracts in MEDLINE citations cross-referenced for each mouse gene and its human and rat homologs. These gene documents were assembled and parsed into a dictionary of terms (tokens) and weighted frequencies that are required for the term-by-gene document (sparse) matrix. In effect, each gene document is viewed as a bag of words upon which operations can be performed. There are a number of different word weighting schemes that can be used in vector space modeling (Baeza-Yates and Ribeiro-Neto, 1999). The aim of any scheme is to measure similarity within a document while at the same time measuring the dissimilarity of a gene document from the other gene documents. In SGO, we use a log entropy weighting scheme to decrease the weight of high frequency words, while giving distinguishing words higher weights (Berry and Browne, 1999). 

<P>Term and document vectors for the LSI model deployed by SGO were generated by truncating the singular value decompisition (SVD) of the term-by-gene document matrix to s factors (i.e., only s columns of the orthogonal matrices U and V are used). LSI therefore produces a rank-reduced space in which to compare two gene documents at different conceptual levels. In practice, the maximum number of factors is limited by the number of documents in the collection. Fewer factors may be used for broad (more conceptual) comparisons, whereas a larger number of factors may be used for specific (more literal) comparisons. Other studies have demonstrated that for large documents collections the optimal number of factors is approximately 300 (Landauer et al., 2004).  

<P>For more information on SGO please refer to <a href="https://grits.eecs.utk.edu/sgo/sgo.html" target="_top">https://grits.eecs.utk.edu/sgo/sgo.html</a>

</DIR>

<P class="title"><a name="tissueCorr">Tissue Correlation</a></P>


<P>The <A HREF="http://gn1.genenetwork.org/glossary.html#tissueCorrelation" target="_blank"  class="fs14">tissue correlation</A> is an estimate of the similarity of expression of two genes or transcripts across different cells, tissues, or organs. Tissue correlations were generated by analyzing gene expression in multiple tissues taken from single animals (C57BL/6J, DBA/2J mice, and BN rats). Both Pearson product-moment correlations and Spearman rank order correlations have been computed for all pair of genes using data from a set of tissue samples. Both correlation types -- r and rho -- as well as their associated p value are displayed in Trait Correlation pages to the far right. While we used mouse tissues to compute the tissue correlations, we display these values even in tables generated for rat and human transcripts and gene. 

<P>This tissue correlation analysis was carried out by Drs. Xusheng Wang, Lu Lu, and Robert W. Williams at the University of Tennessee Health Science Center in collaboration with Illumina Inc. (Jan and Feb 2008) using the <A HREF="http://www.illumina.com/pages.ilmn?ID=198">MouseWG-6 v2.0 </A>array. The GN interface was created by Xiaodong Zhou. We generated data from approximately 60 samples. The correlations in GeneNetwork were computed for a subset of 25 tissues or tissue pools that have moderately independent expression patterns. We merging many CNS samples into a single pooled value. We also merged data for ileum, jejunum, and duodenum. 

<P>In many cases, the expression of a single gene is estimated by multiple probes or probe sets, multiple exons, or alternative transcripts. In the case of the Illumina array that we used, there are typically two to three probes per gene and all may be equally valid estimates of different aspects of the expression of a gene. To provide an approximate first-order summary of joint expression of genes across tissues we simply selected that probe associated with the single highest estimate of expression averaged across multiple tissues.  
[Dec 2008, RWW].


<a name="tissue_p_r">&nbsp;</a><a name="tissue_rho">&nbsp;</a><a name="tissue_p_rho">&nbsp;</a>


<P class="title"><a name="tissue_r">Tissue Correlations: Pearson's r and Spearman's rho</a> </P>


<P>Conventional Pearson product-moment correlations (r)  or Spearman rank order correlations (rho) were computed across approximately 25 different organs and tissue types. The rank order correlations will be less dependent on the distribution of expression estimates or the particular set of 25 tissue types.

<P>The Tissue P (r) is the probability associated with the Pearson product-moment correlation. The Tissue P (rho) is the corresponding probability associated with the Spearman rank order statistic. Both P values are currently computed for an n of 25 organs and tissue types. The rank order correlation will be more conservative. This p value may be appropriate if the bivariate distribution of points across the plots is normally distributed in both x and y axes.



<P class="title"><a name="genetic_r">Sample Correlation: Pearson's r</a></P>

<P><B>Pearson' s Sample Correlation, r, </B> is computed using trait values measured across a population of genetically diverse cases (individuals or strains). This is the Pearson's r value computed across cases or samples. The correlation is generated by a combination of shared genetic, environmental, and experimental factors. In other words, this is a correlation of phenotypes across a population. It is only a good estimate of a genetic correlation when developmental, environmental, technical, and error variance in the sample is low. In the case of sets of recombinant inbred strains it is possible to reduce non-genetic sources of variance by pooling samples and by resampling genetically identical individuals. 


<P><a name="genetic_p_r">p Value of Sample Correlation (Pearson's r)</a>: The p value associated with the Pearson product-moment correlation type described above. The p value takes into account differences in the sample size. Correlations and traits are usually ranked with the smallest p value (most significant) on the top.


<P class="title"><a name="genetic_rho">Sample Correlation: Spearman's Rank Order, rho</a></P

<P><B>Spearman's Sample Correlation, rho, </B> is computed using trait values measured across a population of genetically diverse cases (individuals or strains). This is the Spearman rank order correlation (called rho rather than r) that has been computed across the samples. This correlation is not unduly affected by outliers, and should also generally be used when sample size is small (less than 20). Correlation is generated by a combination of genetic, environmental, and experimental factors. It is only a good estimate of a genetic correlation when developmental, environmental, technical, and error variance in the sample is low. In the case of sets of recombinant inbred strains it is possible to reduce non-genetic sources of variance by pooling samples and by resampling genetically identical individuals.  


<P><a name="genetic_p_rho">p Value of Sample Correlation (Spearman's rho)</a>: The p value associated with the Spearman rand order correlation type described above. The p value takes into account differences in the sample size. Correlations and traits are usually ranked with the smallest p value (most significant) on the top.







		
		</TD>
		</TR></TABLE>
		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
		
	 
	 
	
		<!--Start of footer-->
		<TABLE width="90%">
			<TR>
				<TD align="left">
					<A HREF="http://www.mapmanager.org/mmQTX.html">
					<IMG src="images/mmicon.gif" alt="Map Manager" border="0">
					</A>
				</TD>
				<TD align="Center" class="fs12">
					Service initiated <I>June 15, 2001</I>. 
					Page maintained by <A class="fs12"
					HREF="mailto:hli7@utmem.edu">
					Hongqiang Li</A>, <A class="fs12"
					HREF="mailto:fzhang7@utmem.edu">
					Fan Zhang</A>, and <A class="fs12"
					HREF="mailto:rwilliam@nb.utmem.edu">
					Robert W. Williams</A>. Site built by Jintao Wang,
					Kenneth Manly, RWW, and many <A class="fs12"
					HREF="credit.html">colleagues</A>.
				</TD>
				<TD align="right">
					<A HREF="http://www.python.org/"><IMG src="images/PythonPoweredAnimSmall.gif"
					alt="Python Powered" border="0"></A>
				</TD>
			</TR>
			<TR>
				<TD colspan=3 class="fs12"> 
					<UL>
					<LI><a class="fs12" target="_blank" href="http://www.iniastress.org">NIAAA</A> Integrative Neuroscience Initiative on Alcoholism (U01AA13499, U24AA13513)
					<LI>
					A <a class="fs12" target="_blank" href="http://www.nimh.nih.gov/neuroinformatics/index.cfm">Human Brain Project</A> funded jointly by the  <a class="fs12" target="_blank" href="http://www.drugabuse.gov/about/organization/Genetics/geneexpression/index.html">NIDA</A> , <a class="fs12" target="_blank" href="http://www.nimh.nih.gov/">NIMH</A>, and <a class="fs12" target="_blank" href="http://www.niaaa.nih.gov/">NIAAA</A> (P20-DA 21131)
					<LI>NCI <a class="fs12" target="_blank" href="http://emice.nci.nih.gov/">MMHCC</A> (U01CA105417)  
					<LI>Biomedical Informatics Research Network (<a class="fs12" target="_blank" href="http://www.nbirn.net/TestBeds/Mouse/index.htm">BIRN</A>), <a class="fs12" target="_blank" href="http://www.ncrr.nih.gov/">NCRR</A> (U24 RR021760)
					</UL>
				</TD>
			</TR>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<script language="JavaScript" src="javascript/menu_new.js"></script>
<script language="JavaScript" src="javascript/menu_items.js"></script>
<script language="JavaScript" src="javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	new menu (MENU_ITEMS, MENU_POS);
	//-->
</script>
</BODY>
</HTML>
