<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Links</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='css/menu.css'>
<SCRIPT SRC="javascript/webqtl.js"></SCRIPT>

<!-- Updated by Rob Williams, April 12, 2019 -->

</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		<TD vAlign=top width="100%" height=200 bgColor=#eeeeee>


<P class="title">Links for Exploring Networks of Genes and Phenotypes (04/12/2019)<A HREF="/webqtl/main.py?FormID=editHtml"> <img src="images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>


<Blockquote class="subtitle">GeneNetwork and WebQTL have integrated links to the following resources</Blockquote>
<Blockquote>
<Blockquote>GN PARTNERS, DEVELOPMENT SITES, AND MIRRORS</Blockquote>
<UL>

<LI><a href="http://systems-genetics.org" target="_empty">PheWAS server</A> in the Laboratory of Integrative Systems Physiology, Drs. Hao Li and Johan Auwerx, EPFL in Lausanne (service initiated March 2019, initial GN mirror initiated Nov 9, 2009) 

<LI><a href="http://www.geneweaver.org" target="_empty">GeneWeaver</A> at The Jackson Laboratory (Drs. Elissa Chesler, Erich Baker and colleagues). GeneWeaver enables users to integrate gene-centered data sets across species, tissue, and experimental platform. Gene sets can be stored, shared and compared privately, among user defined groups of investigators, and across all users. Service initiated as the Ontological Discovery Environment (ODE system, April 2009) 
<a href="http://www.geneweaver.org" target="_empty">PMID: 19733230</A>

<!--

<LI><a href="http://www.genenetwork.waimr.uwa.edu.au">Australian GeneNetwork</a> mirror and development sites, Centre for Diabetes Research, University of Western Australia, in the research group of Dr. Grant Morahan  <!--was http://130.95.9.19/--> (initiated May 2007)

<LI><a href="http://webqtl.bic.nus.edu.sg/">Singapore GeneNetwork</a> mirror site at the National University of Singapore, Bioinformatics Center, Dr. Mark De Silva  (initiated Oct 2009)

<LI><a href="http://gn.genetics.ucla.edu" target="_empty">California GeneNetwork</A> mirror site in the Department of Genetics, University of California at Los Angeles, in the research group of Dr. A. Jake Lusis (initiated Aug 2008)

<LI><a href="http://gnat.versailles.inra.fr/" target="_empty">INRA IJPB, Versailles</A> mirror site in the VAST lab of Dr. Olivier Loudet,  Plant Breeding and Genetics Unit (initiated Dec 2008)

<LI><a href="http://xzhou3.memphis.edu/" target="_empty">University of Memphis</A> mirror sites in the Bioinformatics Program and research group of Drs. Ramin Homayouni and Mohammed Yazin (initiated June 2009)

-->

<LI><a href="http://www.computablegenomix.com/">Computable Genomix </a> and the <A HREF="https://grits.eecs.utk.edu/sgo/sgo.html" target="_empty">Semantic Gene Organizer</A> systems are the source of GeneNetwork literature correlation data sets. 

<BR><BR><BR>
<LI><a href="http://genenetwork.org/share/annotations/">Custom Microarray Annotation Files</a> used by GeneNetwork
</UL>
</Blockquote>

<BR>

<Blockquote class="subtitle">KEY RESOURCES CONNECTED WITH GENENETWORK</Blockquote>
<Blockquote>
<UL>
<LI><a href="http://mus.well.ox.ac.uk/mouse/HS/" target="_empty">Wellcome Trust Oxford Mouse Resource Portal</a>
<LI><a href="http://www.sanger.ac.uk/mouseportal/" target="_empty">WTSI Mouse Resource Portal</a>

<LI><a href="http://www.sanger.ac.uk/cgi-bin/modelorgs/mousegenomes/lookseq/index.pl?show=1:10000000-10000000,paired_pileup&win=100&lane=129S1.bam&width=700" target="_empty">Sanger Mouse Sequencing Portal</a> (David Adams and colleagues)

<LI><a href="http://www.grissom.gr/stranger/StRAnGER.php" target="_empty">StRAnGER</a> Gene Ontology and KEGG pathway analyzer

<LI><a href="http://phenogen.ucdenver.edu/PhenoGen/index.jsp" target="_empty">PhenoGen Informatics</a>
<LI><a href="http://www.brain-map.org/welcome.do" target="_empty">The Allen Mouse Brain Atlas</a>
<LI><a href="http://amigo.geneontology.org/cgi-bin/amigo/go.cgi" target="_empty">AmiGO</a> 
<LI><a href="http://syndb.cbi.pku.edu.cn/" target="_empty">Synapse DB</a> 
<LI><a href="https://www.pantherdb.org" target="_empty">ABI Panther</a> 
<LI><a href="http://biogps.gnf.org" target="_empty">BioGPS</A> <font>and its predecessor </font> <a href="http://symatlas.gnf.org/SymAtlas/" target="_empty">SymAtlas</A>
<LI><a href="http://mouse.perlegen.com/mouse/download.html
" target="_empty">Perlegen/NIEHS Mouse Sequence Data</A> 
<LI><a href="http://www.ontologicaldiscovery.org/" target="_empty">Ontological Discovery Environment</a>
<LI><a href="https://shad.eecs.utk.edu/sgo/sgo.html" target="_empty">Semantic Gene Organizer</a>
<LI><a href="http://genome.ucsc.edu/" target="_empty">UCSC Genome Browser</A> and <a href="http://genome.brc.mcw.edu/" target="_empty">mirror</A> 
<LI><a href="http://www.ensembl.org/Mus_musculus/index.html" target="_empty">Ensembl</A>
<LI><a href="http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar" target="_empty">NCBI</A>
<LI><a href="http://phenome.jax.org/pub-cgi/phenome/mpdcgi?rtn=docs/home" target="_empty">Mouse Phenome Database</A> and the  <a href="http://phenome.jax.org/pub-cgi/phenome/mpdcgi?rtn=snps/list_pre" target="_empty">MPP SNP Browser</A>
<LI><a href="http://bioinfo.vanderbilt.edu/webgestalt/" target="_empty">WebGestalt</A> 
<LI><a href="http://grappa.eecs.utk.edu" target="_empty">GrAPPA</A> Clique-based clustering tool
</UL>
</Blockquote>	

<BR>

<Blockquote class="subtitle">Resources for Analysis of Single Genes, SNPs, mRNAs, and Proteins</Blockquote>
<Blockquote>
<Blockquote>
<a href="http://harvester.fzk.de/harvester/" target="_empty">Harvester</a> retrieves summary data on any one of 57,000 proteins from several bioinformatic resources.
<SMALL>[Added Dec 22, 2004; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>

<Blockquote>
<a href="http://mus.well.ox.ac.uk/cgi/gscan/wwwqtl.cgi" target="_empty">GSCAN</a>: the Oxford Wellcome Trust Genome Viewer retrieves mapping data for many mouse experimental mapping populations including the Heterogenous Stock, the Pre-Collaborative Cross mice, and Mouse Diversity Panel.
<SMALL>[Added Oct 13, 2010; last site review Oct 13, 2010 by RWW.]</SMALL>
</Blockquote>




<Blockquote>
<a href="http://nif-apps-stage.neuinfo.org/nif/nifgwt.html" target="_empty">NIF</a>: The Neuroscience Information Framework retrieves summary from a wide variety of neuroscience and  bioinformatic resources. 
<SMALL>[Added Dec 3, 2009; last site review Dec 3, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://genomics.senescence.info/" target="_empty">Genomics of aging resources</a>. A collection of databases and tools designed to help researchers understand the genetics of human ageing through a combination of functional genomics and evolutionary biology. 
<SMALL>[Added Feb 28, 2010; last site review Feb 28, 2010 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.hgmd.cf.ac.uk/ac/index.php" target="_empty">Human Disease Gene Database</a> from Cardiff (requires log in)
<SMALL>[Added Nov 5, 2009; last site review Nov 5, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.snp-nexus.org/" target="_empty">SNPnexus</a> database of human SNPs is designed to simplify and assist in the selection of functionally relevant SNPs for large-scale genotyping studies of multifactorial disorders.<SMALL>[Added Jan 15, 2010; last site review Jan 15, 2010 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.snpedia.com/index.php/SNPedia" target="_empty">SNPedia</a> is a wiki investigating human genetics. We share information about the effects of variations in DNA, citing peer-reviewed scientific publications. It is used by Promethease to analyze and help explain your DNA.<SMALL>[Added Jan 15, 2010; last site review Jan 15, 2010 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="https://www.genoglyphix.com/ggx_browser/search/
" target="_empty">Genoglyphix Browser</a> includes extensive data on human copy number variants as well as maps of low copy number repeat regions. 
<SMALL>[Added Nov 5, 2009; last site review Nov 5, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.hprd.org/query" target="_empty">Human Protein Reference DB</a>: a manually curated resource with data on 20,000 proteins. Very effective interface and rich data.
<SMALL>[Added Oct 30, 2005; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://xmap.picr.man.ac.uk" target="_empty">x:map</a> is a terrific visual display tool for exploring Affymetrix Exon array data sets for mouse and human transcriptomes. 
<SMALL>[Added April 18, 2008; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.pdg.cnb.uam.es/UniPub/iHOP/" target="_empty">iHOP</a> retrieves PubMed sentences that report interactions between a reference gene and associate genes and proteins. It allows the assembly of complex graphs that plot the literature interactions of genes. Effective interface for humans and machines. 
<SMALL>[Added Dec 25, 2004; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.proteinatlas.org/" target="_empty">Human Protein Atlas</a> displays expression and localization of proteins in a large variety of normal human tissues and cancer cells as high resolution images of immunohistochemically stained tissues and cell lines. 
<SMALL>[Added Sept 22, 2007; last site review Sept 22, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.gopubmed.com/" target="_empty">GoPubMed</a> is a simple tool that searches PubMed and sorts the results by GO and MeSH terms. 
<SMALL>[Added July 5, 2007 by RWW; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.ncbi.nih.gov/IEB/Research/Acembly/index.html?human" target="_empty">AceView</a> and the <a href="http://www.ebi.ac.uk/astd/main.html" target="_empty">Alternative Splicing and Transcript Diversity database</A> provide excellent resources for systematic information about the many alternative transcripts produced from single genes.
<SMALL>[Added Jan 1, 2005; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	

<Blockquote>
<a href="http://www.informatics.jax.org/menus/allsearch_menu.shtml" target="_empty">MGI</a> and <a href="http://rgd.mcw.edu/tool-entry.shtml" target="_empty">RGD</a> are reference sites for mouse and rat genetics, respectively. 
<SMALL>[Added Dec 22, 2004; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>

	
<Blockquote>
<a href="http://www.dsi.univ-paris5.fr/genatlas/" target="_empty">GenAtlas</a> provides summary data for approximately 19300 human genes and has a useful link that will fetch 10 Kb of upstream sequence for promoter analysis. 
[Added Jan 9, 2005; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://polly.wustl.edu/promolign/main.html" target="_empty">PromoLign</a> aligns homologous regions of mouse and human promoters and highlights SNPs and transcription factor binding sites. Check the quick tutorial to see how to extract key data. This site requires an SVG plugin that may not be supported by some browsers and operating systems.
[Added May 10, 2005; FAILED: last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://gai.nci.nih.gov/cgi-bin/GeneViewer.cgi?" target="_empty">CGAP SNP Viewer</a> allows users to view SNPs in the context of transcripts, ORFs and protein motifs for either human or mouse genes. Try the <I>Ahr</I> gene in mouse as an example. 
<SMALL>[Added April 10, 2006; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://syndb.cbi.pku.edu.cn/" target="_empty">Synapse Database</a> (SynDB) is a comprehensive database of genes and proteins associated with the neuronal or neuromuscular synapse. Many <B>Trait Data and Analysis</B> pages provide links to SynDB. 
<SMALL>[Added May 29, 2005; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://crb118.med.upenn.edu/syn/dev/syndb/main.php" target="_empty">Synapse Database</a> at University of Pennsylvania is a comprehensive database of roughly 200 genes and proteins associated with the synapse. 
<SMALL>[Added Nov 26, 2006; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://biosingapore.org/index.php/Databases_and_tools" target="_empty">Singapore</a> Bio Databases and Tool. 
<SMALL>[Added Dec 22, 2004; Dragon Genome Explorer site FAILED last site review, changed link; Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
 <a href="http://mutdb.org/AnnoSNP/" target="_empty" >MutDB</a> and <a href="http://www.snps3d.org/modules.php?name=Search&op=advanced%20search" target="_empty">SNPs3D</a> provide great data on functional SNPs in human genes.  To analyze the functional impact of non-synonymous SNPs you will also find <a href="http://snpanalyzer.uthsc.edu/" target="_empty" >SNP Analyzer</a> useful because it evaluates SNP impact in terms of the whole protein structural context.
<SMALL>[Added Dec 22, 2004; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://moult.umbi.umd.edu/mouse2004/modules.php?name=Targets" target="_empty" >Alternative Splicing</a> Project provides great summaries and output graphs on splice variants in human, mouse, and Drosophila.
<SMALL>[Added Nov 8, 2005; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	
</Blockquote>

<BR>
	
<Blockquote class="subtitle">Resources on Imprinting and Parental Origin Effects</Blockquote>
<Blockquote>
<Blockquote>
<a href="http://www.geneimprint.org/site/genes-by-species" target="_empty">Geneimprint</a> is a portal into the burgeoning field of genomic imprinting, collecting relevant articles and reviews, press reports, video and audio lectures, and genetic information   
<SMALL>[Added June 23, 2010 by RWW; last site review June 23, 2010 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://igc.otago.ac.nz/Search.html" target="_empty">Catalogue of Parent of Origin Effects</a> provides a list of imprinted and putatively imprinted genes with commentary by Ian Morison (University of Otago, New Zealand). Database was last updated in 2008.   
<SMALL>[Added June 23, 2010 by RWW; last site review June 23, 2010 by RWW.]</SMALL>
</Blockquote>	
</Blockquote>

<BR>	

<Blockquote class="subtitle">Resources for the Spatial Analysis of Gene and Protein Expression</Blockquote>
<Blockquote>
<Blockquote>
<a href="http://bioinformatics.ubc.ca/resources/links_directory/?subcategory_id=101
" target="_empty">UBC Bioinformatic and Gene Expression Links</a> is a very extensive and well curated collection of on-line resources for the analysis of biological data sets.   
<SMALL>[Added May 24, 2007 by RWW; last site review Sept 19, 2008 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
The <a href="http://mamep.molgen.mpg.de/mamep/search.php?searchtype=simple" target="_empty">mamep GeneExpression Links</a> image database of whole-mounted in situ hybridization of mid-gestation mouse embryos. Try entering the symbol <I>Ptch1</I>.  
<SMALL>[Added May 28, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
The <a href="http://www.genes2cognition.org/genetics.html" target="_empty">Genes to Cognition</a> databases have a focus on proteins expressed in several key cellular compartments related to synpase function.  
<SMALL>[Added Aug 9, 2007 by RWW.]</SMALL>
</Blockquote>

<Blockquote>
<a href="http://www.cdtdb.brain.riken.jp/CDT/CDTSearch.jsp" target="_empty">Cerebellar Development Transcriptome Database</a>. Expression data for the mouse cerebellum, both microarray and in situ. 
<SMALL>[Added Sept 1, 2010; last site review Sept 1, 2010 by RWW.]</SMALL>
</Blockquote>

	
<Blockquote>
Several excellent resources can be used to explore patterns of gene expression primarily in C57BL/6J mice. This strain is one of the parents of the BXD, AXB/BXA, BXH, and CXB genetic reference populations that are key resources in the Gene Network and its companion site, the  <a href="http://www.mbl.org" target="_empty" >Mouse Brain Library</A>.
<UL>
<LI><a href="http://www.stjudebgem.org/web/search/keyword/searchByKeyWordForm.php" target="_empty">BGEM</a> and <a href="http://www.ncbi.nlm.nih.gov/projects/gensat/" target="_empty">GENSAT</a> provide images of gene expression in brains of embryos, neonates, and adult mice (roughly 2008 genes as of July 2005).  
<BR>
<BR>
<LI><a href="http://www.genepaint.org/Frameset.html" target="_empty">GenePaint</a> and <a href="http://www.geneatlas.org/gene/search/searchgene.jsp" target="_empty">GeneAtlas</a> are companion sites that also provide expression data in embryos, neonates, and adults at high spatial resolution. GeneAtlas has excellent but slow image searching and matching capabilities.
<BR>
<BR>
<LI><a href="http://www.brain-map.org/index.jsp" target="_empty">Allen Brain Atlas</a> has expression data for ~12000 transcripts (adult males in the sagittal plane).
<BR>
<BR>
<LI><a href="http://genex.hgu.mrc.ac.uk/Emage/database/emageIntro.html" target="_empty">EMAP</a> (Edinburgh Mouse Atlas Project) provides data on expression of ~800 genes during development (in situ, immunohistochemistry, and reporter knock-in expression patterns). Most data are from wholemounts between Theiler stages 11 and 20 (embryonic days E7 to E13). EMAP can be used as a Java WebStart application.
<BR>
<BR>
<LI><a href="http://www.mouseatlas.org/data/mouse/project_tissue_view" target="_empty">Mouse Atlas of Gene Expression</a> is a massive SAGE library. The Atlas has quantified the normal state for many tissues by determining the number and identity of genes expressed throughout development. The scope of the project encompasses multiple stages of development of C57BL/6J mouse, from the single cell zygote to the adult, and includes an extensive initial collection of 200 tissues. DiscoverySpace is a WebStart application for use with The Mouse Atlas of Gene Expression.
<BR>
<BR>
<LI><a href="http://mahoney.chip.org/mahoney/database.html" target="_empty">Mahoney</a> Center maintains a rich image collection for ~1000 transcription factors expressed in brain (developmental stages, coronal plane).
<SMALL>[Added Dec 22, 2004; sites reviewed last on Sept 26, 2005 by RWW.]</SMALL>
<BR>
<BR>
</UL>
</Blockquote>	
</Blockquote>	

<BR>

<Blockquote class="subtitle">Resources for the Analysis of Sets and Networks of Transcripts, Genes, Proteins, and SNPs</Blockquote>
<Blockquote>
<Blockquote>
<a href="http://www.genemania.org/full.jsf" target="_empty">GeneMANIA</a> helps you predict the function of your favourite genes and gene sets. Powerful and fast computational methods and a great use of Cytoscape Web. (<A HREF="http://nar.oxfordjournals.org/cgi/content/full/38/suppl_2/W214" target="_empty">2010 PDF</A>).
<SMALL>[Added July 1, 2010; last site review Aug 8, 2010 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://toppgene.cchmc.org/" target="_empty">ToppGene Suite</a> A one-stop portal for gene list enrichment analysis and candidate gene prioritization based on functional annotations and protein interactions network 
<SMALL>[Added Jan 16, 2010; last site review Jan 16, 2010 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.wikipathways.org/index.php/WikiPathways" target="_empty">WikiPathways</a> (WGCNA) is an open, public platform dedicated to the curation of biological pathways by and for the scientific community. 
<SMALL>[Added Nov 12, 2009; last site review Nov 12, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.pathwaycommons.org/pc/" target="_empty">Pathway Commons</a> (WGCNA) is a search tool to find  and visualize public biological pathway information. This site collates from several major sites.
<SMALL>[Added Nov 12, 2009; last site review Nov 12, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.sanger.ac.uk/cgi-bin/modelorgs/mousegenomes/snps.pl" target="_empty">Sanger Mouse Genome Project SNP Finder</a> provides access to SNP and indels generated by sequencing 17 strains of mice (plus C57BL/6J). Marvelous.
<SMALL>[Added Nov 18, 2009; last site review Nov 18, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/" target="_empty">Weighted Gene Coexpression Network Analysis</a> (WGCNA) is a collection of R functions to perform weighted correlation network analysis that includes functions for network construction, module detection, gene selection, calculations of topological properties, data simulation, visualization, and interfacing with external software. See the recent paper on <A HREF="http://www.biomedcentral.com/1471-2105/9/559">WGCNA</A>. 
<SMALL>[Added Aug 21, 2009; last site review Aug 21, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.genome.jp/" target="_empty">GenomeNet</a> is a terrific site for the analysis of molecular networks. Download the very effective <a href="http://www.genome.jp/download/" target="_empty">KegArray</A> 0.2.6beta package (May 2005) for exploratory data analysis of microarray data set. This package is as good as most commerical software and includes with built-in linkage to the KEGG databases. Versions are available for Mac OS X and Windows.   
<SMALL>[Added Jan 3, 2005; last site review Aug 5, 2005 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.bioinf.ebc.ee/EP/EP/" target="_empty">Expression Profiler at http://ep.ebi.ac.uk/</a> is a set of tools for clustering, analysis and visualization of gene expression and other genomic data. Tools in the Expression Profiler allow you to perform cluster analysis, pattern discovery, pattern visualization, study and search Gene Ontology categories, generate sequence logos, extract regulatory sequences, study protein interactions, as well as to link analysis results to external tools and databases.
<SMALL>[Added May 20, 2008; last site review May 20, 2008 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.thebiogrid.org/" target="_empty">BioGRID</a>: the Biological General Repository for Interaction Datasets is a freely accessible database of protein and genetic interactions from Mt. Sinai, Toronto. 
<SMALL>[Added July 28, 2007; last site review July 28, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://cismols.cchmc.org/" target="_empty">CisMols Analyzer</a> at Cincinnati Children's Hospital (Aronow and colleagues) is a server and database for the analysis of cis element co-occurences in the promoters of a list of genes. The <a href="http://polydoms.cchmc.org/polydoms/" target="_empty">PolyDoms Analyzer</a> is a tool for scanning through gene lists for those members of a pathway, ontolog, or disease that contain potentially harmful protein-coding SNPs. <a href="http://genometrafac.cchmc.org/" target="_empty">GenomeTraFaC</a> is a comparative genomics-based resource for initial characterization of gene models and the identification of putative cis-regulatory regions of RefSeq Gene Orthologs.
<SMALL>[Added Sept 23, 2006; last site review Sept 23, 2006 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://compbio.uthsc.edu/miRSNP/" target="_empty">PolymiRTS</a> database that searches for microRNA (miRNA) targets in transcripts that overlap SNPs.  This database will also search for genes with associated phenotype variants that may have variants in miRNA target sequence (Yan Cui, Lei Bao  and colleagues). 
 <SMALL>[Added Sept 23, 2006; last site review Sept 23, 2006 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.broad.mit.edu/gsea/msigdb/msigdb_index.html" target="_empty">MSigDB</a>: The Molecular Signature Database is part of the Broad Institute Gene Set Enrichment Analysis suite. MSigDB contains large numbers of static and partly annotated sets of genes/transcripts. Registration is not actually required to download data sets. 
 <SMALL>[Added Jan 18, 2007; last site review Jan 18, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://inia.uchsc.edu/INIA/index.jsp" target="_empty">C-INIA</a> MAGIC-B microarray knowledgebase from the Department of Pharmacology, University of Colorado, Denver, part of the NIAAA INIA project. Extensive public and privated brain array data sets in a powerful analytic web environment. 
 <SMALL>[Added May 31, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.genmapp.org/introduction.asp" target="_empty">GenMAPP 2.0</a> (2004), the Gene Map Annotation and Pathway Prolifer, is a free Windows application (simple registration required) with which you can visualize expression and other genomic data sets on maps of biological pathways. Very flexible suite of programs that you can also use to make custom gene annotation maps (and more).
<SMALL>[Added Aug 5, 2005; last site review Aug 5, 2005 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://bind.ca/" target="_empty">BIND</a> and <a href="http://string.embl.de/" target="_empty">STRING</a> and <a href="http://www.ebi.ac.uk/intact/index.jsp" target="_empty">IntAct</a> are great sites that provide access to well curated data on protein-protein interactions. BIND and IntAct focus on experimentally verified interactions whereas STRING and preBIND incorporate inferred interaction based on other data types, including gene expression. Links to BIND and STRING have been added to the Trait Data and Analysis forms on the GeneNetwork BETA site.
<SMALL>[Added Aug 21, 2005; last site reviews Aug 27, 2005 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://ai.stanford.edu/~erans/cancer/index.html" target="_empty">Microarray Module Maps</a> is a great site that databases a large number of coexpression modules defined using many cancer array studies.
<SMALL>[Added Aug 26, 2005; last site review Aug 26, 2005 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.geneontology.org/GO.tools.microarray.shtml" target="_empty">The Gene Ontology Consortium</a> maintains a well annotated list of open resources for the analysis of large expression data sets and gene ontologies. Note that there are several different lists, each with valuable links.
<SMALL>[Added July 15, 2005; last site review July 15, 2005 by RWW.]</SMALL>
</Blockquote>
<Blockquote>
<a href="http://www.prioritizer.nl/" target="_empty">Prioritizer</a>: Prioritizer is a stand-alone Java program that uses a functional human gene network, available at <a href="http://www.genenetwork.nl" target="_empty">www.genenetwork.nl</a>, to prioritize positional candidate genes that reside within susceptibility loci, by assuming that real disease genes, residing within different loci are functionally closely related within the gene network.</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.bioinformatics.ic.ac.uk/eqtl/" target="_empty">eQTL Explorer</a> is a Java WebStart application that has also been designed for the calculation and display of QTL maps for large rat data sets, particuarly those generated using the HXB strains. Locations of QTLs for both mRNA traits and conventional physiological traits are displayed on chromosome ideograms. High precision QTL maps can also be generated. A password is required to gain access to the primary data files.
<SMALL>[Added January 7, 2006; last site review Jan 7, 2006 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://cliquego.uthsc.edu/" target="_empty">Clique-GO Analysis</a> is a novel tool for extracting cliques of coregulated transcripts. The current data requires Affymetrix U74Av2 probe set IDs as input. Try "103370_at" (the gene <I>Lin7c</I>) as an example.
<SMALL>[Added Jan 4, 2005; last site review Jan 4, 2005 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.bioinformatics.ubc.ca/pavlidis/lab/software.html" target="_empty">Gemma and ErmineJ</a> are powerful resources for analysis and metaanalysis of gene expression data sets at UBC. Pavlidis and colleagues also provide updated <A HREF="http://bioinformatics.ubc.ca/microannots/
">GO data</A> for common microarray platforms.  
<SMALL>[Added Jan 4, 2005; last site review June 7, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.cytoscape.org/index.php" target="_empty">Cytoscape</a> is one of several <a href="http://sbml.org/index.psp" target="_empty">SBML</a>-compatible open source programs for visualizing molecular interaction networks and overlaying these networks with gene expression profiles and other data sets to generate and test specific hypotheses. 
<SMALL>[Added Jan 5, 2005; last site review Jan 5, 2005 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://cgap.nci.nih.gov/Pathways/Pathway_Searcher" target="_empty">Pathway Searcher</a> provides fast access to gene/protein interaction pathways. An intuitive interface. 
<SMALL>[Added Dec 30, 2004; last site review Dec 30, 2004 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://genome.ucsc.edu/cgi-bin/hgNear" target="_empty">Gene Sorter</a> is a tool for generating and sorting sets of genes using a wide variety of information integrated into UCSC's Genome Brower. 
<SMALL>[Added Dec 31, 2004; last site review Dec 31, 2004 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.cisreg.ca/cgi-bin/oPOSSUM/opossum" target="_empty">oPPOSUM</a> is a tool for finding over-represented transcription factor binding sites in lists of mouse and human genes. It handles about 100 out of greater than 600 TFBSs. 
<SMALL>[Added Jan 27, 2005; last site review Nov 21, 2005 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://motif.genome.jp/" target="_empty">MOTIF</a> and <a href="http://www.dbi.tju.edu/dbi/tools/paint/index.php?op=FnetBuilder" target="_empty">PAINT</a> search for motifs in submitted sequences or lists of genes. Paint makes use of the TRANSFAC Pro database. 
<SMALL>[Added Dec 22, 2004; last site review Dec 25, 2004 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://arrowsmith.psych.uic.edu/arrowsmith_uic/index.html" target="_empty">Arrowsmith</a> provides a fast way to evaluate known interactions or common mechanisms between two genes or proteins. It carries out a sophisticated comparison of the current PubMed database. 
<SMALL>[Added Dec 22, 2004; last site review June 7, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.chilibot.net/" target="_empty">Chilibot</a> applies natural-language processing to the PubMed database to hunt for directed relationships among pairs or sets of genes, proteins, and keywords. <SMALL>[Added Dec 30, 2004; last site review Aug 13, 2009 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
Mouse Imprinting Resources:
<a href="http://www.mgu.har.mrc.ac.uk/research/imprinting" target="_empty">The Harwell Mouse Imprinting Resource</a>, 
<a href="http://www.geneimprint.com/" target="_empty">Duke University - Jirtle's Laboratory</a>, 
<a href="http://fantom2.gsc.riken.go.jp/imprinting/" target="_empty">RIKEN Candidate Imprinted Transcript Maps</a>, and
<a href="http://igc.otago.ac.nz/home.html">Imprinted Gene Catalogue - University of Otago</a>.
<SMALL>[Added Oct 20, 2006; last site review Oct 20, 2006 by RWW.]</SMALL>
</Blockquote>	
</Blockquote>

<BR>
	
<Blockquote class="subtitle">Resources for the Analysis of Phenotypes in Genetic Reference Populations</Blockquote>
<Blockquote>
<Blockquote>
<a href="http://www.mbl.org/NewMBL_MySQL/tmbl.php" target="_empty">MBL</a> is a extensive image database of brain sections from genetic reference populations of mice, including the BXD, AXB, CXB, BXH strains included in WebQTL. The MBL is a companion database of WebQTL. 
<SMALL>[Added Dec 22, 2004; last site review Aug 6, 2005 by RWW.]</SMALL>
</Blockquote>	 
<Blockquote>
<a href="http://snp.ucsd.edu/mouse/" target="_empty">MPAD</a> Mouse Phenome Association Database v 1.0, by Eleazar Eskin and Hyun Min Kang. This resource performs genome-wide association mapping. Phenotype data sets are derived from the Mouse Phenome Project set of standard mouse strains. The permutation procedures account for the genetic relations among these strains and provide much more appropriate genome-wide significance thresholds than previous mouse association mapping methods.
<SMALL>[Added Nov 19, 2006; last site review Nov 19, 2006 by RWW. Link broken June 2007 probably due to move from UCSD to UCLA; check with EE.]</SMALL>
</Blockquote>	 
<Blockquote>
<a href="http://gscan.well.ox.ac.uk/gs/wwwqtl.cgi#" target="_empty">GScan</a> at the Wellcome Trust, Oxford, is a sophisticated viewer and analysis tool with which to explore the genetic control of diverse phenotypes (including array data) generated using heterogeneous stock mice (Flint, Mott, and colleagues).
<SMALL>[Added May 28, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://phenome.jax.org/pub-cgi/phenome/mpdcgi?rtn=docs/home" target="_empty">Phenome Project</a> provides access to a wide variety of phenotype data many common and wild inbred strains of mice. 
<SMALL>[Added Dec 22, 2004; last site review Dec 25, 2004 by RWW.]</SMALL>
</Blockquote>	
</Blockquote>	

<BR>

<Blockquote class="subtitle">QTL Mapping Resources</Blockquote>
<Blockquote>
<Blockquote>
<a href="http://compbio.uthsc.edu/QSB/" target="_empty">QSB</a>: QSB is a stand-alone JAVA program with a sophisticated GUI developed for genetical genomics or systems genetics, an emerging field that combines quantitative genetics and genomics. QSB stands for QTL mapping, Sequence polymorphism analysis (or SNP analysis) and Bayesian network analysis. QSB takes marker and array data from a segregating population as input and identifies significant QTLs and then evaluated networks of candidate genes associated with these QTLs. 
<SMALL>[Added July 29, 2005; last site review Jan 7, 2006 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://omicspace.riken.jp/PosMed" target="_empty">Positional Medline (PosMed)</A> is a knowledge-based ranking system of candidate genes within QTL intervals for human, mouse, rat, Arabidopsis, and rice. 
<SMALL>[Added Nov 4, 2009; last site review Nov 4, 2009 by RWW.]</SMALL>
</Blockquote>
<Blockquote><a href="http://omicspace.riken.jp" target="_empty">Genome - Phenome Superbrain Project</a>  integrates various databases to build a comprehensive computerized encyclopedia of omic sciences in several species, including mouse, rat, human, and arabidopsis, etc. The goal is to evolve this intelligent system into a form of artificial intelligence that can solve a researcher's problems by exploiting a vast amount of information accumulated in documents and published data ranging from genomes to phenomes. <SMALL>[Added Sept 13, 2007; last site review Sept12, 2007 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://qtlreaper.sourceforge.net/" target="_empty">QTL Reaper</a>  is software, written in C and compiled as a Python module, for rapidly scanning microarray expression data for QTLs. It is essentially the batch-oriented version of WebQTL. It requires, as input, expression data from members of a set of recombinant inbred lines and genotype information for the same lines. It searches for an association between each expression trait and all genotypes and evaluates that association by a permutation test. For the permutation test, it performs only as many permutations as are necessary to define the empirical P-value to a reasonable precision. It also performs bootstrap resampling to estimate the confidence region for the location of a putative QTL.
<SMALL>[Added Jan 27, 2005; last site review Jan 27, 2005 by KFM.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://ibi.zju.edu.cn/software/qtlnetwork/" target="_empty">QTLNetwork 2.0</a> is a software package ofr mapping QTLs with epistatic and GXE interaction effects in experimental populations including double-haploid, recombinant inbred, backcross, F2, IF2 and BxFy populations. The program provides graphical presentations of QTL mapping results. The software is programmed by C++ programming language under Microsoft Visual C++ 6.0 environment. It works with Microsoft Windows operating systems, including Windows 95/98, NT, 2000, XP, 2003server. A new version of QTLNetwork is under developing, and its functions will be extended to include linkage group construction and marker-assisted virtual breeding.<SMALL>[Added June 21, 2007.]</SMALL>
</Blockquote>	 
<Blockquote>
<a href="http://www.broad.mit.edu/personal/claire/MouseHapMap/Inbred.htm" target="_empty">MouseHapMap</a> project genotypes from Mark Daly and colleagues. Approximately 140,000 SNPs across 49 strains. Updated Feb 2006.used to explore the Oxford Wellcome Heterogeneous stock QTL mapping project population. It currently includes mapping data for 100+ phenotypes typed across 2000 animals and 13,000 SNPs.
<SMALL>[Added May 10, 2006; last site review May 10, 2006 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
A valuable list of <a href="http://www.stat.wisc.edu/%7Eyandell/qtl/software/" target="_empty">Software for QTL Data Analysis</a>  and <B>Gene Expression Microarray Software</B> is managed by Brian Yandell at University of Wisconsin.
<SMALL>[Added May 16, 2006; last site review May 16, 2006 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://gscan.well.ox.ac.uk/gs/wwwqtl.cgi" target="_empty">GSCAN DB</a> is a browser used to explore the Oxford Wellcome Heterogeneous stock QTL mapping project population. It currently includes mapping data for 100+ phenotypes typed across 2000 animals and 13,000 SNPs.
<SMALL>[Added May 10, 2006; last site review May 10, 2006 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://www.well.ox.ac.uk/mouse/INBREDS/" target="_empty">The Wellcome Trust-CTC SNP Data Set</a> consists of high density SNP data for approximately 490 strains of mice at 13,377 SNPs. These data fiels were processed slightly to generate many of the mouse mapping files used in WebQTL.
<SMALL>[Added Sept 27, 2005; last site review Sept 27, 2005 by RWW.]</SMALL>
</Blockquote>	
<Blockquote>
<a href="http://mouse.perlegen.com/mouse/summary_reports.html" target="_empty">The NIEHS-Perlegen Mouse Strain Resequencing Project</a> provides links to SNP data for up to 15 strains of mice. Very high density data for many chromosomes. These data are integrated to some extent in the GeneNetwork.
<SMALL>[Added Sept 25, 2005; last site review Sept 25, 2005 by RWW.]</SMALL>
</Blockquote>	
</Blockquote>

<BR>

<Blockquote class="subtitle">Affymetrix Array Annotation Resources</Blockquote>
<Blockquote>
<Blockquote>
<a href="http://research.stowers-institute.org/efg/ScientificSoftware/Applications/Affy/Annotations/" target="_empty">Affy MOE430A and MOE430B Annotation</a> files are explained more clearly that Affymetrix has ever done by Earl F Glynn  at the Stowers Institute. (efg@stowers-insitute.org). 
<SMALL>[Added July 17, 2006; last site review June 7, 2007 by RWW. This Oct 7, 2005 file caused Grace Wheeler's Mac internet connection to break.]</SMALL>
</Blockquote>	
</Blockquote>

<BR>

<Blockquote class="subtitle">Information about this HTML page: </Blockquote>
<Blockquote>
<Blockquote><P><P>This text originally generated by RWW, Dec 21, 2004. Updated by EJC, Feb 27, 2005; by RWW, July 15, Sept 25.
<P>Management of GeneNetwork access and trait <A HREF="http://gn1.genenetwork.org/manager.html">pages</A>.
</Blockquote>
</Blockquote>
		
		</TD>
		</TR></TABLE>
		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	new menu (MENU_ITEMS, MENU_POS);
	//-->
</script>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3782271-1";
urchinTracker();
</script>
</BODY>
</HTML>