<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Mouse Cross Information</TITLE>
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<LINK REL="stylesheet" TYPE="text/css" HREF='css/general.css'>
<LINK REL="stylesheet" TYPE="text/css" HREF='css/menu.css'>
<SCRIPT SRC="javascript/webqtl.js"></SCRIPT>


</HEAD>
<BODY  bottommargin="2" leftmargin="2" rightmargin="2" topmargin="2" text=#000000 bgColor=#ffffff>
<TABLE cellSpacing=5 cellPadding=4 width="100%" border=0>
	<TBODY>
	<TR>
		<script language="JavaScript" src="/javascript/header.js"></script>
	</TR>
	<TR>
		<TD bgColor=#eeeeee class="solidBorder">
		<Table width= "100%" cellSpacing=0 cellPadding=5><TR>
		<!-- Body Start from Here -->
		<TD valign="top" height="200" width="100%" bgcolor="#eeeeee">
		

<P class="title">Information on Groups of Mice <A HREF="/webqtl/main.py?FormID=editHtml"><img src="images/modify.gif" alt="modify this page" border= 0 valign="middle"></A></P>

&nbsp;&nbsp;&nbsp;&nbsp;<B>GENERAL ADVICE</B> 
		
<Blockquote>
GeneNetwork (GN) contains data from different groups of mice. The BXD genetic reference panel (GRP) will be your best bet if you are just exploring GN. All groups must have a genotype data set (under <B>Type</B>) and most also have a phenotpe data set (also under <B>Type</B>). <I>Phenotype</I> in this case usually means classical trait data that we have gathered from various publications. Molecular expression data, when available, are listed with the suffix "mRNA." 

<P>Here are the groups to select for access to particular types of data:</P>

<UL>

<LI>AKXD: Expression data for mammary tumors generated by Kent Hunter at NCI.

<LI>AXB/BXA: Classical phenotypes that have usualy been collected from the literature. 

<LI>B6BTBRF2: Expression data for liver generated by Alan Attie and colleagues.

<LI>B6D2F2: Expression data for whole brain generated by Robert Hitzemann and colleagues.

<LI>BDF2-2005: Expression data for striatum generated by Robert Hitzemann and colleagues. The striatum is a large forebrain region involved in learning and movement control that is severely affected in Huntington's and Parkinson's diseases.

<LI>BXD: Expression data for brain and several brain regions, including the cerebellum, hippocampus, and striatum. BXD expression data for whole eye, liver, and hematopoietic stem cells.  Classical phenotypes that have usualy been collected from the literature.

<LI>BHF2: F2 cross of BXH. Expression data for adipose, brain, liver, and muscle tissue generated by Jake Lusis at UCLA and Eric Schadt at Rosetta.

<LI>BXH: Only classical phenotypes that have usually been collected from the literature. 

<LI>CXB: Expression data for the hippocampus; a brain region involved in learning and memory, epilepsy, and neurodegenerative diseases. Classical phenotypes have also been collected from the literature. 

<LI>Heterogeneous Stock (HS) mice: Three expression data sets for hippocampus, lung, and liver are currently available. Phenotype and genotype data are also available at http://gscan.well.ox.ac.uk/.

<LI>LXS: Phenotypes that have usually been generated by investigators at the Institute of Behavior Genetics and at the University of Tennessee over the last few years.  We expect the addition of several large brain expression data sets late in 2006.

<LI>MDP: The great majority of data on the Mouse Diversity Panel is taken from the Phenome Project. Unlike the PHone Project, the MDP also includes limited data from the older literature. If you would like to add phenotypes, please submit your data to RW Williams and colleagues.


</Blockquote>



&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="AKXD" class="subtitle">
AKXD:</A> 
		
<Blockquote>
The AKXD recombinant inbred (<A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">RI</A>) strains are derived from AKR/J (AK) and DBA/2J (D). All of these strains were made by Benjamin A. Taylor at the Jackson Laboratory. </P>

<P>All of the AKXD data in GeneNetwork is from an experiment by Kent Hunter and colleagues. GN does not yet include a Phenotypes database for this strain set.</P>

<P>All strains have been genotyped using the Wellcome-CTC-Illumina set of SNPs (13377), as well as some microsatellites, and other markers. WebQTL exploits a total of 5448 markers that are infomative in this mapping panel (Aug 2005).</P>

<P>There are a total of 1027 known recombinations in the AKXD set; an average of 42.8 recombinations per strain (Shifman et al., 2006). </P>


<P>How to obtain these strains: These strains are now cryopreserved. To rederive these strains please contact the Jackson Laboratory and see <a href="http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml"  target="_blank" class="fs14">http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml</a>.
</P>

<P>For more details on the history, generation, and use of RI strains as genetic reference populations for systems genetics please see Silver <A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">(1995)</A>. Additional useful literature links are provided in the <B>References</B> link at the top center of this page.
</P>

</Blockquote>


&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="AXB" class="subtitle"></A><A NAME="BXA" class="subtitle"></A><A NAME="AXB/BXA" class="subtitle">
AXB/BXA:
</A> 
<Blockquote><P>
The AXB and BXA set of recombinant inbred (<A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">RI</A>) strains were derived from a reciprocal cross between A/J (A) and C57BL/6J (B). A particular advantage of this RI set (shared with BXD) is that the two parental strains have both been sequenced and are known to differ at approximately 1.80 million SNPs. Variants (mostly single nucleotide polymorphisms and insertion-deletions) that may produce interesting phenotypes can be located efficiently. The zoomable physical maps in WebQTL display the positions of these A-type versus B-type SNPs down at high resolution.

<P>Data acquired using AXB and BXA subsets should be combined; the only difference being the polarity of intercross matings that generated (A x B)F1s and (B x A)F1s. AXB and BXA strains were all produced by Muriel Nesbitt at UCSD in the mid and late 1970s and first used in the early 1980s (Skamene et al., 1984; Peleg and Nesbitt, 1984; Marshal et al., <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=1477475" target="_blank" class="fs14">1992</a>). The set was imported into The Jackson Laboratory by Beverly Paigen (Pgn) in the early 1990s. As of 2004, approximately <a href="http://jaxmice.jax.org/library/notes/465b.html" target="_blank" class="fs14">25</A> viable and fully independent AXB/BXA strains are available. These strains are not segregating for the more recent A/J retrotransposon mutation in the dysferlin gene (Ho et al., <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15254015&query_hl=7" target="_blank" class="fs14">2004</A>).</P>

<P>Approximately 122 traits are currently included in the AXBXA Phenotypes database  (July 2005).</P> 

<P>All strains have been genotyped using the Wellcome-CTC-Illumina set of SNPs (n = 13377), as well as some microsatellites, and other markers. WebQTL exploits a total of 8514 markers that are infomative in this mapping panel (Aug 2005).</P>

<P>There are a total of approximately 1600 known recombinations (corrected for five "duplicate strains") in the AXB/BXA set; an average of 54.4 recombinations per strain (Shifman et al., 2006). </P>


<P>Several nominally independent strains in the AXB and BXA sets are very closely related. These duplicates should not be used without special statistical precaution. The most obvious option is to combine and average data from these strains except when their phenotypes differ significantly (Taylor <a href="http://jaxmice.jax.org/library/notes/465b.html" target="_blank" class="fs14">1996</a>; Williams et al., <a href="http://genomebiology.com/2001/2/11/research/0046" target="_blank" class="fs14">2001</a>).

<BR>
<BR>AXB13=AXB14: 92.74% identity in an analysis of 8429 markers.
<BR>AXB18=AXB19=AXB20:  97 to 99% identity  (AXB18 to AXB19 = 98.16% identity, AXB18 to AXB20 = 95.72% identity, AXB19 to AXB20 = 97.34% identity n an analysis of 8429 markers) 
<BR>BXA8=BXA17: 99.79% identity in an analysis of 8429 markers. (Updated from Williams et al. 2001).
</P>

<P>How to obtain these strains: Please see <a href="http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml"  target="_blank" class="fs14">http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml</a>.
</P>

<P>For more details on the history, generation, and use of RI strains as genetic reference populations for systems genetics please see Silver <A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">(1995)</A>. Additional useful literature links are provided in the <B>References</B> link at the top center of this page.
</P>

</Blockquote>
		

&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="BXD" class="subtitle">
BXD:
</A> 

<Blockquote><P>
The BXD family of recombinant inbred (<A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">RI</A>) strains were derived by crossing C57BL/6J (B6) and DBA/2J (D2) and inbreeding progeny for 20 or more generations. This genetic reference panel is a remarkable resource because data for thousands of phenotypes and nearly 100 gene, protein, and metabolite expression data sets have been acquired over a nearly a 40-year period. Another advantage of the BXD family is that the both parents have been sequenced (C57BL/6J as part of a public effort, and DBA/2J by Celera Genomics, by the UTHSC group and by Sanger). Based on our analysis of the sequence data, these two strains differ at approximately 4.8 million SNPs. Variants (mostly single nucleotide polymorphisms and about 500,000 insertion-deletions) that produce interesting phenotypes can be located efficiently. The zoomable physical maps in GeneNetwork can display the positions of B versus D-type SNPs at high resolution.
</P>

Our DBA/2J sequence data (from Wang et al. 2016) have been used to generate a virtual genome for this strain using a C57BL/6J framework. In other words, all SNPs and small DBA/2J indels have inserted in place of original C57BL/6J sequence. 

<P>This DBA/2J genome is available at <a target="_blank" href="http://spring211.uthsc.edu/user/mm10_snp/mm10_ucsc_D2snp.fa">here</a>.

<P>Complete variant files for the DBA/2J versus reference genome are here (also on an mm10 assembly): <a target="_blank" href="ftp://gnftp:gnftp@tyche.uthsc.edu/DBA2J/">ftp://gnftp:gnftp@tyche.uthsc.edu/DBA2J/</a>

</BLOCKQUOTE>

<P>
<DIR>
<A HREF="http://gn1.genenetwork.org/bxd/"><img src="/images/upload/BXD20.png" alt="BXD20" border= 0 valign="middle" width="273" height="178"></A>
</DIR>
<DIR><P><SMALL><B>Legend:</B> Photo gallery of BXD strains</SMALL></P>
</DIR>

<BLOCKQUOTE>

<P><B>EPOCH DIFFERENCES or "Batch Effect" among BXD strains</B>. BXD strains (1 through 103) were produced as at least four separate groups or subfamilies. BXD1 through BXD30 were produced by Benjamin A. Taylor starting in about 1971, with the first publication using early generation BXD lines at F7 to F10 in 1973 (<A HREf="http://www.ncbi.nlm.nih.gov/pubmed/4719448">Taylor et al., 1973</A>  <A HREF="/images/upload/Taylor_1973.pdf"> Full text</A>, 1975 (<A HREf="http://www.ncbi.nlm.nih.gov/pubmed/807855">Taylor et al., 1975</A>, <A HREf="http://www.ncbi.nlm.nih.gov/pubmed/1203051">Womack et al., 1975</A>). A distinction is made between an RI line, which is not necessarily fully inbred (<20 F generations of inbreeding, and an RI strain, which should be the progeny of 20 or more sequential sib matings). 

<P>BXD31 and BXD32 are exceptional BXD strains. They were created by the Mouse Mutant Resource at The Jackson Laboratory in the propagation of different visible mutations.  Mutations arose in C57BL/6J or DBA/2 and then outcrossed to the other strain and subsequently inbred while maintaining the mutation. In the case of BXD31, inbreeding was followed after the F2 generation. In BXD32, there was a backcross to DBA/2 before sib mating. BXD32 is therefore approximately 75% DBA/2 and 25% B6. When Taylor learned about these strains he decided that they would be a useful addition to the BXD RI set. He eliminated the visible mutations, continued inbreeding, and added them to the BXDs set. The first publication using these two strains was in 1980. Both are usually lumped together with BXD1 through BXD30 as a "single" first cohort, but obviously, they should perhaps be excluded from all cohorts or BXD "epochs." BXD32 is also exceptional because it has a <I>D</I> mitochondrial genome and a <I>B</I>-type Y chromosome. (Information in this paragraph mainly from BA Taylor, email of Aug 17, 2014 to RWW).

<P>BXD33 through BXD42 were also produced by Benjamin Taylor (Taylor et al. <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/10087289">1999</A>), but from a new set of F2 crosses initiated in the early 1990s.

<P>BXD43 through BXD102 were produced by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <A href="http://www.biomedcentral.com/1471-2156/5/7" target="_blank" class="fs14">2004</A>). These strains have roughly twice the number of recombinations as conventional F2-derived RI strains.

<P>BXD104 through BXD157 were produced by Lu Lu and Robert W. Williams starting in 2008 using standard F2 stock. 

<P>BXD160 through BXD186 were produced by Lu Lu and Robert W. Williams starting in 2010 using G8 and G9 advanced intercross stock donated by Abraham Palmer. 

<P>BXD187 through BXD220 were produced by Lu Lu and Robert W. Williams starting in 2014 using F2 stock. 

<P> Initial genotypes in 2008 highlighted breeding errors that resulted in sets of very closely related sister substrains that are almost genetically identical. In collaboration with the Jackson Laboratory, we have made the following changes in strain names to clarify the strong genetic relations among the newer BXD strains. 

<P>In general, data for the following sister substrains needs to be handled with special care prior to or during statistical analysis or gene mapping for the simple reason that the substrains are not genetically independent. However, if investigators do discover significant phenotype differences among strains, then data can be treated independently.  (The latest version of GeneNetwork (GN 2) includes a mapping method called Python Linear Mixed Model (pyLMM) that corrects for the effects of shared pedigrees and corrects mapping results (code written by Nick Furlotte in 2011-2013). )

<OL>

<LI> <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a>, <a href="http://jaxmice.jax.org/strain/007124" target="_blank">BXD73a</a> (original known as BXD80), and <a href="http://jaxmice.jax.org/strain/007146" target="_blank">BXD73b</a> (originally known as BXD103) are genetically very similar. <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a> and BXD80 are genetically identical at 82264 of 100290 markers (82% identical by descent). <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a> keeps its original name and JAX identifier number (JR#7117), whereas BXD80 is now referred to as <a href="http://jaxmice.jax.org/strain/007124" target="_blank">BXD73a</a> (JR#7124). <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a> and BXD103 are genetically identical at 90917 of 100290 markers (90.6% identical by descent). BXD103 is now referred to as <a href="http://jaxmice.jax.org/strain/007146" target="_blank">BXD73b</a> (JR#7146). 

<LI> <a href="http://jaxmice.jax.org/strain/007097" target="_blank">BXD48</a> and <a href="http://jaxmice.jax.org/strain/007139" target="_blank">BXD48a</a> (originally known as BXD96) are sister substrains, and are genetically identical at 93485 of 100290 markers (93.2% identical by descent). <a href="http://jaxmice.jax.org/strain/007097" target="_blank">BXD48</a> retains its original name and JAX identifier number (JAX JR#7097) whereas BXD96 is now referred to as <a href="http://jaxmice.jax.org/strain/007139" target="_blank">BXD48a</a> (JR#7139). 

<LI> <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a>, <a href="http://jaxmice.jax.org/strain/007140" target="_blank">BXD65a</a> (originally known as BXD97), and <a href="http://jaxmice.jax.org/strain/009677" target="_blank">BXD65b</a> (originally known as BXD92) are sister substrains.  <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a> and BXD97 are genetically identical at 92225 of 100290 markers (92% identical by descent). BXD97 is now referred to as <a href="http://jaxmice.jax.org/strain/007140" target="_blank">BXD65a</a> (JR#7140). <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a> and BXD92 are genetically identical at 6155 of 6459 markers (95.3% identical by descent). <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a> retains its original name and JAX identifier (JR#7110) whereas BXD92 is now referred to as <a href="http://jaxmice.jax.org/strain/009677" target="_blank">BXD65b</a> (JR#9677).

</OL>

<P> 

<P>While the strains used to generate these subsets of BXDs have the same official names and were all made using stock from the Jackson Laboratory, the individual parents were are not genetically identical due to inevitable genetic drift and mutation. Shifman and colleagues detected a surprisingly large number of new SNPs (n = 47 out of about 13000 SNPs studied) in the set of strains generated by BA Taylor in the early 1990s, and a small number (n = 5) of even newer SNPs in the set of BXD strains generated at UTHSC in the late 1990s (see <A HREF="http://http://www.plosbiology.org/article/info:doi/10.1371/journal.pbio.0040395">Shifman et al., 2006</A>).

<DIR><SMALL><P>"In the BXD set, 52 SNPs showed variation in genotypes that corresponded to the different phases of development of the BXD RIs [24–26] (Table S4). Forty-seven SNPs are not polymorphic in the 26 BXD strains established from a single cross of a C57BL/6J female to a DBA/2J male, but are polymorphic in similar BXD strains established more than 20 y later. Five SNPs are not polymorphic in the first 36 BXD strains, but are polymorphic in the newest set of 53 BXD lines (BXD43–100)."
</DIR></SMALL>

<P><B>Correction for Family or Epoch Substructure</B>

<P>The BXDs have the following epoch substructure:
<OL>
<LI> BXD1 through 30 make up the first epoch. Breeding for this group of BXD strains started in about 1970, with the first publication of fully inbred BXD strains in 1975 (Taylor et al., <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/807855>1975</A>)see Trait ID 13032 from BXD Published Phenotypes).
 
<LI>BXD31 and 32 are usually considered part of this same epoch but were not part of the same original cross, and their first use in a publication did not occur until the early 1980 (e.g., Brownstein, <A HREF="http://iai.asm.org/cgi/reprint/41/1/308?view=long&pmid=6305840>1983</A>, see Trait ID 10715). In fact, BXD32 has a mitochondrion that is inherited from DBA/2J. BXD32 could be considered the first DXB strain (DXB32).

<LI>BXD33 to BXD42 make up Ben Taylor's final addition to the BXD strains (Taylor et al, <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/10087289>1999</A>). These strains were added to enhance the utility of the BXDs for complex trait analysis starting in about 1992. The first published use of this strains appears was in 2001 (Williams et al., <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/11529276?dopt=Abstract>2001</A>, Trait ID 10645).

<LI>BXD43 to BXD103. This is a complex cohort of strains generated at UTHSC from advanced intercross progeny (Peirce et al., <A HREF="http://www.biomedcentral.com/1471-2156/5/7">2004</A>). 

<LI>BXD104 to BXD157. This is a single cohort of strains generated at UTHSC from F2 intercross stock. 

<LI>BXD160 to BXD186. This is a single cohort of strains generated at UTHSC from G8 and G9 advanced intercross progeny donated to RW Williams by Abraham Palmer in 2008. 

<LI>BXD187 to BXD220. This is a single cohort of strains generated at UTHSC from F2 intercross stock. 

</OL>

Users of the expanded BXD panel should take this epoch substructure into account. This is easy to do using the "Epoch" traits that are included in the BXD Phenotype database. For example, BXD Phenotype 12688 (BXD epoch batch trait 1) provides a simple code for the major phases of BXD production using the code of -1 for the first set through to BXD32, 0 for the second set (33 to 42), and +1 for the newer UTHSC set (43 to 103). 

<OL>
<LI> Determine whether your trait covaries well with any one of the three Epoch traits in GeneNetwork. Also check the status of BXD31 and BXD32. They may not belong to any group.

<LI>Determine if your trait maps extremely well to Chr 4 at 62 Mb (near the ALAD segmental duplication in DBA/2J).

</OL> 

<P>Strain nomenclature:  Some of the BXD strains have accumulated new mutations that have recently been characterized. When these mutations are known, the full nomenclature of the strain is now being modified. For example, BXD24/TyJ (aka BXD24 in most GeneNetwork databases), suffered a mutation in the <I>Cep290</I> gene in the late 1980s. The mutant allele (<I>rd16</I> is associated with autosomal recessive retinal degeneration. The original BXD strain was briefly referred to as BXD24a/TyJ, while the blind co-isogenic mutant was referred to as BXD24b/TyJ. The great majority of phenotype, expression, and genotype data in GeneNetwork was generated using these blind BXD24b/TyJ animals. However, in 2010, the nomenclature was changed again and the blind variant (JAX stock 000031) is now known as  BXD24/TyJ-Cep290rd16/J. The original BXD with normal vision was rederived from frozen stock and is now known once again as BXD24/TyJ, although the stock number has now been changed to 005243.

<P>BXD29/TyJ was also known as BXD29/TyJ-Tlr4, but is now formally BXD29-Tlr4lps-2J/J (JAX stock 000029). The original non-mutant stock is currently known as BXD29/TyJ again but the stock number of these rederived non-mutants has been changed to 010981.

<P>The mitochondrial DNA of all BXD strains were typed by Jing Gu and Shuhua Qi (Nov 2004) using DNAs obtained from the Jackson Laboratory (BXD1 through 42) or from the UTHSC colony. This typing relied on a SNP marker identified by Jan Jiao in Weikuan Gu's laboratory at nucleotide position 9461 in the reference C57BL/6J mitochondrial sequence. Most strains have inherited mitochondria from C57BL/6J. However, the following strains have mitochondria with a <I>D</I> allele at the rs8281487 (UT-M-9461) SNP in ND3 (T is reference and C is alternate allele): BXD32, 61, 74, 78, 82, 89, 90, 91, 95, and BXD99. These ten strains could be considered DXB recombinant inbred strains. </P>

<P>Genotypes of these strains: All BXD strains were genotyped in the first half of 2005 at 13377 markers as part of a CTC-Wellcome Trust collaboration. When combined with previous markers, there are a total of 7636 informative markers that differ betweeen the parental strains and that are useful for mapping with the BXD strains. The locations of these makers are known on the latest assembly of the mouse genome (Build 34, mm6). The median distance between these informative markers is 178,831 bp. The mean distance is 324,493 bp. There are only 26 intervals between markers that are longer than 5 Mb. No interval is greater than 10 Mb except on Chr X. These long intervals are essentially monomorphic between the parental strains. 
</P>

<P>The BXD genotype file used in GeneNetwork up to 2014 includes a selected subset of approximately 3795 markers (out of 7636) and includes all those markers with unique strain distribution patterns (<A HREF="http://gn1.genenetwork.org/glossary.html#SDP" target="_blank" class="fs14">SDP</A>) as well as pairs of markers--the most proximal and most distal--for SDPs represented by two or more markers. Slightly updated versions of this BXD genotype data set can be downloaded by ftp at <A HREF="ftp://atlas.uthsc.edu/Public/BXD_WebQTL_Genotypes" target="_blank" class="fs14">ftp://atlas.uthsc.edu/Public/BXD_WebQTL_Genotypes</A>.
</P>

<P>There are a total of 1848 known recombinations in the 36 older (JAX) BXD set; an average of 48.1 recombinations per strain. </P>

<P>There are a total of 4366 known recombinations in the 53 of the first set of UTHSC BXD strains (BXD43 to BXD102); an average of 82.4 recombinations per strain (Shifman et al., 2006). These RI strains were generated from an advanced intercross, and this accounts for the higher recombination load (Peirce et al., 2005).</P>

<P>The "classic" genotypes of the BXD strains (used through Dec 2016 in GeneNetwork) rely mainly on the Mouse Universal Genotyping Array (MUGA) genotyping platform. They also rely to some extent and earlier genotyping in 2008 using the Affymetrix Mouse Diversity array and even earlier work by Williams et al (2001) using microsatellite markers.

<P>In January 2017, the genotypes of most extant BXD strains were updated. The new data also include initial genotypes for the newest cohort of BXDs (BXD104 to BXD220). This January 2017 genotype file provides consensus genotypes for 198 BXD strains. Of the 198 BXD strains, 191 are independent, whereas 7 are substrains (e.g., BXD48 and BXD48a). This file provides approximate locations of 11500 recombinations, an average of 58 per strain. Genotypes were generated using Affymetrix, MUGA, MegaMUGA, and GigaMUGA Illumina platforms. Microsatellites and eQTL genotypes were generated by the Williams and Lu laboratory. Unknown genotypes were imputed as B or D, or were called as H (heterozygous) if the genotype was uncertain. Genotypes were manually curated by RW Williams. Genotypes were smoothed to remove unlikely recombination events. Almost all recombinations are supported by multiple markers, although only one or two representative markers may be provided in this file. The original parent file (BXD_El_Grande_Master_Used_to_Proof_Final_Genotypes_2016.xlxs) contains data for approximately 37000 markers. A subset of the most informative 7300 markers are included in the 2017 genotype file. Genotypes for Chr Y and  Chr M are provisional and will be verified in 2017. As of 2016, many strains with higher numbers (BXD100 and above) are not fully inbred.


<P>Approximately 5100 phenotypes are currently included in the BXD Phenotype database in GeneNetwork as of Jan 2017. You can get an update on this number by typing in an asterisk (*) in the search box. </P>

<P>How to obtain these strains: Please see <a href="http://jaxmice.jax.org/strain/000105.html"  target="_blank" class="fs14">http://jaxmice.jax.org/strain/000105.html</a>. Cost of the JAX BXD strains was approximately $65.40 each in 2008, $135.00 each in 2014, and $139.90 in 2017. To obtain strains BXD43 and higher please contact <A HREF="mailto:rwilliams@uthsc.edu" class="fs14">Rob Williams</A>. All strains through BXD102 are now fully inbred. We expect to generate as many as 160 viable BXDs strains by 2020. All extant BXD strains are being sequenced in early 2017 at 30X using 10X Chromium libraries at Hudson Alpha (supported by the UTHSC CITG, Williams, Lu, and colleagues at UTHSC, Abraham Palmer and colleagues at UCSD, and by Jonathan Pritchard at Stanford).
</P>

<P>For more details on the history, generation, and use of RI strains as genetic reference populations for systems genetics please see Silver <A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">(1995)</A>. Additional useful literature links are provided in the <B>References</B> link at the top center of this page.
</P>

</Blockquote>


&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="BHF2" class="subtitle">
BHF2:</A> 
<Blockquote>

Information on array platform <A href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL2510">GEO GPL2510</A>.
<P>
This June 2005 data freeze provides estimate of mRNA expression in (adult) brains of F2 intercross mice (C57BL/6J x C3H/HeJ) on ApoE null backgrouds, measured using Agilent microarray pairs. Data were generated at The Univesity of California Los Angeles (UCLA), by Jake Lusis and Thomas Drake. Data were processed using mlratio method developed by He and colleagues (2003 -- Paper with He and Schadt).

<P>
The (C57BL/6J X C3H/HeJ)F2 intercross consists of 334 animals of both sexes. All are ApoE null and received a high fat "Western" diet from 8-24 weeks of age. These have been genotyped for QTL mapping, and various phenotypes measured.

<P>For more information contact Leslie Ingram or Jake Lusis at UCLA.

</Blockquote>



&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="BXH" class="subtitle">
BXH:</A> 
<Blockquote>
<P>
The BXH set of recombinant inbred strains (<A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">RI</A>) were made by crossing female C57BL/6J (B) with male C3H/HeJ (H) mice. Benjamin Taylor created the initial 12 BXH recombinant inbred strains at The Jackson Laboratory in 1969. A second set of eight BXH strains were generated by Linda Siracusa at the Kimmel Cancer Center (Kcc) in 1995. She selected for tyrosinase-negative albinos; a gene on the central part of chromosome (Chr) 7. Four of these new BXH strains (one now relabeled as a recombinant congenic) are now also available from The Jackson Laboratory. The following are the old and new symbols for the four recent additions:

<UL>
<LI>BXHA1/Sr = BXH20/Kcc
<LI>BXHA2/Sr = BXH21/Kcc
<LI>BXHB2/Sr = BXH22/Kcc
<LI>BXHE1/Sr = B6cC3-1/Kcc  (backcrossed to B6 and a recombinant congenic)
</UL>

<P>Approximately 142 traits are currently included in the BXH Phenotype database (July 2005).</P>

<P>All strains have been genotyped using the Wellcome-CTC-Illumina set of SNPs (13377), as well as some microsatellites, and other markers. WebQTL exploits a total of 8311 markers that are infomative in this mapping panel (Aug 2005).</P>

<P>There are a total of 775 known recombinations in the 16 core BXH strains; an average of 48.4 recombinations per strain (Shifman et al., 2006). </P>

<P>How to obtain these strains: Please see <a href="http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml"  target="_blank" class="fs14">http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml</a>.
</P>

<P>For more details on the history, generation, and use of RI strains as genetic reference populations for systems genetics please see Silver <A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">(1995)</A>. Additional useful literature links are provided in the <B>References</B> link at the top center of this page.
</P>

<P>BXH2 is susceptible to M. bovis (tuberculosis) and malaria infections despite Nramp1 resistance due to an Icsbp1 (Irf8) mutation. (P Gros and colleagues).

</Blockquote>

&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="CFW" class="subtitle">CFW:</a>
<Blockquote>
Carworth Farms White (CFW) mice are a commercially available outbred mouse population that exhibit rapid LD decay in comparison to other available mouse populations. We performed a genome-wide association study (GWAS) of behavioral, physiological and gene expression phenotypes using 1,200 male CFW mice. We used genotyping by sequencing (GBS) to obtain genotypes at 92,734 SNPs. We also measured gene expression using RNA sequencing in three brain regions. Our study identified numerous behavioral, physiological and expression quantitative trait loci (QTLs). We integrated the behavioral QTL and eQTL results to implicate specific genes, including Azi2 in sensitivity to methamphetamine and Zmynd11 in anxiety-like behavior. The combination of CFW mice, GBS and RNA sequencing constitutes a powerful approach to GWAS in mice.
</Blockquote>
&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="CXB" class="subtitle">
CXB:</A> 
<Blockquote>
The CXB set is the first, oldest, and smallest group of mouse recombinant inbred (<A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">RI</A>) strains with the first publication by Donald Bailey in 1971 (Recombinant-inbred strains. An aid to finding identity, linkage, and function of histocompatibility and other genes. <A HREF="/images/upload/Bailey_1971_Transplantation.pdf" target="_blank" class="fs14">Transplantation 11:325-328</A>). By 1975 the first set of 8 CXB strains were all beyond generation F30. The maternal strain is BALB/cBy and the paternal strain is C57BL/6By. Donald Bailey, the inventor of recombinant inbred strains, created the initial eight BXH recombinant inbred strains at The Jackson Laboratory in the mid 1960s. They were then called CXB-A through CXB-H. They have been used extensively by immunologists and neurogeneticists. A total of 13 of these strains are now available. </P>

<P>Approximately 506 traits are now included in the CXB Phenotype database (July 2005).</P>

<P>All strains have been genotyped using the Wellcome-CTC-Illumina set of SNPs (13377), as well as some microsatellites, and other markers. WebQTL exploits a total of 1384 markers that are infomative in this mapping panel (Aug 2005).</P>

<P>There are a total of 694 known recombinations in the 13 CXB strains; an average of 53.4 recombinations per strain (Shifman et al., 2006). </P>
	
<P>How to obtain these strains: Please see <a href="http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml"  target="_blank" class="fs14">http://jaxmice.jax.org/jaxmicedb/html/rcbinbred.shtml</a>.
</P>

<P>For more details on the history, generation, and use of RI strains as genetic reference populations for systems genetics please see Silver <A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">(1995)</A>. Additional useful literature links are provided in the <B>References</B> link at the top center of this page.
</P>

</BLockquote>
	

&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="LXS" class="subtitle">LXS:</A> 
		
<Blockquote><P>
The parental strains of the LXS recombinant inbred (<A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">RI</A>) set are Inbred Long-Sleep (ILS) and Inbred Short-Sleep (ISS) strains. These parental strains have been phenotyped intensively by behavioral geneticists and neuropharmacologists for a decade. </P>

P>The LXS RI set has an intriguing history and trace back to an 8-way cross initiated in the 1950s by Gerald McClearn, the dean of mouse behavior genetics. The following 8 strains were bred using a circle breeding method: A, AKR, BALB/c, C3H/2, C57BL, DBA/2, IS/Bi, and RIII. All of these strains were maintained at the Institute for Behavior Genetics, Bolder Colorado by McClearn and colleagues. C3H/2 is presumably the same as C3H/Crgl/2 (see paper by Green V (1981) Behavioral and Neural Biology 31:56). C57BL is presumably the same as C57BL/Crgl. IS/Bi is extinct. 

<P>See Williams, Bennett, Johnson and colleagues <A HREF="/images/upload/LXS.pdf">(2004)</A> for more details on the LXS panel.
</P>

<P>The LXS panel has recently been genotyped at approximately 330 microsatellites (Williams et al., 2005)  and at 5000 informative SNPs (Wellcome-CTC consortium). The GeneNetwork uses a subset of 2659 markers to map Mendelian and quantitative trait loci in this large panel.</P>

<P>As of July 2005 approximately 10 cohorts of traits (71 total records) have been entered into GeneNetwork's Phenotypes database. </P>

<P>All of the LXS strains have been genotyped using the Wellcome-CTC-Illumina set of SNPs (13377), as well as some microsatellites, and other markers. WebQTL exploits a total of 5178 markers that are infomative in this mapping panel (Aug 2005).</P>

<P>There are a total of 3598 known recombinations in the 77 LXS strains genotyped by Wellcome-CTC; an average of 46.7 recombinations per strain (Shifman et al., 2006). </P>

<P>For information on the availability of the LXS strains please contact <a href="mailto:bennettb@colorado.edu" class="fs14">Beth Bennett</span></a>.
</P>

<P>For more details on the history, generation, and use of RI strains as genetic reference populations for systems genetics please see Silver <A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">(1995)</A>. Additional useful literature links are provided in the <B>References</B> link at the top center of this page.
</P>

</Blockquote>



&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="MDP" class="subtitle">
Mouse Diversity Panel:</A> 

<Blockquote>
The Mouse Diversity Panel consists of approximately 122 of the major inbred strains, substrains, congenic strains, and even some common mutant strains of mice used in biomedical research. Full descriptions of most of these strains are available from the <A HREF="http://jaxmice.jax.org/query/" class="fs14">Jackson Laboratory</A>, and especially the <A HREF="http://phenome.jax.org/pub-cgi/phenome/mpdcgi?rtn=docs/home" target="_blank" class="fs14">Phenome Project</A> web site. We have usually listed specific substrains of mice, most with the "/J" suffix.

<P>For information on the correct nomenclature for the 129 strains see <A HREF="http://www.informatics.jax.org/mgihome/nomen/strain_129.shtml" target="_blank" class="fs14">http://www.informatics.jax.org/mgihome/nomen/strain_129.shtml</A> 

<P>Known nomenclature problems (April 2008) 
<OL>
<LI>BTBR T+ tf/J is also listed at the bottom of the Trait Data page as BTBRT<+>tf/J. This is a single strain.
<LI>C57Bl/6ByJ should be listed as C57BL/6ByJ
<LI>CZECHI/EiJ should probably be CZECHII/EiJ
</OL>

</Blockquote>





&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="B6D2F2" class="subtitle">
B6D2F2:</A> 
		<Blockquote>
			
<P>
Fifty-six <i>Filial generation 2</I> (F2) mice were generated by crossing C57BL/6J (B6) and DBA/2J (D2) stock  from the Jackson Laboratory. The F1s were mated reciprocally to create B6D2F2 and D2B6F2 progeny. At present , WebQTL includes one large microarray data set (Affymetrix M430) for the entire brain of these F2 progeny.</P>

<P>
For further information, please contact <A HREF="mailto:belknajo@ohsu.edu" class="fs14">John Belknap</A>,  Department of Behavioral Neuroscience, Oregon Health & Science University, Portland VA Medical Center, Portland, OR 97239.</P>
</Blockquote>



&nbsp;&nbsp;&nbsp;&nbsp;<A NAME="B6BTBRF2" class="subtitle">
B6BTBRF2:</A> 

<Blockquote>
This cross consists of a subset of 60 F2 progeny generated by crossing C57BL/6J and BTBR strains. All of these cases are homozygous for the spontaneous obese mutation in the leptin gene (Lep-ob/ob). Metabolic function, liver mRNA expression (Agilent platform), and other physiological and molecular traits related to type 2 diabetes and obesity were quantified. Liver gene expression data were generated by Hong Lan and Alan Attie at The University of Wisconsin-Madison. Please contact Drs.<A HREF="mailto:attie@biochem.wisc.edu" class="fs14"> Alan Attie</A> regarding use of this data set in publications or projects.
</Blockquote>


<P class="subtitle">&nbsp;&nbsp;&nbsp;&nbsp;About this file:</P> 
<Blockquote><P> The file started, Nov 5, 2004 by RWW. Last update by RWW,  Nov 6, Dec 17, 2004; April 10, 2005; July 15, 2005. EJC Mar 23, 2006. RWW July 26, 2006. RWW April, 2008. EGW July, 2008. RWW, June 2012.</P></Blockquote>

		
		</TD>
		</TR></TABLE>
		</TD>
	</TR>
	<TR>
		<TD align=center bgColor=#ddddff class="solidBorder">
		<!--Start of footer-->
		<TABLE width="90%">
		<script language='JavaScript' src='/javascript/footer.js'></script>
		</TABLE>
		<!--End of footer-->
		</TD>
	</TR>
</TABLE>
<!-- /Footer -->
<script language="JavaScript" src="/javascript/menu_new.js"></script>
<script language="JavaScript" src="/javascript/menu_items.js"></script>
<script language="JavaScript" src="/javascript/menu_tpl.js"></script>
<script language="JavaScript">
	<!--//
	new menu (MENU_ITEMS, MENU_POS);
	//-->
</script>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3782271-1";
urchinTracker();
</script>
</BODY>
</HTML>